<!doctype html>
<html lang="en">
  <head>
     
    <title>Colorectal Oncogenomics Group: Publications</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <link href="assets/css/site.css" rel="stylesheet" />
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
         

      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">COG Lab</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="projects.html">Projects</a>
                                  <a class="nav-item nav-link"href="team.html">Team</a>
                                  <a class="nav-item nav-link active"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="partners.html">Partners</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Publications</h1>


  <h3 class="my-5">2017</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Tumor testing to identify lynch syndrome in two Australian colorectal cancer cohorts</a>, Journal of gastroenterology and hepatology, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Clendenning M, Rosty C, Eriksen SV, Walsh MD, Walters RJ, Thibodeau SN, Stewart J, Preston S, Win AK, Flander L, Ait Ouakrim D, Macrae FA, Boussioutas A, Winship IM, Giles GG, Hopper JL, Southey MC, English D, Jenkins MA, Melbourne Collaborative Cohort,  Study, the Australasian Colorectal Cancer Family Registry Cohort,  investigators</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of gastroenterology and hepatology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">32</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1111/jgh.13468</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27273229</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1111/jgh.13468">http://dx.doi.org/10.1111/jgh.13468</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal cancer Lynch syndrome immunohistochemistry microsatellite instability, MLH1, MSH2, MSH6, PMS2, MLH1 methylation, BRAFV600E mismatch repair protein expression</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND AND AIM: Tumor testing of colorectal cancers (CRC) for mismatch repair (MMR) deficiency is an effective approach to identify carriers of germline MMR gene mutation (Lynch syndrome). The aim of this study was to identify MMR gene mutation carriers in two cohorts of population-based CRC utilizing a combination of tumor and germline testing approaches. METHODS: Colorectal cancers from 813 patients diagnosed with CRC < 60 years of age from the Australasian Colorectal Cancer Family Registry (ACCFR) and from 826 patients from the Melbourne Collaborative Cohort Study (MCCS) were tested for MMR protein expression using immunohistochemistry, microsatellite instability (MSI), BRAFV600E somatic mutation, and for MLH1 methylation. MMR gene mutation testing (Sanger sequencing and Multiplex Ligation Dependent Probe Amplification) was performed on germline DNA of patients with MMR-deficient tumors and a subset of MMR-proficient CRCs. RESULTS: Of the 813 ACCFR probands, 90 probands demonstrated tumor MMR deficiency (11.1%), and 42 had a MMR gene germline mutation (5.2%). For the MCCS, MMR deficiency was identified in the tumors of 103 probands (12.5%) and seven had a germline mutation (0.8%). All the mutation carriers were diagnosed prior to 70 years of age. Probands with a MMR-deficient CRC without MLH1 methylation and a gene mutation were considered Lynch-like and comprised 41.1% and 25.2% of the MMR-deficient CRCs for the ACCFR and MCCS, respectively. CONCLUSIONS: Identification of MMR gene mutation carriers in Australian CRC-affected patients is optimized by immunohistochemistry screening of CRC diagnosed before 70 years of age. A significant proportion of MMR-deficient CRCs will have unknown etiology (Lynch-like) proving problematic for clinical management.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lack of evidence for germline RNF43 mutations in patients with serrated polyposis syndrome from a large multinational study</a>, Gut, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Clendenning M, Zhuoer L, Stewart JR, Joseland S, Woodall S, Arnold J, Semotiuk K, Aronson M, Holter S, Gallinger S, Jenkins MA, Sweet K, Macrae FA, Winship IM, Parry S, Rosty C, Genetics of Colonic Polyposis S</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gut</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">66</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1136/gutjnl-2016-312773</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27582512</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1136/gutjnl-2016-312773">http://dx.doi.org/10.1136/gutjnl-2016-312773</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colon carcinogenesis Colonic polyps Colorectal cancer genes Gene mutation Polyposis</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Buchanan, Daniel D Clendenning, Mark Zhuoer, Li Stewart, Jenna R Joseland, Sharelle Woodall, Sonja Arnold, Julie Semotiuk, Kara Aronson, Melyssa Holter, Spring Gallinger, Steven Jenkins, Mark A Sweet, Kevin Macrae, Finlay A Winship, Ingrid M Parry, Susan Rosty, Christophe eng Letter England 2016/09/02 06:00 Gut. 2017 Jun;66(6):1170-1172. doi: 10.1136/gutjnl-2016-312773. Epub 2016 Aug 31.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk of colorectal cancer for carriers of a germline mutation in POLE or POLD1</a>, Genetics in medicine, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Stewart JR, Clendenning M, Rosty C, Mahmood K, Pope BJ, Jenkins MA, Hopper JL, Southey MC, Macrae FA, Winship IM, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Genetics in medicine</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/gim.2017.185</dd>
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/gim.2017.185">http://dx.doi.org/10.1038/gim.2017.185</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">POLE, POLD1, Colorectal cancer, penetrance, polymerase proof reading associated polyposis</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Background: Germline mutations in the exonuclease domains of the POLE and POLD1 genes are associated with an as yet unquantified increased risk of colorectal cancer (CRC). Methods: We identified families with POLE or POLD1 variants by searching PubMed for relevant studies prior to October 2016 and by genotyping 669 population-based CRC cases diagnosed <60 years of age from the Australasian Colorectal Cancer Family Registry. We estimated the age-specific cumulative risks (penetrance) using a modified segregation analysis. Results: We observed 67 CRCs (mean age at diagnosis=50.2 (standard deviation [SD]=13.8) years) among 364 first- and second- degree relatives from 41 POLE families and 6 CRCs (mean age at diagnosis=39.7 (SD=6.83) years) among 69 relatives from 9 POLD1 families. We estimated risks of CRC to age 70 years (95% confidence interval [CI]) for males and females, respectively, to be: 40%(26%–57%) and 32%(20%–47%) for POLE mutation carriers; and 63%(15%–99%) and 52%(11%–99%) for POLD1 mutation carriers. Conclusion: CRC risks for POLE mutation carriers are sufficiently high warranting consideration of annual colonoscopy screening and management guidelines comparable to Lynch syndrome. Refinement of estimates of CRC risk for POLD1 carriers is needed, however, clinical management recommendations could follow those suggested for POLE carriers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Modeling of successive cancer risks in Lynch syndrome families in the presence of competing risks using copulas</a>, Biometrics, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Choi YH, Briollais L, Win AK, Hopper J, Buchanan D, Jenkins M, Lakhal-Chaieb L</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Biometrics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">73</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1111/biom.12561</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27378229</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1111/biom.12561">http://dx.doi.org/10.1111/biom.12561</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Ascertainment correction Missing covariates Penetrance function Successive competing risks</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">In this article, we propose an association model to estimate the penetrance (risk) of successive cancers in the presence of competing risks. The association between the successive events is modeled via a copula and a proportional hazards model is specified for each competing event. This work is motivated by the analysis of successive cancers for people with Lynch Syndrome in the presence of competing risks. The proposed inference procedure is adapted to handle missing genetic covariates and selection bias, induced by the data collection protocol of the data at hand. The performance of the proposed estimation procedure is evaluated by simulations and its use is illustrated with data from the Colon Cancer Family Registry (Colon CFR).</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Somatic mutations of the coding microsatellites within the beta-2-microglobulin gene in mismatch repair-deficient colorectal cancers and adenomas</a>, Familial cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Clendenning M, Huang A, Jayasekara H, Lorans M, Preston S, O'Callaghan N, Pope BJ, Macrae FA, Winship IM, Milne RL, Giles GG, English DR, Hopper JL, Win AK, Jenkins MA, Southey MC, Rosty C, Buchanan DD, investigators from the Melbourne Collaborative Cohort,  Study, the Australasian Colorectal Cancer Family Registry,  Cohort</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-017-0013-y</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28616688</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-017-0013-y">http://dx.doi.org/10.1007/s10689-017-0013-y</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">B2m Colorectal cancer Lynch syndrome MLH1 methylation Microsatellite instability Mismatch repair deficiency</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">In colorectal cancers (CRCs) with tumour mismatch repair (MMR) deficiency, genes involved in the host immune response that contain microsatellites in their coding regions, including beta-2-microglobulin (B2M), can acquire mutations that may alter the immune response, tumour progression and prognosis. We screened the coding microsatellites within B2M for somatic mutations in MMR-deficient CRCs and adenomas to determine associations with tumour subtypes, clinicopathological features and survival. Incident MMR-deficient CRCs from Australasian Colorectal Cancer Family Registry (ACCFR) and the Melbourne Collaborative Cohort Study participants (n = 144) and 63 adenomas from 41 MMR gene mutation carriers from the ACCFR were screened for somatic mutations within five coding microsatellites of B2M. Hazard ratios (HR) and 95% confidence intervals (CI) for overall survival by B2M mutation status were estimated using Cox regression, adjusting for age at CRC diagnosis, sex, AJCC stage and grade. B2M mutations occurred in 30 (20.8%) of the 144 MMR-deficient CRCs (29% of the MLH1-methylated, 17% of the Lynch syndrome and 9% of the suspected Lynch CRCs). No B2M mutations were identified in the 63 adenomas tested. B2M mutations differed by site, stage, grade and lymphocytic infiltration although none reached statistical significance (p > 0.05). The HR for overall survival for B2M mutated CRC was 0.65 (95% CI 0.29-1.48) compared with B2M wild-type. We observed differences in B2M mutation status in MMR-deficient CRC by tumour subtypes, site, stage, grade, immune infiltrate and for overall survival that warrant further investigation in larger studies before B2M mutation status can be considered to have clinical utility.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Alcohol Consumption and the Risk of Colorectal Cancer for Mismatch Repair Gene Mutation Carriers</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Dashti SG, Buchanan DD, Jayasekara H, Ait Ouakrim D, Clendenning M, Rosty C, Winship IM, Macrae FA, Giles GG, Parry S, Casey G, Haile RW, Gallinger S, Le Marchand L, Thibodeau SN, Lindor NM, Newcomb PA, Potter JD, Baron JA, Hopper JL, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">26</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-16-0496</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27811119</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-16-0496">http://dx.doi.org/10.1158/1055-9965.EPI-16-0496</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Background: People with germline mutation in one of the DNA mismatch repair (MMR) genes have increased colorectal cancer risk. For these high-risk people, study findings of the relationship between alcohol consumption and colorectal cancer risk have been inconclusive.Methods: 1,925 MMR gene mutations carriers recruited into the Colon Cancer Family Registry who had completed a questionnaire on lifestyle factors were included. Weighted Cox proportional hazard regression models were used to estimate hazard ratios (HR) and 95% confidence intervals (CI) for the association between alcohol consumption and colorectal cancer.Results: Colorectal cancer was diagnosed in 769 carriers (40%) at a mean (SD) age of 42.6 (10.3) years. Compared with abstention, ethanol consumption from any alcoholic beverage up to 14 g/day and >28 g/day was associated with increased colorectal cancer risk (HR, 1.50; 95% CI, 1.09-2.07 and 1.69; 95% CI, 1.07-2.65, respectively; Ptrend = 0.05), and colon cancer risk (HR, 1.78; 95% CI, 1.27-2.49 and 1.94; 95% CI, 1.19-3.18, respectively; Ptrend = 0.02). However, there was no clear evidence for an association with rectal cancer risk. Also, there was no evidence for associations between consumption of individual alcoholic beverage types (beer, wine, spirits) and colorectal, colon, or rectal cancer risk.Conclusions: Our data suggest that alcohol consumption, particularly more than 28 g/day of ethanol ( approximately 2 standard drinks of alcohol in the United States), is associated with increased colorectal cancer risk for MMR gene mutation carriers.Impact: Although these data suggested that alcohol consumption in MMR carriers was associated with increased colorectal cancer risk, there was no evidence of a dose-response, and not all types of alcohol consumption were associated with increased risk. Cancer Epidemiol Biomarkers Prev; 26(3); 366-75. (c)2016 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Targeted sequencing of 36 known or putative colorectal cancer susceptibility genes</a>, , 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">DeRycke MS, Gunawardena S, Balcom JR, Pickart AM, Waltman LA, French AJ, McDonnell S, Riska SM, Fogarty ZC, Larson MC, Middha S, Eckloff BW, Asmann YW, Ferber MJ, Haile RW, Gallinger S, Clendenning M, Rosty C, Win AK, Buchanan DD, Hopper JL, Newcomb PA, Le Marchand L, Goode EL, Lindor NM, Thibodeau SN</dd>
                  
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/mgg3.317</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28944238</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/mgg3.317">http://dx.doi.org/10.1002/mgg3.317</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal cancer Familial Colorectal Cancer Type X germline variants young onset</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Mutations in several genes predispose to colorectal cancer. Genetic testing for hereditary colorectal cancer syndromes was previously limited to single gene tests; thus, only a very limited number of genes were tested, and rarely those infrequently mutated in colorectal cancer. Next-generation sequencing technologies have made it possible to sequencing panels of genes known and suspected to influence colorectal cancer susceptibility. METHODS: Targeted sequencing of 36 known or putative CRC susceptibility genes was conducted for 1231 CRC cases from five subsets: (1) Familial Colorectal Cancer Type X (n = 153); (2) CRC unselected by tumor immunohistochemical or microsatellite stability testing (n = 548); (3) young onset (age /=50 years (n = 68); and (5) deficient MMR CRCs with no germline mutations in MLH1, MSH2, MSH6, or PMS2 (n = 129). Ninety-three unaffected controls were also sequenced. RESULTS: Overall, 29 nonsense, 43 frame-shift, 13 splice site, six initiator codon variants, one stop codon, 12 exonic deletions, 658 missense, and 17 indels were identified. Missense variants were reviewed by genetic counselors to determine pathogenicity; 13 were pathogenic, 61 were not pathogenic, and 584 were variants of uncertain significance. Overall, we identified 92 cases with pathogenic mutations in APC,MLH1,MSH2,MSH6, or multiple pathogenic MUTYH mutations (7.5%). Four cases with intact MMR protein expression by immunohistochemistry carried pathogenic MMR mutations. CONCLUSIONS: Results across case subsets may help prioritize genes for inclusion in clinical gene panel tests and underscore the issue of variants of uncertain significance both in well-characterized genes and those for which limited experience has accumulated.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Prediction of overall survival in stage II and III colon cancer beyond TNM system: a retrospective, pooled biomarker study</a>, Annals of oncology: official journal of the European Society for Medical Oncology / ESMO, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Dienstmann R, Mason MJ, Sinicrope FA, Phipps AI, Tejpar S, Nesbakken A, Danielsen SA, Sveen A, Buchanan DD, Clendenning M, Rosty C, Bot B, Alberts SR, Milburn Jessup J, Lothe RA, Delorenzi M, Newcomb PA, Sargent D, Guinney J</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Annals of oncology: official journal of the European Society for Medical Oncology / ESMO</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">28</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/annonc/mdx052</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28453697</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/annonc/mdx052">http://dx.doi.org/10.1093/annonc/mdx052</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">*BRAF mutation *KRAS mutation *colon cancer *microsatellite instability *prognosis</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Background: TNM staging alone does not accurately predict outcome in colon cancer (CC) patients who may be eligible for adjuvant chemotherapy. It is unknown to what extent the molecular markers microsatellite instability (MSI) and mutations in BRAF or KRAS improve prognostic estimation in multivariable models that include detailed clinicopathological annotation. Patients and methods: After imputation of missing at random data, a subset of patients accrued in phase 3 trials with adjuvant chemotherapy (n = 3016)-N0147 (NCT00079274) and PETACC3 (NCT00026273)-was aggregated to construct multivariable Cox models for 5-year overall survival that were subsequently validated internally in the remaining clinical trial samples (n = 1499), and also externally in different population cohorts of chemotherapy-treated (n = 949) or -untreated (n = 1080) CC patients, and an additional series without treatment annotation (n = 782). Results: TNM staging, MSI and BRAFV600E mutation status remained independent prognostic factors in multivariable models across clinical trials cohorts and observational studies. Concordance indices increased from 0.61-0.68 in the TNM alone model to 0.63-0.71 in models with added molecular markers, 0.65-0.73 with clinicopathological features and 0.66-0.74 with all covariates. In validation cohorts with complete annotation, the integrated time-dependent AUC rose from 0.64 for the TNM alone model to 0.67 for models that included clinicopathological features, with or without molecular markers. In patient cohorts that received adjuvant chemotherapy, the relative proportion of variance explained (R2) by TNM, clinicopathological features and molecular markers was on an average 65%, 25% and 10%, respectively. Conclusions: Incorporation of MSI, BRAFV600E and KRAS mutation status to overall survival models with TNM staging improves the ability to precisely prognosticate in stage II and III CC patients, but only modestly increases prediction accuracy in multivariable models that include clinicopathological features, particularly in chemotherapy-treated patients.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association of DNA Methylation-Based Biological Age with Health Risk Factors, and Overall and Cause-Specific Mortality</a>, American journal of epidemiology, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Dugue PA, Bassett JK, Joo JE, Baglietto L, Jung CH, Ming Wong E, Fiorito G, Schmidt D, Makalic E, Li S, Moreno-Betancur M, Buchanan DD, Vineis P, English DR, Hopper JL, Severi G, Southey MC, Giles GG, Milne RL</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">American journal of epidemiology</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/aje/kwx291</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">29020168</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/aje/kwx291">http://dx.doi.org/10.1093/aje/kwx291</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">DNA methylation age acceleration aging biological age cancer epigenetic clock health risk factors mortality</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Measures of biological age based on blood DNA methylation, referred to as age acceleration (AA), have been developed. We examined whether AA was associated with health risk factors and overall and cause-specific mortality. At baseline (1990-1994), blood samples were drawn from 2818 participants in the Melbourne Collaborative Cohort Study. DNA methylation was determined using the Illumina HM450K array. Mixed-effects models were used to examine the association of AA with health risk factors. Cox models were used to assess the association of AA with mortality. A total of 831 deaths were observed over a median 10.7 years of follow-up. Associations of AA were observed with male sex, Greek ethnicity, smoking, obesity, diabetes, lower education, and meat intake. AA measures were associated with increased mortality, and this was only partly accounted for by known determinants of health (attenuation of HRs by 20-40%). Weak evidence of heterogeneity in the association was observed by sex (p=0.06) and cause of death (p=0.07) but not by other factors. DNA-methylation-based AA measures are associated with several major health risk factors, but these do not fully explain the association between AA and mortality. Future research should investigate what genetic and environmental factors determine AA.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>RNF43 is mutated less frequently in Lynch Syndrome compared with sporadic microsatellite unstable colorectal cancers</a>, Familial cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Fennell LJ, Clendenning M, McKeone DM, Jamieson SH, Balachandran S, Borowsky J, Liu J, Kawamata F, Bond CE, Rosty C, Burge ME, Buchanan DD, Leggett BA, Whitehall VL</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-017-0003-0</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28573495</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-017-0003-0">http://dx.doi.org/10.1007/s10689-017-0003-0</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal cancer Hnpcc Lynch syndrome Msi Microsatellite instability Rnf43</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The WNT signaling pathway is commonly altered during colorectal cancer development. The E3 ubiquitin ligase, RNF43, negatively regulates the WNT signal through increased ubiquitination and subsequent degradation of the Frizzled receptor. RNF43 has recently been reported to harbor frequent truncating frameshift mutations in sporadic microsatellite unstable (MSI) colorectal cancers. This study assesses the relative frequency of RNF43 mutations in hereditary colorectal cancers arising in the setting of Lynch syndrome. The entire coding region of RNF43 was Sanger sequenced in 24 colorectal cancers from 23 patients who either (i) carried a germline mutation in one of the DNA mismatch repair genes (MLH1, MSH6, MSH2, PMS2), or (ii) showed immunohistochemical loss of expression of one or more of the DNA mismatch repair proteins, was BRAF wild type at V600E, were under 60 years of age at diagnosis, and demonstrated no promoter region methylation for MLH1 in tumor DNA. A validation cohort of 44 colorectal cancers from mismatch repair germline mutation carriers from the Australasian Colorectal Cancer Family Registry (ACCFR) were sequenced for the most common truncating mutation hotspots (X117 and X659). RNF43 mutations were found in 9 of 24 (37.5%) Lynch syndrome colorectal cancers. The majority of mutations were frameshift deletions in the G659 G7 repeat tract (29%); 2 cancers (2/24, 8%) from the one patient harbored frameshift mutations at codon R117 (C6 repeat tract) within exon 3. In the ACCFR validation cohort, RNF43 hotspot mutations were identified in 19/44 (43.2%) of samples, which was not significantly different to the initial series. The proportion of mutant RNF43 in Lynch syndrome related colorectal cancers is significantly lower than the previously reported mutation rate found in sporadic MSI colorectal cancers. These findings identify further genetic differences between sporadic and hereditary colorectal cancers. This may be because Lynch Syndrome cancers commonly arise in colorectal adenomas already bearing the APC mutation, whereas sporadic microsatellite unstable colorectal cancers arise from serrated polyps typically lacking APC mutation, decreasing the selection pressure on other WNT signaling related loci in Lynch syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Associations of alcohol intake, smoking, physical activity and obesity with survival following colorectal cancer diagnosis by stage, anatomic site and tumor molecular subtype</a>, International journal of cancer. Journal international du cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jayasekara H, English DR, Haydon A, Hodge AM, Lynch BM, Rosty C, Williamson EJ, Clendenning M, Southey MC, Jenkins MA, Room R, Hopper JL, Milne RL, Buchanan DD, Giles GG, MacInnis RJ</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.31049</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28921583</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.31049">http://dx.doi.org/10.1002/ijc.31049</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">alcohol intake colorectal cancer obesity physical activity smoking survival</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The influence of lifestyle factors on survival following a diagnosis of colorectal cancer (CRC) is not well established. We examined associations between lifestyle factors measured before diagnosis and CRC survival. The Melbourne Collaborative Cohort Study collected data on alcohol intake, cigarette smoking and physical activity, and body measurements at baseline (1990-1994) and wave 2 (2003-2007). We included participants diagnosed to 31 August 2015 with incident stages I-III CRC within 10-years post exposure assessment. Information on tumor characteristics and vital status was obtained. Tumor DNA was tested for microsatellite instability (MSI) and somatic mutations in oncogenes BRAF (V600E) and KRAS. We estimated hazard ratios (HRs) for associations between lifestyle factors and overall and CRC-specific mortality using Cox regression. Of 724 eligible CRC cases, 339 died (170 from CRC) during follow-up (average 9.0 years). Exercise (non-occupational/leisure-time) was associated with higher CRC-specific survival for stage II (HR = 0.25, 95% CI: 0.10-0.60) but not stages I/III disease (p for interaction = 0.01), and possibly for colon and KRAS wild-type tumors. Waist circumference was inversely associated with CRC-specific survival (HR = 1.25 per 10 cm increment, 95% CI: 1.08-1.44), independent of stage, anatomic site and tumor molecular status. Cigarette smoking was associated with lower overall survival, with suggestive evidence of worse survival for BRAF mutated CRC, but not with CRC-specific survival. Alcohol intake was not associated with survival. Survival did not differ by MSI status. We have identified pre-diagnostic predictors of survival following CRC that may have clinical and public health relevance.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lifetime alcohol intake is associated with an increased risk of KRAS+ and BRAF-/KRAS- but not BRAF+ colorectal cancer</a>, International journal of cancer. Journal international du cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jayasekara H, MacInnis RJ, Williamson EJ, Hodge AM, Clendenning M, Rosty C, Walters R, Room R, Southey MC, Jenkins MA, Milne RL, Hopper JL, Giles GG, Buchanan DD, English DR</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">140</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.30568</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27943267</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.30568">http://dx.doi.org/10.1002/ijc.30568</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenoma/genetics/metabolism Adult Aged Alcohol Drinking/*adverse effects Alcoholism/complications Anthropometry Carcinoma/genetics/metabolism Colorectal Neoplasms/*genetics/*metabolism Female *Genes, ras Humans Male Middle Aged Mutation Prospective Studies Proto-Oncogene Proteins B-raf/genetics/*metabolism Proto-Oncogene Proteins p21(ras)/genetics/*metabolism Risk Factors Surveys and Questionnaires Braf Kras alcohol intake colorectal cancer</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Ethanol in alcoholic beverages is a causative agent for colorectal cancer. Colorectal cancer is a biologically heterogeneous disease, and molecular subtypes defined by the presence of somatic mutations in BRAF and KRAS are known to exist. We examined associations between lifetime alcohol intake and molecular and anatomic subtypes of colorectal cancer. We calculated usual alcohol intake for 10-year periods from age 20 using recalled frequency and quantity of beverage-specific consumption for 38,149 participants aged 40-69 years from the Melbourne Collaborative Cohort Study. Cox regression was performed to derive hazard ratios (HRs) and 95% confidence intervals (CIs) for the association between lifetime alcohol intake and colorectal cancer risk. Heterogeneity in the HRs across subtypes of colorectal cancer was assessed. A positive dose-dependent association between lifetime alcohol intake and overall colorectal cancer risk (mean follow-up = 14.6 years; n = 596 colon and n = 326 rectal cancer) was observed (HR = 1.08, 95% CI: 1.04-1.12 per 10 g/day increment). The risk was greater for rectal than colon cancer (phomogeneity = 0.02). Alcohol intake was associated with increased risks of KRAS+ (HR = 1.07, 95% CI: 1.00-1.15) and BRAF-/KRAS- (HR = 1.05, 95% CI: 1.00-1.11) but not BRAF+ tumors (HR = 0.89, 95% CI: 0.78-1.01; phomogeneity = 0.01). Alcohol intake is associated with an increased risk of KRAS+ and BRAF-/KRAS- tumors originating via specific molecular pathways including the traditional adenoma-carcinoma pathway but not with BRAF+ tumors originating via the serrated pathway. Therefore, limiting alcohol intake from a young age might reduce colorectal cancer originating via the traditional adenoma-carcinoma pathway.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk factors for metachronous colorectal cancer or polyp: A systematic review and meta-analysis</a>, Journal of gastroenterology and hepatology, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jayasekara H, Reece JC, Buchanan DD, Ahnen DJ, Parry S, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of gastroenterology and hepatology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">32</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1111/jgh.13476</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27356122</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1111/jgh.13476">http://dx.doi.org/10.1111/jgh.13476</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">colorectal cancer metachronous risk factors systematic review</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND AND AIM: We conducted a systematic review and meta-analysis to identify personal, lifestyle, and tumor-related risk factors for metachronous colorectal cancer (CRC) and polyp. METHODS: Relevant studies were identified by searching MEDLINE, Web of Science and Cochrane Central Register through 15 May 2016. Estimates for associations were summarized using random effects models. RESULTS: Fifty-five studies were included in the review. For individuals who had a CRC resection, having a synchronous polyp was a risk factor for metachronous CRC or polyp (relative risk [RR], 2.04; 95% confidence interval [CI], 1.48-2.82) and having a synchronous CRC (RR, 1.90; 95% CI, 1.25-2.91) and proximally located CRC (RR, 2.12; 95% CI, 1.24-3.64) were risk factors for metachronous CRC. For individuals who had a polypectomy, larger size (RR, 4.26; 95% CI, 2.11-8.57) or severe dysplasia of the initial polyp (RR, 5.15; 95% CI, 2.02-13.14), and having a synchronous polyp (RR, 2.52; 95% CI, 1.35-4.73) were risk factors for metachronous CRC; and a family history of CRC (RR, 1.90; 95% CI, 1.26-2.87), having a synchronous polyp (RR, 2.47; 95% CI, 1.74-3.50) and a larger size (RR, 1.49; 95% CI, 1.03-2.15) and proximal location of the initial polyp (RR, 1.20; 95% CI, 1.02-1.40) were risk factors for metachronous polyp. Meta-regression showed duration of follow-up was not a source of heterogeneity for most associations. There was no evidence that lifestyle factors were associated with metachronous CRC or polyp risk. CONCLUSION: A comprehensive list of risk factors identified for metachronous CRC or polyp may have important clinical implications.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Family history of cancer predicts endometrial cancer risk independently of Lynch Syndrome: Implications for genetic counselling</a>, Gynecologic oncology, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Johnatty SE, Tan YY, Buchanan DD, Bowman M, Walters RJ, Obermair A, Quinn MA, Blomfield PB, Brand A, Leung Y, Oehler MK, Group,  Anecs, Kirk JA, O'Mara TA, Webb PM, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gynecologic oncology</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.ygyno.2017.08.011</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28822557</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.ygyno.2017.08.011">http://dx.doi.org/10.1016/j.ygyno.2017.08.011</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Endometrial cancer Family cancer history Lynch syndrome Mismatch repair Risk</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">OBJECTIVE: To determine endometrial cancer (EC) risk according to family cancer history, including assessment by degree of relatedness, type of and age at cancer diagnosis of relatives. METHODS: Self-reported family cancer history was available for 1353 EC patients and 628 controls. Logistic regression was used to quantify the association between EC and cancer diagnosis in >/=1 first or second degree relative, and to assess whether level of risk differed by degree of relationship and/or relative's age at diagnosis. Risk was also evaluated for family history of up to three cancers from known familial syndromes (Lynch, Cowden, hereditary breast and ovarian cancer) overall, by histological subtype and, for a subset of 678 patients, by EC tumor mismatch repair (MMR) gene expression. RESULTS: Report of EC in >/=1 first- or second-degree relative was associated with significantly increased risk of EC (P=3.8x10-7), independent of lifestyle risk factors. There was a trend in increasing EC risk with closer relatedness and younger age at EC diagnosis in relatives (PTrend=4.43x10-6), and with increasing numbers of Lynch cancers in relatives (PTrend/=1 relative. Associations remained significant irrespective of proband MMR status, and after excluding MMR pathogenic variant carriers, indicating that Lynch syndrome genes do not fully explain familial EC risk.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Germline miRNA DNA variants and the risk of colorectal cancer by subtype</a>, Genes, chromosomes & cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Lindor NM, Larson MC, DeRycke MS, McDonnell SK, Baheti S, Fogarty ZC, Win AK, Potter JD, Buchanan DD, Clendenning M, Newcomb PA, Casey G, Gallinger S, Le Marchand L, Hopper JL, Jenkins MA, Goode EL, Thibodeau SN</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Genes, chromosomes & cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">56</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/gcc.22420</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27636879</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/gcc.22420">http://dx.doi.org/10.1002/gcc.22420</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Biomarkers, Tumor/*genetics Case-Control Studies Colorectal Neoplasms/*genetics Follow-Up Studies Genetic Variation/*genetics Germ-Line Mutation/*genetics Humans MicroRNAs/*genetics Neoplasm Staging Prognosis Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">MicroRNAs (miRNAs) regulate up to one-third of all protein-coding genes including genes relevant to cancer. Variants within miRNAs have been reported to be associated with prognosis, survival, response to chemotherapy across cancer types, in vitro parameters of cell growth, and altered risks for development of cancer. Five miRNA variants have been reported to be associated with risk for development of colorectal cancer (CRC). In this study, we evaluated germline genetic variation in 1,123 miRNAs in 899 individuals with CRCs categorized by clinical subtypes and in 204 controls. The role of common miRNA variation in CRC was investigated using single variant and miRNA-level association tests. Twenty-nine miRNAs and 30 variants exhibited some marginal association with CRC in at least one subtype of CRC. Previously reported associations were not confirmed (n = 4) or could not be evaluated (n = 1). The variants noted for the CRCs with deficient mismatch repair showed little overlap with the variants noted for CRCs with proficient mismatch repair, consistent with our evolving understanding of the distinct biology underlying these two groups. (c) 2016 The Authors Genes, Chromosomes & Cancer Published by Wiley Periodicals, Inc.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Pro-inflammatory fatty acid profile and colorectal cancer risk: A Mendelian randomisation analysis</a>, European journal of cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">May-Wilson S, Sud A, Law PJ, Palin K, Tuupanen S, Gylfe A, Hanninen UA, Cajuso T, Tanskanen T, Kondelin J, Kaasinen E, Sarin AP, Eriksson JG, Rissanen H, Knekt P, Pukkala E, Jousilahti P, Salomaa V, Ripatti S, Palotie A, Renkonen-Sinisalo L, Lepisto A, Bohm J, Mecklin JP, Al-Tassan NA, Palles C, Farrington SM, Timofeeva MN, Meyer BF, Wakil SM, Campbell H, Smith CG, Idziaszczyk S, Maughan TS, Fisher D, Kerr R, Kerr D, Passarelli MN, Figueiredo JC, Buchanan DD, Win AK, Hopper JL, Jenkins MA, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Aaltonen LA, Cheadle JP, Tomlinson IP, Dunlop MG, Houlston RS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">European journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">84</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.ejca.2017.07.034</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28829991</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.ejca.2017.07.034">http://dx.doi.org/10.1016/j.ejca.2017.07.034</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Biomarkers, Tumor/*genetics Case-Control Studies Colorectal Neoplasms/blood/ethnology/*genetics/prevention & control Diet/*adverse effects Diet, Mediterranean European Continental Ancestry Group/genetics Fatty Acids/*adverse effects/blood Gene-Environment Interaction Genetic Predisposition to Disease Genome-Wide Association Study Healthy Diet Humans Inflammation Mediators/*adverse effects/blood Mendelian Randomization Analysis Odds Ratio Phenotype *Polymorphism, Single Nucleotide Protective Factors Risk Assessment Risk Factors Risk Reduction Behavior Colorectal cancer Fatty acids Mendelian randomisation Plasma fatty acids Risk</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: While dietary fat has been established as a risk factor for colorectal cancer (CRC), associations between fatty acids (FAs) and CRC have been inconsistent. Using Mendelian randomisation (MR), we sought to evaluate associations between polyunsaturated (PUFA), monounsaturated (MUFA) and saturated FAs (SFAs) and CRC risk. METHODS: We analysed genotype data on 9254 CRC cases and 18,386 controls of European ancestry. Externally weighted polygenic risk scores were generated and used to evaluate associations with CRC per one standard deviation increase in genetically defined plasma FA levels. RESULTS: Risk reduction was observed for oleic and palmitoleic MUFAs (OROA = 0.77, 95% CI: 0.65-0.92, P = 3.9 x 10-3; ORPOA = 0.36, 95% CI: 0.15-0.84, P = 0.018). PUFAs linoleic and arachidonic acid had negative and positive associations with CRC respectively (ORLA = 0.95, 95% CI: 0.93-0.98, P = 3.7 x 10-4; ORAA = 1.05, 95% CI: 1.02-1.07, P = 1.7 x 10-4). The SFA stearic acid was associated with increased CRC risk (ORSA = 1.17, 95% CI: 1.01-1.35, P = 0.041). CONCLUSION: Results from our analysis are broadly consistent with a pro-inflammatory FA profile having a detrimental effect in terms of CRC risk.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Reducing the polyp burden in serrated polyposis by serial colonoscopy: the impact of nationally coordinated community surveillance</a>, The New Zealand medical journal, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Parry S, Burt RW, Win AK, Aung YK, Woodall S, Arnold J, Clendenning M, Buchanan DD, Price TJ, Rosty C, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The New Zealand medical journal</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">130</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1451</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28253245</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/28253245">https://www.ncbi.nlm.nih.gov/pubmed/28253245</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Colonic Polyps/*diagnosis/pathology/*therapy Colonoscopy/*methods Colorectal Neoplasms/*epidemiology Female Humans Incidence Male Mass Screening/*methods Middle Aged New Zealand/epidemiology Risk Factors Time Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Serrated polyposis syndrome (SPS) is associated with an increased risk of colorectal cancer (CRC) and an evolving management approach. The aims of this study were to assess the polyp burden reduction over time, and the incidence of CRC in serrated polyposis patients undergoing community surveillance. METHODS: This is an observational study based on prospectively collected data. A total of 96 SPS patients with no personal history of CRC were prospectively enrolled in a surveillance program under the guidance of a tertiary center. Patients underwent surveillance colonoscopy in multiple centres across New Zealand. RESULTS: Patients underwent a median of four colonoscopies with a median interval of 15 months over a median follow-up period of 4.8 years. Five of 96 patients (5%) were referred for surgery, and the remaining 91 were managed by colonoscopy alone. In patients referred for surgery, 92% of the surveillance intervals to the fourth colonoscopy had been /=20 pancolonic polyps after four procedures compared with only 5/91 (5%) in those managed by colonoscopy alone. In patients successfully managed by colonoscopy, 86% had 75% no longer had polyps >/=10mm and >90% no longer had proximal serrated polyps >/=10mm after the fourth colonoscopy. No patients were found to develop CRC during the study time period. CONCLUSIONS: Patients with SPS were managed by proactive surveillance colonoscopy in wider hospital settings under tertiary centre guidance, with only 5% requiring surgical management. No CRC was diagnosed in any patient during surveillance.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Targeted Sequencing of Established and Candidate Colorectal Cancer Genes in the Colon Cancer Family Registry</a>, Oncotarget, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Raskin L, Guo Y, Du L, Clendenning M, Rosty C, Registry,  Colon Cancer Family, Lindor NM, Gruber SB, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Oncotarget</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                  
                  
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The underlying genetic cause of colorectal cancer (CRC) can be identified for 5-10% of all cases, while at least 20% of CRC cases are thought to be due to inherited genetic factors. Screening for highly penetrant mutations in genes associated with Mendelian cancer syndromes using next-generation sequencing (NGS) can be prohibitively expensive for studies requiring large samples sizes. The aim of the study was to identify rare single nucleotide variants and small indels in 40 established or candidate CRC susceptibility genes in 1,046 familial CRC cases (including both MSS and MSI-H tumor subtypes) and 1,006 unrelated controls from the Colon Cancer Family Registry Cohort using a robust and cost-effective DNA pooling NGS strategy. We identified 264 variants in 38 genes that were observed only in cases, comprising either very rare (minor allele frequency <0.001) or not previously reported (n=90, 34%) in reference databases, including six stop-gain, three frameshift, and 255 nonsynonymous variants predicted to be damaging. We found novel germline mutations in established CRC genes MLH1, APC, and POLE, and likely pathogenic variants in cancer susceptibility genes BAP1, CDH1, CHEK2, ENG, and MSH3. For the candidate CRC genes, we identified likely pathogenic variants in the helicase domain of POLQ and in the LRIG1, SH2B3, and NOS1 genes and present their clinicopathological characteristics. Using a DNA pooling NGS strategy, we identified novel germline mutations in established CRC susceptibility genes in familial CRC cases. Further studies are required to support the role of POLQ, LRIG1, SH2B3 and NOS1 as CRC susceptibility genes.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Mendelian randomisation implicates hyperlipidaemia as a risk factor for colorectal cancer</a>, International journal of cancer. Journal international du cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rodriguez-Broadbent H, Law PJ, Sud A, Palin K, Tuupanen S, Gylfe A, Hanninen UA, Cajuso T, Tanskanen T, Kondelin J, Kaasinen E, Sarin AP, Ripatti S, Eriksson JG, Rissanen H, Knekt P, Pukkala E, Jousilahti P, Salomaa V, Palotie A, Renkonen-Sinisalo L, Lepisto A, Bohm J, Mecklin JP, Al-Tassan NA, Palles C, Martin L, Barclay E, Farrington SM, Timofeeva MN, Meyer BF, Wakil SM, Campbell H, Smith CG, Idziaszczyk S, Maughan TS, Kaplan R, Kerr R, Kerr D, Passarelli MN, Figueiredo JC, Buchanan DD, Win AK, Hopper JL, Jenkins MA, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Aaltonen LA, Cheadle JP, Tomlinson IP, Dunlop MG, Houlston RS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">140</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.30709</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28340513</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.30709">http://dx.doi.org/10.1002/ijc.30709</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Mendelian randomisation cholesterol colorectal cancer hyperlipidaemia risk</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">While elevated blood cholesterol has been associated with an increased risk of colorectal cancer (CRC) in observational studies, causality is uncertain. Here we apply a Mendelian randomisation (MR) analysis to examine the potential causal relationship between lipid traits and CRC risk. We used single nucleotide polymorphisms (SNPs) associated with blood levels of total cholesterol (TC), triglyceride (TG), low-density lipoprotein (LDL), and high-density lipoprotein (HDL) as instrumental variables (IV). We calculated MR estimates for each risk factor with CRC using SNP-CRC associations from 9,254 cases and 18,386 controls. Genetically predicted higher TC was associated with an elevated risk of CRC (odds ratios (OR) per unit SD increase = 1.46, 95% confidence interval [CI]: 1.20-1.79, p = 1.68 x 10-4 ). The pooled ORs for LDL, HDL, and TG were 1.05 (95% CI: 0.92-1.18, p = 0.49), 0.94 (95% CI: 0.84-1.05, p = 0.27), and 0.98 (95% CI: 0.85-1.12, p = 0.75) respectively. A genetic risk score for 3-hydoxy-3-methylglutaryl-coenzyme A reductase (HMGCR) to mimic the effects of statin therapy was associated with a reduced CRC risk (OR = 0.69, 95% CI: 0.49-0.99, p = 0.046). This study supports a causal relationship between higher levels of TC with CRC risk, and a further rationale for implementing public health strategies to reduce the prevalence of hyperlipidaemia.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Genome-wide association study and meta-analysis in Northern European populations replicate multiple colorectal cancer risk loci</a>, International journal of cancer. Journal international du cancer, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Tanskanen T, van den Berg L, Valimaki N, Aavikko M, Ness-Jensen E, Hveem K, Wettergren Y, Bexe Lindskog E, Tonisson N, Metspalu A, Silander K, Orlando G, Law PJ, Tuupanen S, Gylfe AE, Hanninen UA, Cajuso T, Kondelin J, Sarin AP, Pukkala E, Jousilahti P, Salomaa V, Ripatti S, Palotie A, Jarvinen H, Renkonen-Sinisalo L, Lepisto A, Bohm J, Mecklin JP, Al-Tassan NA, Palles C, Martin L, Barclay E, Tenesa A, Farrington SM, Timofeeva MN, Meyer BF, Wakil SM, Campbell H, Smith CG, Idziaszczyk S, Maughan TS, Kaplan R, Kerr R, Kerr D, Buchanan DD, Win AK, Hopper J, Jenkins MA, Newcomb PA, Gallinger S, Conti D, Schumacher FR, Casey G, Cheadle JP, Dunlop MG, Tomlinson IP, Houlston RS, Palin K, Aaltonen LA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.31076</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">28960316</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.31076">http://dx.doi.org/10.1002/ijc.31076</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">colorectal cancer genetic predisposition to disease genome-wide association study single-nucleotide polymorphism</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Genome-wide association studies have been successful in elucidating the genetic basis of colorectal cancer (CRC), but there remains unexplained variability in genetic risk. To identify new risk variants and to confirm reported associations, we conducted a genome-wide association study in 1,701 CRC cases and 14,082 cancer-free controls from the Finnish population. A total of 9,068,015 genetic variants were imputed and tested, and 30 promising variants were studied in additional 11,647 cases and 12,356 controls of European ancestry. The previously reported association between the single-nucleotide polymorphism (SNP) rs992157 (2q35) and CRC was independently replicated (p = 2.08 x 10-4 ; OR, 1.14; 95% CI, 1.06-1.23), and it was genome-wide significant in combined analysis (p = 1.50 x 10-9 ; OR, 1.12; 95% CI, 1.08-1.16). Variants at 2q35, 6p21.2, 8q23.3, 8q24.21, 10q22.3, 10q24.2, 11q13.4, 11q23.1, 14q22.2, 15q13.3, 18q21.1, 20p12.3 and 20q13.33 were associated with CRC in the Finnish population (false discovery rate < 0.1), but new risk loci were not found. These results replicate the effects of multiple loci on the risk of CRC and identify shared risk alleles between the Finnish population isolate and outbred populations.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Prevalence and Penetrance of Major Genes and Polygenes for Colorectal Cancer</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2017.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Jenkins MA, Dowty JG, Antoniou AC, Lee A, Giles GG, Buchanan DD, Clendenning M, Rosty C, Ahnen DJ, Thibodeau SN, Casey G, Gallinger S, Le Marchand L, Haile RW, Potter JD, Zheng Y, Lindor NM, Newcomb PA, Hopper JL, MacInnis RJ</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">26</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2017</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-16-0693</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27799157</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-16-0693">http://dx.doi.org/10.1158/1055-9965.EPI-16-0693</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Background: Although high-risk mutations in identified major susceptibility genes (DNA mismatch repair genes and MUTYH) account for some familial aggregation of colorectal cancer, their population prevalence and the causes of the remaining familial aggregation are not known.Methods: We studied the families of 5,744 colorectal cancer cases (probands) recruited from population cancer registries in the United States, Canada, and Australia and screened probands for mutations in mismatch repair genes and MUTYH We conducted modified segregation analyses using the cancer history of first-degree relatives, conditional on the proband's age at diagnosis. We estimated the prevalence of mutations in the identified genes, the prevalence of HR for unidentified major gene mutations, and the variance of the residual polygenic component.Results: We estimated that 1 in 279 of the population carry mutations in mismatch repair genes (MLH1 = 1 in 1,946, MSH2 = 1 in 2,841, MSH6 = 1 in 758, PMS2 = 1 in 714), 1 in 45 carry mutations in MUTYH, and 1 in 504 carry mutations associated with an average 31-fold increased risk of colorectal cancer in unidentified major genes. The estimated polygenic variance was reduced by 30% to 50% after allowing for unidentified major genes and decreased from 3.3 for age /=70 years (equivalent to sibling relative risks of 5.1 to 1.3, respectively).Conclusions: Unidentified major genes might explain one third to one half of the missing heritability of colorectal cancer.Impact: Our findings could aid gene discovery and development of better colorectal cancer risk prediction models. Cancer Epidemiol Biomarkers Prev; 26(3); 404-12. (c)2016 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2016</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Multivitamin, calcium and folic acid supplements and the risk of colorectal cancer in Lynch syndrome</a>, International journal of epidemiology, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Chau R, Dashti SG, Ait Ouakrim D, Buchanan DD, Clendenning M, Rosty C, Winship IM, Young JP, Giles GG, Macrae FA, Boussioutas A, Parry S, Figueiredo JC, Levine AJ, Ahnen DJ, Casey G, Haile RW, Gallinger S, Le Marchand L, Thibodeau SN, Lindor NM, Newcomb PA, Potter JD, Baron JA, Hopper JL, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of epidemiology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">45</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/ije/dyw036</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27063605</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/ije/dyw036">http://dx.doi.org/10.1093/ije/dyw036</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal cancer DNA mismatch repair Lynch syndrome calcium folic acid multivitamin</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: People with a DNA mismatch repair (MMR) gene mutation have a substantially elevated risk of colorectal cancer (CRC) but the modifiers of this risk are not well established. We investigated the association between dietary supplement intake and CRC risk for carriers. METHODS: This study included 1966 (56% female) carriers of an MMR gene mutation (719 MLH1, 931 MSH2, 211 MSH6 and 105 PMS2) who were recruited from the USA, Canada, Australia and New Zealand into the Colon Cancer Family Registry between 1997 and 2012. Information on lifestyle factors including supplement intake was collected at the time of recruitment. Using Cox proportional hazards regression weighted to correct for ascertainment bias, we estimated hazard ratios (HRs) and 95% confidence intervals (CIs) for associations between self-reported multivitamin, calcium and folic acid supplement intake and CRC risk. RESULTS: Of 744 carriers with CRC, 18%, 6% and 5% reported intake of multivitamin, calcium and folic acid supplements for at least 1 month, respectively, compared with 27%, 11% and 10% of 1222 carriers without CRC. After adjusting for identified confounding variables, a decreased CRC risk was associated with multivitam inintake for at least 3 years (HR 0.47, 95% CI 0.32-0.69) and calcium intake for at least 3 years(HR 0.42, 95% CI 0.23-0.74), compared with never users. There was no evidence of an association between folic acid supplement intake and CRC risk (P = 0.82). CONCLUSION: Intake of multivitamin and calcium supplements might be associated with a decreased risk of CRC for MMR gene mutation carriers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Determining the familial risk distribution of colorectal cancer: a data mining approach</a>, Familial cancer, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Chau R, Jenkins MA, Buchanan DD, Ait Ouakrim D, Giles GG, Casey G, Gallinger S, Haile RW, Le Marchand L, Newcomb PA, Lindor NM, Hopper JL, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">15</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-015-9860-6</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26681340</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-015-9860-6">http://dx.doi.org/10.1007/s10689-015-9860-6</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal cancer Data mining Familial aggregation Familial risk</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">This study was aimed to characterize the distribution of colorectal cancer risk using family history of cancers by data mining. Family histories for 10,066 colorectal cancer cases recruited to population cancer registries of the Colon Cancer Family Registry were analyzed using a data mining framework. A novel index was developed to quantify familial cancer aggregation. Artificial neural network was used to identify distinct categories of familial risk. Standardized incidence ratios (SIRs) and corresponding 95% confidence intervals (CIs) of colorectal cancer were calculated for each category. We identified five major, and 66 minor categories of familial risk for developing colorectal cancer. The distribution the major risk categories were: (1) 7% of families (SIR = 7.11; 95% CI 6.65-7.59) had a strong family history of colorectal cancer; (2) 13% of families (SIR = 2.94; 95% CI 2.78-3.10) had a moderate family history of colorectal cancer; (3) 11% of families (SIR = 1.23; 95% CI 1.12-1.36) had a strong family history of breast cancer and a weak family history of colorectal cancer; (4) 9 % of families (SIR = 1.06; 95 % CI 0.96-1.18) had strong family history of prostate cancer and weak family history of colorectal cancer; and (5) 60% of families (SIR = 0.61; 95% CI 0.57-0.65) had a weak family history of all cancers. There is a wide variation of colorectal cancer risk that can be categorized by family history of cancer, with a strong gradient of colorectal cancer risk between the highest and lowest risk categories. The risk of colorectal cancer for people with the highest risk category of family history (7% of the population) was 12-times that for people in the lowest risk category (60%) of the population. Data mining was proven an effective approach for gaining insight into the underlying cancer aggregation patterns and for categorizing familial risk of colorectal cancer.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Cross-Cancer Genome-Wide Analysis of Lung, Ovary, Breast, Prostate, and Colorectal Cancer Reveals Novel Pleiotropic Associations</a>, Cancer research, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Fehringer G, Kraft P, Pharoah PD, Eeles RA, Chatterjee N, Schumacher FR, Schildkraut JM, Lindstrom S, Brennan P, Bickeboller H, Houlston RS, Landi MT, Caporaso N, Risch A, Amin Al Olama A, Berndt SI, Giovannucci EL, Gronberg H, Kote-Jarai Z, Ma J, Muir K, Stampfer MJ, Stevens VL, Wiklund F, Willett WC, Goode EL, Permuth JB, Risch HA, Reid BM, Bezieau S, Brenner H, Chan AT, Chang-Claude J, Hudson TJ, Kocarnik JK, Newcomb PA, Schoen RE, Slattery ML, White E, Adank MA, Ahsan H, Aittomaki K, Baglietto L, Blomquist C, Canzian F, Czene K, Dos-Santos-Silva I, Eliassen AH, Figueroa JD, Flesch-Janys D, Fletcher O, Garcia-Closas M, Gaudet MM, Johnson N, Hall P, Hazra A, Hein R, Hofman A, Hopper JL, Irwanto A, Johansson M, Kaaks R, Kibriya MG, Lichtner P, Liu J, Lund E, Makalic E, Meindl A, Muller-Myhsok B, Muranen TA, Nevanlinna H, Peeters PH, Peto J, Prentice RL, Rahman N, Sanchez MJ, Schmidt DF, Schmutzler RK, Southey MC, Tamimi R, Travis RC, Turnbull C, Uitterlinden AG, Wang Z, Whittemore AS, Yang XR, Zheng W, Buchanan DD, Casey G, Conti DV, Edlund CK, Gallinger S, Haile RW, Jenkins M, Le Marchand L, Li L, Lindor NM, Schmit SL, Thibodeau SN, Woods MO, Rafnar T, Gudmundsson J, Stacey SN, Stefansson K, Sulem P, Chen YA, Tyrer JP, Christiani DC, Wei Y, Shen H, Hu Z, Shu XO, Shiraishi K, Takahashi A, Bosse Y, Obeidat M, Nickle D, Timens W, Freedman ML, Li Q, Seminara D, Chanock SJ, Gong J, Peters U, Gruber SB, Amos CI, Sellers TA, Easton DF, Hunter DJ, Haiman CA, Henderson BE, Hung RJ, Ovarian Cancer Association,  Consortium, Consortium,  Practical, Hereditary B, Ovarian Cancer Research Group,  Netherlands, Colorectal Transdisciplinary S, African American Breast Cancer,  Consortium, African Ancestry Prostate Cancer,  Consortium</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer research</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">76</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">17</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/0008-5472.CAN-15-2980</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27197191</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-15-2980">http://dx.doi.org/10.1158/0008-5472.CAN-15-2980</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Identifying genetic variants with pleiotropic associations can uncover common pathways influencing multiple cancers. We took a two-stage approach to conduct genome-wide association studies for lung, ovary, breast, prostate, and colorectal cancer from the GAME-ON/GECCO Network (61,851 cases, 61,820 controls) to identify pleiotropic loci. Findings were replicated in independent association studies (55,789 cases, 330,490 controls). We identified a novel pleiotropic association at 1q22 involving breast and lung squamous cell carcinoma, with eQTL analysis showing an association with ADAM15/THBS3 gene expression in lung. We also identified a known breast cancer locus CASP8/ALS2CR12 associated with prostate cancer, a known cancer locus at CDKN2B-AS1 with different variants associated with lung adenocarcinoma and prostate cancer, and confirmed the associations of a breast BRCA2 locus with lung and serous ovarian cancer. This is the largest study to date examining pleiotropy across multiple cancer-associated loci, identifying common mechanisms of cancer development and progression. Cancer Res; 76(17); 5103-14. (c)2016 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Mendelian randomisation analysis strongly implicates adiposity with risk of developing colorectal cancer</a>, British journal of cancer, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jarvis D, Mitchell JS, Law PJ, Palin K, Tuupanen S, Gylfe A, Hanninen UA, Cajuso T, Tanskanen T, Kondelin J, Kaasinen E, Sarin AP, Kaprio J, Eriksson JG, Rissanen H, Knekt P, Pukkala E, Jousilahti P, Salomaa V, Ripatti S, Palotie A, Jarvinen H, Renkonen-Sinisalo L, Lepisto A, Bohm J, Meklin JP, Al-Tassan NA, Palles C, Martin L, Barclay E, Farrington SM, Timofeeva MN, Meyer BF, Wakil SM, Campbell H, Smith CG, Idziaszczyk S, Maughan TS, Kaplan R, Kerr R, Kerr D, Buchanan DD, Win AK, Hopper JL, Jenkins MA, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Taipale J, Aaltonen LA, Cheadle JP, Dunlop MG, Tomlinson IP, Houlston RS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">British journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">115</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/bjc.2016.188</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27336604</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/bjc.2016.188">http://dx.doi.org/10.1038/bjc.2016.188</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Observational studies have associated adiposity with an increased risk of colorectal cancer (CRC). However, such studies do not establish a causal relationship. To minimise bias from confounding we performed a Mendelian randomisation (MR) analysis to examine the relationship between adiposity and CRC. METHODS: We used SNPs associated with adult body mass index (BMI), waist-hip ratio (WHR), childhood obesity and birth weight as instrumental variables in a MR analysis of 9254 CRC cases and 18 386 controls. RESULTS: In the MR analysis, the odds ratios (ORs) of CRC risk per unit increase in BMI, WHR and childhood obesity were 1.23 (95% CI: 1.02-1.49, P=0.033), 1.59 (95% CI: 1.08-2.34, P=0.019) and 1.07 (95% CI: 1.03-1.13, P=0.018), respectively. There was no evidence for association between birth weight and CRC (OR=1.22, 95% CI: 0.89-1.67, P=0.22). Combining these data with a concurrent MR-based analysis for BMI and WHR with CRC risk (totalling to 18 190 cases, 27 617 controls) provided increased support, ORs for BMI and WHR were 1.26 (95% CI: 1.10-1.44, P=7.7 x 10(-4)) and 1.40 (95% CI: 1.14-1.72, P=1.2 x 10(-3)), respectively. CONCLUSIONS: These data provide further evidence for a strong causal relationship between adiposity and the risk of developing CRC highlighting the urgent need for prevention and treatment of adiposity.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk factors for metachronous colorectal cancer following a primary colorectal cancer: A prospective cohort study</a>, International journal of cancer. Journal international du cancer, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jayasekara H, Reece JC, Buchanan DD, Rosty C, Dashti SG, Ait Ouakrim D, Winship IM, Macrae FA, Boussioutas A, Giles GG, Ahnen DJ, Lowery J, Casey G, Haile RW, Gallinger S, Le Marchand L, Newcomb PA, Lindor NM, Hopper JL, Parry S, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">139</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.30153</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27098183</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.30153">http://dx.doi.org/10.1002/ijc.30153</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">colorectal cancer metachronous risk factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Individuals diagnosed with colorectal cancer (CRC) are at risk of developing a metachronous CRC. We examined the associations between personal, tumour-related and lifestyle risk factors, and risk of metachronous CRC. A total of 7,863 participants with incident colon or rectal cancer who were recruited in the USA, Canada and Australia to the Colon Cancer Family Registry during 1997-2012, except those identified as high-risk, for example, Lynch syndrome, were followed up approximately every 5 years. We estimated the risk of metachronous CRC, defined as the first new primary CRC following an interval of at least one year after the initial CRC diagnosis. Observation time started at the age at diagnosis of the initial CRC and ended at the age at diagnosis of the metachronous CRC, last contact or death whichever occurred earliest, or were censored at the age at diagnosis of any metachronous colorectal adenoma. Cox regression was used to derive hazard ratios (HRs) and 95% confidence intervals (CIs). During a mean follow-up of 6.6 years, 142 (1.81%) metachronous CRCs were diagnosed (mean age at diagnosis 59.8; incidence 2.7/1,000 person-years). An increased risk of metachronous CRC was associated with the presence of a synchronous CRC (HR = 2.73; 95% CI: 1.30-5.72) and the location of cancer in the proximal colon at initial diagnosis (compared with distal colon or rectum, HR = 4.16; 95% CI: 2.80-6.18). The presence of a synchronous CRC and the location of the initial CRC might be useful for deciding the intensity of surveillance colonoscopy for individuals diagnosed with CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Quantifying the utility of single nucleotide polymorphisms to guide colorectal cancer screening</a>, Future oncology , 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jenkins MA, Makalic E, Dowty JG, Schmidt DF, Dite GS, MacInnis RJ, Ait Ouakrim D, Clendenning M, Flander LB, Stanesby OK, Hopper JL, Win AK, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Future oncology </dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.2217/fon.15.303</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26846999</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.2217/fon.15.303">http://dx.doi.org/10.2217/fon.15.303</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">cancer screening colorectal cancer risk prediction single nucleotide polymorphisms</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">AIM: To determine whether single nucleotide polymorphisms (SNPs) can be used to identify people who should be screened for colorectal cancer. METHODS: We simulated one million people with and without colorectal cancer based on published SNP allele frequencies and strengths of colorectal cancer association. We estimated 5-year risks of colorectal cancer by number of risk alleles. RESULTS: We identified 45 SNPs with an average 1.14-fold increase colorectal cancer risk per allele (range: 1.05-1.53). The colorectal cancer risk for people in the highest quintile of risk alleles was 1.81-times that for the average person. CONCLUSION: We have quantified the extent to which known susceptibility SNPs can stratify the population into clinically useful colorectal cancer risk categories.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Comparison of Prediction Models for Lynch Syndrome Among Individuals With Colorectal Cancer</a>, Journal of the National Cancer Institute, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Kastrinos F, Ojha RP, Leenen C, Alvero C, Mercado RC, Balmana J, Valenzuela I, Balaguer F, Green R, Lindor NM, Thibodeau SN, Newcomb P, Win AK, Jenkins M, Buchanan DD, Bertario L, Sala P, Hampel H, Syngal S, Steyerberg EW, Lynch Syndrome prediction model validation study,  group</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">108</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/jnci/djv308</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26582061</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/jnci/djv308">http://dx.doi.org/10.1093/jnci/djv308</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics Adult Aged Area Under Curve Australia/epidemiology Colorectal Neoplasms/*complications Colorectal Neoplasms, Hereditary Nonpolyposis/diagnosis/epidemiology/*genetics DNA-Binding Proteins/*genetics Europe/epidemiology Female *Heterozygote Humans Male Middle Aged Models, Statistical MutL Protein Homolog 1 MutS Homolog 2 Protein/*genetics *Mutation North America/epidemiology Nuclear Proteins/*genetics Pedigree Predictive Value of Tests Prevalence ROC Curve</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Recent guidelines recommend the Lynch Syndrome prediction models MMRPredict, MMRPro, and PREMM1,2,6 for the identification of MMR gene mutation carriers. We compared the predictive performance and clinical usefulness of these prediction models to identify mutation carriers. METHODS: Pedigree data from CRC patients in 11 North American, European, and Australian cohorts (6 clinic- and 5 population-based sites) were used to calculate predicted probabilities of pathogenic MLH1, MSH2, or MSH6 gene mutations by each model and gene-specific predictions by MMRPro and PREMM1,2,6. We examined discrimination with area under the receiver operating characteristic curve (AUC), calibration with observed to expected (O/E) ratio, and clinical usefulness using decision curve analysis to select patients for further evaluation. All statistical tests were two-sided. RESULTS: Mutations were detected in 539 of 2304 (23%) individuals from the clinic-based cohorts (237 MLH1, 251 MSH2, 51 MSH6) and 150 of 3451 (4.4%) individuals from the population-based cohorts (47 MLH1, 71 MSH2, 32 MSH6). Discrimination was similar for clinic- and population-based cohorts: AUCs of 0.76 vs 0.77 for MMRPredict, 0.82 vs 0.85 for MMRPro, and 0.85 vs 0.88 for PREMM1,2,6. For clinic- and population-based cohorts, O/E deviated from 1 for MMRPredict (0.38 and 0.31, respectively) and MMRPro (0.62 and 0.36) but were more satisfactory for PREMM1,2,6 (1.0 and 0.70). MMRPro or PREMM1,2,6 predictions were clinically useful at thresholds of 5% or greater and in particular at greater than 15%. CONCLUSIONS: MMRPro and PREMM1,2,6 can well be used to select CRC patients from genetics clinics or population-based settings for tumor and/or germline testing at a 5% or higher risk. If no MMR deficiency is detected and risk exceeds 15%, we suggest considering additional genetic etiologies for the cause of cancer in the family.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>SNP rs16906252C>T Is an Expression and Methylation Quantitative Trait Locus Associated with an Increased Risk of Developing MGMT-Methylated Colorectal Cancer</a>, Clinical cancer research: an official journal of the American Association for Cancer Research, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Kuroiwa-Trzmielina J, Wang F, Rapkins RW, Ward RL, Buchanan DD, Win AK, Clendenning M, Rosty C, Southey MC, Winship IM, Hopper JL, Jenkins MA, Olivier J, Hawkins NJ, Hitchins MP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Clinical cancer research: an official journal of the American Association for Cancer Research</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">22</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">24</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1078-0432.CCR-15-2765</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27267851</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-15-2765">http://dx.doi.org/10.1158/1078-0432.CCR-15-2765</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">PURPOSE: Methylation of the MGMT promoter is the major cause of O6-methylguanine methyltransferase deficiency in cancer and has been associated with the T variant of the promoter enhancer SNP rs16906252C>T. We sought evidence for an association between the rs16906252C>T genotype and increased risk of developing a subtype of colorectal cancer featuring MGMT methylation, mediated by genotype-dependent epigenetic silencing within normal tissues. EXPERIMENTAL DESIGN: By applying a molecular pathologic epidemiology case-control study design, associations between rs16906252C>T and risk for colorectal cancer overall, and colorectal cancer stratified by MGMT methylation status, were estimated using multinomial logistic regression in two independent retrospective series of colorectal cancer cases and controls. The test sample comprised 1,054 colorectal cancer cases and 451 controls from Sydney, Australia. The validation sample comprised 612 colorectal cancer cases and 245 controls from the Australasian Colon Cancer Family Registry (ACCFR). To determine whether rs16906252C>T was linked to a constitutively altered epigenetic state, quantitative allelic expression and methylation analyses were performed in normal tissues. RESULTS: An association between rs16906252C>T and increased risk of developing MGMT-methylated colorectal cancer in the Sydney sample was observed [OR, 3.3; 95% confidence interval (CI), 2.0-5.3; P T represents an expression and methylation quantitative trait locus. CONCLUSIONS: We provide evidence that rs16906252C>T is associated with elevated risk for MGMT-methylated colorectal cancer, likely mediated by constitutive epigenetic repression of the T allele. Clin Cancer Res; 22(24); 6266-77. (c)2016 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Clinicopathologic Risk Factor Distributions for MLH1 Promoter Region Methylation in CIMP-Positive Tumors</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Levine AJ, Phipps AI, Baron JA, Buchanan DD, Ahnen DJ, Cohen SA, Lindor NM, Newcomb PA, Rosty C, Haile RW, Laird PW, Weisenberger DJ</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">25</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-15-0935</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26512054</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-15-0935">http://dx.doi.org/10.1158/1055-9965.EPI-15-0935</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics Aged Biomarkers, Tumor Case-Control Studies Colorectal Neoplasms/epidemiology/*genetics/*pathology CpG Islands/*genetics *DNA Methylation Female Follow-Up Studies Gene Expression Regulation, Neoplastic Humans Male *Microsatellite Instability Middle Aged Mutation/genetics Neoplasm Staging Nuclear Proteins/*genetics Prognosis Promoter Regions, Genetic/*genetics Risk Factors Survival Rate</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: The CpG island methylator phenotype (CIMP) is a major molecular pathway in colorectal cancer. Approximately 25% to 60% of CIMP tumors are microsatellite unstable (MSI-H) due to DNA hypermethylation of the MLH1 gene promoter. Our aim was to determine if the distributions of clinicopathologic factors in CIMP-positive tumors with MLH1 DNA methylation differed from those in CIMP-positive tumors without DNA methylation of MLH1. METHODS: We assessed the associations between age, sex, tumor-site, MSI status BRAF and KRAS mutations, and family colorectal cancer history with MLH1 methylation status in a large population-based sample of CIMP-positive colorectal cancers defined by a 5-marker panel using unconditional logistic regression to assess the odds of MLH1 methylation by study variables. RESULTS: Subjects with CIMP-positive tumors without MLH1 methylation were significantly younger, more likely to be male, and more likely to have distal colon or rectal primaries and the MSI-L phenotype. CIMP-positive MLH1-unmethylated tumors were significantly less likely than CIMP-positive MLH1-methylated tumors to harbor a BRAF V600E mutation and significantly more likely to harbor a KRAS mutation. MLH1 methylation was associated with significantly better overall survival (HR, 0.50; 95% confidence interval, 0.31-0.82). CONCLUSIONS: These data suggest that MLH1 methylation in CIMP-positive tumors is not a completely random event and implies that there are environmental or genetic determinants that modify the probability that MLH1 will become methylated during CIMP pathogenesis. IMPACT: MLH1 DNA methylation status should be taken into account in etiologic studies.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Variation at 2q35 (PNKD and TMBIM1) influences colorectal cancer risk and identifies a pleiotropic effect with inflammatory bowel disease</a>, Human molecular genetics, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Orlando G, Law PJ, Palin K, Tuupanen S, Gylfe A, Hanninen UA, Cajuso T, Tanskanen T, Kondelin J, Kaasinen E, Sarin AP, Kaprio J, Eriksson JG, Rissanen H, Knekt P, Pukkala E, Jousilahti P, Salomaa V, Ripatti S, Palotie A, Jarvinen H, Renkonen-Sinisalo L, Lepisto A, Bohm J, Mecklin JP, Al-Tassan NA, Palles C, Martin L, Barclay E, Tenesa A, Farrington S, Timofeeva MN, Meyer BF, Wakil SM, Campbell H, Smith CG, Idziaszczyk S, Maughan TS, Kaplan R, Kerr R, Kerr D, Buchanan DD, Ko Win A, Hopper J, Jenkins M, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Taipale J, Cheadle JP, Dunlop MG, Tomlinson IP, Aaltonen LA, Houlston RS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human molecular genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">25</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/hmg/ddw087</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27005424</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/hmg/ddw087">http://dx.doi.org/10.1093/hmg/ddw087</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">To identify new risk loci for colorectal cancer (CRC), we conducted a meta-analysis of seven genome-wide association studies (GWAS) with independent replication, totalling 13 656 CRC cases and 21 667 controls of European ancestry. The combined analysis identified a new risk association for CRC at 2q35 marked by rs992157 (P = 3.15 x 10-8, odds ratio = 1.10, 95% confidence interval = 1.06-1.13), which is intronic to PNKD (paroxysmal non-kinesigenic dyskinesia) and TMBIM1 (transmembrane BAX inhibitor motif containing 1). Intriguingly this susceptibility single-nucleotide polymorphism (SNP) is in strong linkage disequilibrium (r2 = 0.90, D' = 0.96) with the previously discovered GWAS SNP rs2382817 for inflammatory bowel disease (IBD). Following on from this observation we examined for pleiotropy, or shared genetic susceptibility, between CRC and the 200 established IBD risk loci, identifying an additional 11 significant associations (false discovery rate [FDR]) < 0.05). Our findings provide further insight into the biological basis of inherited genetic susceptibility to CRC, and identify risk factors that may influence the development of both CRC and IBD.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Germline mutations in PMS2 and MLH1 in individuals with solitary loss of PMS2 expression in colorectal carcinomas from the Colon Cancer Family Registry Cohort</a>, BMJ open, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Clendenning M, Walsh MD, Eriksen SV, Southey MC, Winship IM, Macrae FA, Boussioutas A, Poplawski NK, Parry S, Arnold J, Young JP, Casey G, Haile RW, Gallinger S, Le Marchand L, Newcomb PA, Potter JD, DeRycke M, Lindor NM, Thibodeau SN, Baron JA, Win AK, Hopper JL, Jenkins MA, Buchanan DD, Colon Cancer Family Registry,  Cohort</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">BMJ open</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1136/bmjopen-2015-010293</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26895986</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1136/bmjopen-2015-010293">http://dx.doi.org/10.1136/bmjopen-2015-010293</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Histopathology</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">OBJECTIVES: Immunohistochemistry for DNA mismatch repair proteins is used to screen for Lynch syndrome in individuals with colorectal carcinoma (CRC). Although solitary loss of PMS2 expression is indicative of carrying a germline mutation in PMS2, previous studies reported MLH1 mutation in some cases. We determined the prevalence of MLH1 germline mutations in a large cohort of individuals with a CRC demonstrating solitary loss of PMS2 expression. DESIGN: This cohort study included 88 individuals affected with a PMS2-deficient CRC from the Colon Cancer Family Registry Cohort. Germline PMS2 mutation analysis (long-range PCR and multiplex ligation-dependent probe amplification) was followed by MLH1 mutation testing (Sanger sequencing and multiplex ligation-dependent probe amplification). RESULTS: Of the 66 individuals with complete mutation screening, we identified a pathogenic PMS2 mutation in 49 (74%), a pathogenic MLH1 mutation in 8 (12%) and a MLH1 variant of uncertain clinical significance predicted to be damaging by in silico analysis in 3 (4%); 6 (9%) carried variants likely to have no clinical significance. Missense point mutations accounted for most alterations (83%; 9/11) in MLH1. The MLH1 c.113A> G p.Asn38Ser mutation was found in 2 related individuals. One individual who carried the MLH1 intronic mutation c.677+3A>G p.Gln197Argfs*8 leading to the skipping of exon 8, developed 2 tumours, both of which retained MLH1 expression. CONCLUSIONS: A substantial proportion of CRCs with solitary loss of PMS2 expression are associated with a deleterious MLH1 germline mutation supporting the screening for MLH1 in individuals with tumours of this immunophenotype, when no PMS2 mutation has been identified.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>GWASeq: targeted re-sequencing follow up to GWAS</a>, BMC genomics, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Salomon MP, Li WL, Edlund CK, Morrison J, Fortini BK, Win AK, Conti DV, Thomas DC, Duggan D, Buchanan DD, Jenkins MA, Hopper JL, Gallinger S, Le Marchand L, Newcomb PA, Casey G, Marjoram P</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">BMC genomics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">17</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1186/s12864-016-2459-y</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26940994</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s12864-016-2459-y">http://dx.doi.org/10.1186/s12864-016-2459-y</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10"> our data consisted of 1993 cases</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: For the last decade the conceptual framework of the Genome-Wide Association Study (GWAS) has dominated the investigation of human disease and other complex traits. While GWAS have been successful in identifying a large number of variants associated with various phenotypes, the overall amount of heritability explained by these variants remains small. This raises the question of how best to follow up on a GWAS, localize causal variants accounting for GWAS hits, and as a consequence explain more of the so-called \missing\" heritability. Advances in high throughput sequencing technologies now allow for the efficient and cost-effective collection of vast amounts of fine-scale genomic data to complement GWAS. RESULTS: We investigate these issues using a colon cancer dataset. After QC</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Promoter methylation of ITF2, but not APC, is associated with microsatellite instability in two populations of colorectal cancer patients</a>, BMC cancer, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Savio AJ, Daftary D, Dicks E, Buchanan DD, Parfrey PS, Young JP, Weisenberger D, Green RC, Gallinger S, McLaughlin JR, Knight JA, Bapat B</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">BMC cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">16</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1186/s12885-016-2149-9</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26884349</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1186/s12885-016-2149-9">http://dx.doi.org/10.1186/s12885-016-2149-9</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenomatous Polyposis Coli Protein/*genetics Basic Helix-Loop-Helix Leucine Zipper Transcription Factors/*genetics Cohort Studies Colon/chemistry Colorectal Neoplasms/chemistry/*genetics DNA Methylation/*genetics Female Humans Male *Microsatellite Instability Middle Aged Promoter Regions, Genetic Transcription Factors/*genetics Wnt Signaling Pathway</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Aberrant Wnt signaling activation occurs commonly in colorectal carcinogenesis, leading to upregulation of many target genes. APC (adenomatous polyposis coli) is an important component of the beta-catenin destruction complex, which regulates Wnt signaling, and is often mutated in colorectal cancer (CRC). In addition to mutational events, epigenetic changes arise frequently in CRC, specifically, promoter hypermethylation which silences tumor suppressor genes. APC and the Wnt signaling target gene ITF2 (immunoglobulin transcription factor 2) incur hypermethylation in various cancers, however, methylation-dependent regulation of these genes in CRC has not been studied in large, well-characterized patient cohorts. The microsatellite instability (MSI) subtype of CRC, featuring DNA mismatch repair deficiency and often promoter hypermethylation of MutL homolog 1 (MLH1), has a favorable outcome and is characterized by different chemotherapeutic responses than microsatellite stable (MSS) tumors. Other epigenetic events distinguishing these subtypes have not yet been fully elucidated. METHODS: Here, we quantify promoter methylation of ITF2 and APC by MethyLight in two case-case studies nested in population-based CRC cohorts from the Ontario Familial Colorectal Cancer Registry (n = 330) and the Newfoundland Familial Colorectal Cancer Registry (n = 102) comparing MSI status groups. RESULTS: ITF2 and APC methylation are significantly associated with tumor versus normal state (both P < 1.0 x 10(-6)). ITF2 is methylated in 45.8% of MSI cases and 26.9% of MSS cases and is significantly associated with MSI in Ontario (P = 0.002) and Newfoundland (P = 0.005) as well as the MSI-associated feature of MLH1 promoter hypermethylation (P = 6.72 x 10(-4)). APC methylation, although tumor-specific, does not show a significant association with tumor subtype, age, gender, or stage, indicating it is a general tumor-specific CRC biomarker. CONCLUSIONS: This study demonstrates, for the first time, MSI-associated ITF2 methylation, and further reveals the subtype-specific epigenetic events modulating Wnt signaling in CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Methylation of Breast Cancer Predisposition Genes in Early-Onset Breast Cancer: Australian Breast Cancer Family Registry</a>, PloS one, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Scott CM, Joo JE, O'Callaghan N, Buchanan DD, Clendenning M, Giles GG, Hopper JL, Wong EM, Southey MC</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/journal.pone.0165436</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27902704</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pone.0165436">http://dx.doi.org/10.1371/journal.pone.0165436</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">DNA methylation can mimic the effects of both germline and somatic mutations for cancer predisposition genes such as BRCA1 and p16INK4a. Constitutional DNA methylation of the BRCA1 promoter has been well described and is associated with an increased risk of early-onset breast cancers that have BRCA1-mutation associated histological features. The role of methylation in the context of other breast cancer predisposition genes has been less well studied and often with conflicting or ambiguous outcomes. We examined the role of methylation in known breast cancer susceptibility genes in breast cancer predisposition and tumor development. We applied the Infinium HumanMethylation450 Beadchip (HM450K) array to blood and tumor-derived DNA from 43 women diagnosed with breast cancer before the age of 40 years and measured the methylation profiles across promoter regions of BRCA1, BRCA2, ATM, PALB2, CDH1, TP53, FANCM, CHEK2, MLH1, MSH2, MSH6 and PMS2. Prior genetic testing had demonstrated that these women did not carry a germline mutation in BRCA1, ATM, CHEK2, PALB2, TP53, BRCA2, CDH1 or FANCM. In addition to the BRCA1 promoter region, this work identified regions with variable methylation at multiple breast cancer susceptibility genes including PALB2 and MLH1. Methylation at the region of MLH1 in these breast cancers was not associated with microsatellite instability. This work informs future studies of the role of methylation in breast cancer susceptibility gene silencing.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Cholecystectomy and the risk of colorectal cancer by tumor mismatch repair deficiency status</a>, International journal of colorectal disease, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Shang J, Reece JC, Buchanan DD, Giles GG, Figueiredo JC, Casey G, Gallinger S, Thibodeau SN, Lindor NM, Newcomb PA, Potter JD, Baron JA, Hopper JL, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of colorectal disease</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">31</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s00384-016-2615-5</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27286977</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s00384-016-2615-5">http://dx.doi.org/10.1007/s00384-016-2615-5</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Cholecystectomy Colorectal cancer Gallbladder Mismatch repair</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">PURPOSE: Gallbladder diseases and cholecystectomy may play a role in the development of colorectal cancer (CRC). Our aim was to investigate the association between cholecystectomy and CRC risk overall and by sex, family history, anatomical location, and tumor mismatch repair (MMR) status. METHODS: This study comprised 5847 incident CRC cases recruited from population cancer registries in Australia, Canada, and the USA into the Colon Cancer Family Registry between 1997 and 2012 and 4970 controls with no personal history of CRC who were either randomly selected from the general population or were spouses of the cases. The association between cholecystectomy and CRC was estimated using logistic regression, after adjusting for confounding factors. RESULTS: Overall, there was no evidence for an association between cholecystectomy and CRC (odds ratio [OR] = 0.88, 95 % confidence interval 0.73, 1.08). In the stratified analyses, there was no evidence for a difference in the association between women and men (P = 0.54), between individuals with and without family history of CRC in first-degree relative (P = 0.64), between tumor anatomical locations (P = 0.45), or between MMR-proficient and MMR-deficient cases (P = 0.54). CONCLUSION: Cholecystectomy is not a substantial risk factor for CRC, regardless of sex, family history, anatomical location, or tumor MMR status.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk of extracolonic cancers for people with biallelic and monoallelic mutations in MUTYH</a>, International journal of cancer. Journal international du cancer, 2016.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Reece JC, Dowty JG, Buchanan DD, Clendenning M, Rosty C, Southey MC, Young JP, Cleary SP, Kim H, Cotterchio M, Macrae FA, Tucker KM, Baron JA, Burnett T, Le Marchand L, Casey G, Haile RW, Newcomb PA, Thibodeau SN, Hopper JL, Gallinger S, Winship IM, Lindor NM, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">139</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2016</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.30197</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">27194394</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.30197">http://dx.doi.org/10.1002/ijc.30197</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Mutyh MUTYH-associated polyposis cancer risk penetrance</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Germline mutations in the DNA base excision repair gene MUTYH are known to increase a carrier's risk of colorectal cancer. However, the risks of other (extracolonic) cancers for MUTYH mutation carriers are not well defined. We identified 266 probands (91% Caucasians) with a MUTYH mutation (41 biallelic and 225 monoallelic) from the Colon Cancer Family Registry. Mutation status, sex, age and histories of cancer from their 1,903 first- and 3,255 second-degree relatives were analyzed using modified segregation analysis conditioned on the ascertainment criteria. Compared with incidences for the general population, hazard ratios (HRs) (95% confidence intervals [CIs]) for biallelic MUTYH mutation carriers were: urinary bladder cancer 19 (3.7-97) and ovarian cancer 17 (2.4-115). The HRs (95% CI) for monoallelic MUTYH mutation carriers were: gastric cancer 9.3 (6.7-13); hepatobiliary cancer 4.5 (2.7-7.5); endometrial cancer 2.1 (1.1-3.9) and breast cancer 1.4 (1.0-2.0). There was no evidence for an increased risk of cancers at the other sites examined (brain, pancreas, kidney or prostate). Based on the USA population incidences, the estimated cumulative risks (95% CI) to age 70 years for biallelic mutation carriers were: bladder cancer 25% (5-77%) for males and 8% (2-33%) for females and ovarian cancer 14% (2-65%). The cumulative risks (95% CI) for monoallelic mutation carriers were: gastric cancer 5% (4-7%) for males and 2.3% (1.7-3.3%) for females; hepatobiliary cancer 3% (2-5%) for males and 1.4% (0.8-2.3%) for females; endometrial cancer 3% (2%-6%) and breast cancer 11% (8-16%). These unbiased estimates of both relative and absolute risks of extracolonic cancers for people, mostly Caucasians, with MUTYH mutations will be important for their clinical management.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2015</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Aspirin, Ibuprofen, and the Risk of Colorectal Cancer in Lynch Syndrome</a>, Journal of the National Cancer Institute, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Ait Ouakrim D, Dashti SG, Chau R, Buchanan DD, Clendenning M, Rosty C, Winship IM, Young JP, Giles GG, Leggett B, Macrae FA, Ahnen DJ, Casey G, Gallinger S, Haile RW, Le Marchand L, Thibodeau SN, Lindor NM, Newcomb PA, Potter JD, Baron JA, Hopper JL, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">107</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">9</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/jnci/djv170</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26109217</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/jnci/djv170">http://dx.doi.org/10.1093/jnci/djv170</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/genetics Adenosine Triphosphatases/genetics Adult Aged Anti-Inflammatory Agents, Non-Steroidal/*administration & dosage Anticarcinogenic Agents/*administration & dosage Aspirin/*administration & dosage Bias (Epidemiology) Colorectal Neoplasms/*epidemiology/genetics/*prevention & control Colorectal Neoplasms, Hereditary Nonpolyposis/*genetics Confounding Factors (Epidemiology) DNA Mismatch Repair/*genetics DNA Repair Enzymes/genetics DNA-Binding Proteins/genetics Female *Germ-Line Mutation *Heterozygote Humans Ibuprofen/*administration & dosage Male Middle Aged MutS Homolog 2 Protein/genetics Nuclear Proteins/genetics Prevalence Proportional Hazards Models Registries United States/epidemiology</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Inheritance of a germline mutation in one of the DNA mismatch repair (MMR) genes MLH1, MSH2, MSH6, and PMS2 causes a high risk of colorectal and other cancers (Lynch Syndrome). Use of aspirin has been shown to be associated with a reduced risk of colorectal cancer for the general population as well as for MMR gene mutation carriers. The aim of this study was to determine whether use of aspirin and ibuprofen in a nontrial setting is associated with the risk of colorectal cancer risk for MMR gene mutation carriers. METHODS: We included 1858 participants in the Colon Cancer Family Registry who had been found to have a pathogenic germline mutation in a MMR gene (carriers). We used weighted Cox proportional hazards regression to estimate hazard ratios (HRs) and 95% confidence intervals (CIs). All statistical tests were two-sided. RESULTS: A total of 714 carriers (38%) were diagnosed with colorectal cancer at a mean age of 42.4 (standard deviation 10.6) years. A reduced risk of colorectal cancer was associated with aspirin use (for 1 month to 4.9 years: HR = 0.49, 95% CI = 0.27 to 0.90, P = .02; for >/=5 years: HR = 0.25, 95% CI = 0.10 to 0.62, P = .003) and ibuprofen use (for 1 month to 4.9 years: HR = 0.38, 95% CI = 0.18 to 0.79, P = .009; for >/=5 years: HR = 0.26, 95% CI = 0.10 to 0.69, P = .007), compared with less than one month of use. CONCLUSION: Our results provide additional evidence that, for MMR gene mutation carriers, use of aspirin and ibuprofen might be effective in reducing their high risk of colorectal cancer.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Erratum: A new GWAS and meta-analysis with 1000Genomes imputation identifies novel risk variants for colorectal cancer</a>, Scientific reports, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Al-Tassan NA, Whiffin N, Hosking FJ, Palles C, Farrington SM, Dobbins SE, Harris R, Gorman M, Tenesa A, Meyer BF, Wakil SM, Kinnersley B, Campbell H, Martin L, Smith CG, Idziaszczyk S, Barclay E, Maughan TS, Kaplan R, Kerr R, Kerr D, Buchanan DD, Win AK, Hopper J, Jenkins M, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Dunlop MG, Tomlinson IP, Cheadle JP, Houlston RS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Scientific reports</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/srep12372</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26237130</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/srep12372">http://dx.doi.org/10.1038/srep12372</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Al-Tassan, Nada A Whiffin, Nicola Hosking, Fay J Palles, Claire Farrington, Susan M Dobbins, Sara E Harris, Rebecca Gorman, Maggie Tenesa, Albert Meyer, Brian F Wakil, Salma M Kinnersley, Ben Campbell, Harry Martin, Lynn Smith, Christopher G Idziaszczyk, Shelley Barclay, Ella Maughan, Timothy S Kaplan, Richard Kerr, Rachel Kerr, David Buchanan, Daniel D Win, Aung Ko Hopper, John Jenkins, Mark Lindor, Noralane M Newcomb, Polly A Gallinger, Steve Conti, David Schumacher, Fred Casey, Graham Dunlop, Malcolm G Tomlinson, Ian P Cheadle, Jeremy P Houlston, Richard S eng Published Erratum England 2015/08/04 06:00 Sci Rep. 2015 Aug 3;5:12372. doi: 10.1038/srep12372.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>A new GWAS and meta-analysis with 1000Genomes imputation identifies novel risk variants for colorectal cancer</a>, Scientific reports, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Al-Tassan NA, Whiffin N, Hosking FJ, Palles C, Farrington SM, Dobbins SE, Harris R, Gorman M, Tenesa A, Meyer BF, Wakil SM, Kinnersley B, Campbell H, Martin L, Smith CG, Idziaszczyk S, Barclay E, Maughan TS, Kaplan R, Kerr R, Kerr D, Buchanan DD, Ko Win A, Hopper J, Jenkins M, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Dunlop MG, Tomlinson IP, Cheadle JP, Houlston RS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Scientific reports</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/srep10442</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25990418</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/srep10442">http://dx.doi.org/10.1038/srep10442</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Genome-wide association studies (GWAS) of colorectal cancer (CRC) have identified 23 susceptibility loci thus far. Analyses of previously conducted GWAS indicate additional risk loci are yet to be discovered. To identify novel CRC susceptibility loci, we conducted a new GWAS and performed a meta-analysis with five published GWAS (totalling 7,577 cases and 9,979 controls of European ancestry), imputing genotypes utilising the 1000 Genomes Project. The combined analysis identified new, significant associations with CRC at 1p36.2 marked by rs72647484 (minor allele frequency [MAF] = 0.09) near CDC42 and WNT4 (P = 1.21 x 10(-8), odds ratio [OR] = 1.21 ) and at 16q24.1 marked by rs16941835 (MAF = 0.21, P = 5.06 x 10(-8); OR = 1.15) within the long non-coding RNA (lncRNA) RP11-58A18.1 and ~500 kb from the nearest coding gene FOXL1. Additionally we identified a promising association at 10p13 with rs10904849 intronic to CUBN (MAF = 0.32, P = 7.01 x 10(-8); OR = 1.14). These findings provide further insights into the genetic and biological basis of inherited genetic susceptibility to CRC. Additionally, our analysis further demonstrates that imputation can be used to exploit GWAS data to identify novel disease-causing variants.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lynch syndrome and cervical cancer</a>, International journal of cancer. Journal international du cancer, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Antill YC, Dowty JG, Win AK, Thompson T, Walsh MD, Cummings MC, Gallinger S, Lindor NM, Le Marchand L, Hopper JL, Newcomb PA, Haile RW, Church J, Tucker KM, Buchanan DD, Young JP, Winship IM, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">137</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.29641</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26077226</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.29641">http://dx.doi.org/10.1002/ijc.29641</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Aged Australia Canada Colorectal Neoplasms, Hereditary Nonpolyposis/*genetics DNA Mismatch Repair/genetics Female Humans Incidence Middle Aged Mutation/genetics New Zealand Registries Risk Risk Factors United States Uterine Cervical Neoplasms/*genetics Young Adult Lynch syndrome cervical cancer mismatch repair</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Carriers of germline mutations in DNA mismatch repair (MMR) genes are at increased risk of several cancers including colorectal and gynecologic cancers (Lynch syndrome). There is no substantial evidence that these mutations are associated with an increased risk of cervical cancer. A total of 369 families with at least one carrier of a mutation in a MMR gene (133 MLH1, 174 MSH2, 35 MSH6 and 27 PMS2) were ascertained via population cancer registries or via family cancer clinics in Australia, New Zealand, Canada, and USA. Personal and family histories of cancer were obtained from participant interviews. Modified segregation analysis was used to estimate the hazard ratio (incidence rates for carriers relative to those for the general population), and age-specific cumulative risks of cervical cancer for carriers. A total of 65 cases of cervical cancer were reported (including 10 verified by pathology reports). The estimated incidence was 5.6 fold (95% CI: 2.3-13.8; p = 0.001) higher for carriers than for the general population with a corresponding cumulative risk to 80 years of 4.5% (95% CI: 1.9-10.7%) compared with 0.8% for the general population. The mean age at diagnosis was 43.1 years (95% CI: 40.0-46.2), 3.9 years younger than the reported USA population mean of 47.0 years (p = 0.02). Women with MMR gene mutations were found to have an increased risk of cervical cancer. Due to limited pathology verification we cannot be certain that a proportion of these cases were not lower uterine segment endometrial cancers involving the endocervix, a recognized cancer of Lynch syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association between body mass index and mortality for colorectal cancer survivors: overall and by tumor molecular phenotype</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Campbell PT, Newton CC, Newcomb PA, Phipps AI, Ahnen DJ, Baron JA, Buchanan DD, Casey G, Cleary SP, Cotterchio M, Farris AB, Figueiredo JC, Gallinger S, Green RC, Haile RW, Hopper JL, Jenkins MA, Le Marchand L, Makar KW, McLaughlin JR, Potter JD, Renehan AG, Sinicrope FA, Thibodeau SN, Ulrich CM, Win AK, Lindor NM, Limburg PJ</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">24</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-15-0094</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26038390</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-15-0094">http://dx.doi.org/10.1158/1055-9965.EPI-15-0094</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Microsatellite instability (MSI) and BRAF mutation status are associated with colorectal cancer survival, whereas the role of body mass index (BMI) is less clear. We evaluated the association between BMI and colorectal cancer survival, overall and by strata of MSI, BRAF mutation, sex, and other factors. METHODS: This study included 5,615 men and women diagnosed with invasive colorectal cancer who were followed for mortality (maximum: 14.7 years; mean: 5.9 years). Prediagnosis BMI was derived from self-reported weight approximately one year before diagnosis and height. Tumor MSI and BRAF mutation status were available for 4,131 and 4,414 persons, respectively. Multivariable hazard ratios (HR) and 95% confidence intervals (CI) were estimated from delayed-entry Cox proportional hazards models. RESULTS: In multivariable models, high prediagnosis BMI was associated with higher risk of all-cause mortality in both sexes (per 5-kg/m(2); HR, 1.10; 95% CI, 1.06-1.15), with similar associations stratified by sex (Pinteraction: 0.41), colon versus rectum (Pinteraction: 0.86), MSI status (Pinteraction: 0.84), and BRAF mutation status (Pinteraction: 0.28). In joint models, with MS-stable/MSI-low and normal BMI as the reference group, risk of death was higher for MS-stable/MSI-low and obese BMI (HR, 1.32; P value: 0.0002), not statistically significantly lower for MSI-high and normal BMI (HR, 0.86; P value: 0.29), and approximately the same for MSI-high and obese BMI (HR, 1.00; P value: 0.98). CONCLUSIONS: High prediagnosis BMI was associated with increased mortality; this association was consistent across participant subgroups, including strata of tumor molecular phenotype. IMPACT: High BMI may attenuate the survival benefit otherwise observed with MSI-high tumors.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Meta-analysis of genome-wide association studies identifies common susceptibility polymorphisms for colorectal and endometrial cancer near SH2B3 and TSHZ1</a>, Scientific reports, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Cheng TH, Thompson D, Painter J, O'Mara T, Gorman M, Martin L, Palles C, Jones A, Buchanan DD, Ko Win A, Hopper J, Jenkins M, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Giles GG, Pharoah P, Peto J, Cox A, Swerdlow A, Couch F, Cunningham JM, Goode EL, Winham SJ, Lambrechts D, Fasching P, Burwinkel B, Brenner H, Brauch H, Chang-Claude J, Salvesen HB, Kristensen V, Darabi H, Li J, Liu T, Lindblom A, Hall P, de Polanco ME, Sans M, Carracedo A, Castellvi-Bel S, Rojas-Martinez A, Aguiar Jnr S, Teixeira MR, Dunning AM, Dennis J, Otton G, Proietto T, Holliday E, Attia J, Ashton K, Scott RJ, McEvoy M, Dowdy SC, Fridley BL, Werner HM, Trovik J, Njolstad TS, Tham E, Mints M, Runnebaum I, Hillemanns P, Dork T, Amant F, Schrauwen S, Hein A, Beckmann MW, Ekici A, Czene K, Meindl A, Bolla MK, Michailidou K, Tyrer JP, Wang Q, Ahmed S, Healey CS, Shah M, Annibali D, Depreeuw J, Al-Tassan NA, Harris R, Meyer BF, Whiffin N, Hosking FJ, Kinnersley B, Farrington SM, Timofeeva M, Tenesa A, Campbell H, Haile RW, Hodgson S, Carvajal-Carmona L, Cheadle JP, Easton D, Dunlop M, Houlston R, Spurdle A, Tomlinson I</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Scientific reports</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/srep17369</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26621817</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/srep17369">http://dx.doi.org/10.1038/srep17369</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">High-risk mutations in several genes predispose to both colorectal cancer (CRC) and endometrial cancer (EC). We therefore hypothesised that some lower-risk genetic variants might also predispose to both CRC and EC. Using CRC and EC genome-wide association series, totalling 13,265 cancer cases and 40,245 controls, we found that the protective allele [G] at one previously-identified CRC polymorphism, rs2736100 near TERT, was associated with EC risk (odds ratio (OR) = 1.08, P = 0.000167); this polymorphism influences the risk of several other cancers. A further CRC polymorphism near TERC also showed evidence of association with EC (OR = 0.92; P = 0.03). Overall, however, there was no good evidence that the set of CRC polymorphisms was associated with EC risk, and neither of two previously-reported EC polymorphisms was associated with CRC risk. A combined analysis revealed one genome-wide significant polymorphism, rs3184504, on chromosome 12q24 (OR = 1.10, P = 7.23 x 10(-9)) with shared effects on CRC and EC risk. This polymorphism, a missense variant in the gene SH2B3, is also associated with haematological and autoimmune disorders, suggesting that it influences cancer risk through the immune response. Another polymorphism, rs12970291 near gene TSHZ1, was associated with both CRC and EC (OR = 1.26, P = 4.82 x 10(-8)), with the alleles showing opposite effects on the risks of the two cancers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Female Hormonal Factors and the Risk of Endometrial Cancer in Lynch Syndrome</a>, JAMA: the journal of the American Medical Association, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Dashti SG, Chau R, Ouakrim DA, Buchanan DD, Clendenning M, Young JP, Winship IM, Arnold J, Ahnen DJ, Haile RW, Casey G, Gallinger S, Thibodeau SN, Lindor NM, Le Marchand L, Newcomb PA, Potter JD, Baron JA, Hopper JL, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">JAMA: the journal of the American Medical Association</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">314</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1001/jama.2015.6789</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26151267</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1001/jama.2015.6789">http://dx.doi.org/10.1001/jama.2015.6789</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Aged Aged, 80 and over Colorectal Neoplasms, Hereditary Nonpolyposis/*complications/genetics Contraceptives, Oral, Hormonal/*pharmacology DNA Mismatch Repair/*genetics Endometrial Neoplasms/*etiology/prevention & control *Estrogen Replacement Therapy Female Humans Maternal Age Menarche Menopause Middle Aged *Mutation Risk Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">IMPORTANCE: Apart from hysterectomy, there is no consensus recommendation for reducing endometrial cancer risk for women with a mismatch repair gene mutation (Lynch syndrome). OBJECTIVE: To investigate the association between hormonal factors and endometrial cancer risk in Lynch syndrome. DESIGN, SETTING, AND PARTICIPANTS: A retrospective cohort study included 1128 women with a mismatch repair gene mutation identified from the Colon Cancer Family Registry. Data were analyzed with a weighted cohort approach. Participants were recruited between 1997 and 2012 from centers across the United States, Australia, Canada, and New Zealand. EXPOSURES: Age at menarche, first and last live birth, and menopause; number of live births; hormonal contraceptive use; and postmenopausal hormone use. MAIN OUTCOMES AND MEASURES: Self-reported diagnosis of endometrial cancer. RESULTS: Endometrial cancer was diagnosed in 133 women (incidence rate per 100 person-years, 0.29; 95% CI, 0.24 to 0.34). Endometrial cancer was diagnosed in 11% (n = 70) of women with age at menarche greater than or equal to 13 years compared with 12.6% (n = 57) of women with age at menarche less than 13 years (incidence rate per 100 person-years, 0.27 vs 0.31; rate difference, -0.04 [95% CI, -0.15 to 0.05]; hazard ratio per year, 0.85 [95% CI, 0.73 to 0.99]; P = .04). Endometrial cancer was diagnosed in 10.8% (n = 88) of parous women compared with 14.4% (n = 40) of nulliparous women (incidence rate per 100 person-years, 0.25 vs 0.43; rate difference, -0.18 [95% CI, -0.32 to -0.04]; hazard ratio, 0.21 [95% CI, 0.10 to 0.42]; P < .001). Endometrial cancer was diagnosed in 8.7% (n = 70) of women who used hormonal contraceptives greater than or equal to 1 year compared with 19.2% (n = 57) of women who used contraceptives less than 1 year (incidence rate per 100 person-years, 0.22 vs 0.45; rate difference, -0.23 [95% CI, -0.36 to -0.11]; hazard ratio, 0.39 [95% CI, 0.23 to 0.64]; P < .001). There was no statistically significant association between endometrial cancer and age at first and last live birth, age at menopause, and postmenopausal hormone use. CONCLUSIONS AND RELEVANCE: For women with a mismatch repair gene mutation, some endogenous and exogenous hormonal factors were associated with a lower risk of endometrial cancer. These directions and strengths of associations were similar to those for the general population. If replicated, these findings suggest that women with a mismatch repair gene mutation may be counseled like the general population in regard to hormonal influences on endometrial cancer risk.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Prediagnostic Physical Activity and Colorectal Cancer Survival: Overall and Stratified by Tumor Characteristics</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Hardikar S, Newcomb PA, Campbell PT, Win AK, Lindor NM, Buchanan DD, Makar KW, Jenkins MA, Potter JD, Phipps AI</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">24</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-15-0039</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25976417</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-15-0039">http://dx.doi.org/10.1158/1055-9965.EPI-15-0039</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Physical activity is associated with a lower incidence of colorectal cancer; however, the relationship of physical activity with colorectal cancer survival is not yet clear. We evaluated the association between prediagnostic physical activity and colorectal cancer survival, overall and accounting for tumor markers associated with colorectal cancer survival: BRAF and KRAS mutation status and microsatellite instability (MSI) status. METHODS: Participants were 20- to 74-year-old colorectal cancer patients diagnosed between 1998 and 2007 from the population-based Seattle Colon Cancer Family Registry (S-CCFR). Self-reported physical activity in the years preceding colorectal cancer diagnosis was summarized as average metabolic equivalent task hours per week (MET-h/wk; n = 1,309). Somatic BRAF and KRAS mutations and MSI status were evaluated on a subset of patients (n = 1043). Cox regression was used to estimate HRs and 95% confidence intervals (CI) for overall and disease-specific survival after adjusting for relevant confounders. Stratified analyses were conducted across categories of BRAF, KRAS, and MSI, as well as tumor stage and site. RESULTS: Higher prediagnostic recreational physical activity was associated with significantly more favorable overall survival (HR for highest vs. lowest category, 0.70; 95% CI, 0.52-0.96); associations were similar for colorectal cancer-specific survival. Results consistently indicated a favorable association with physical activity across strata defined by tumor characteristics. CONCLUSION: Individuals who were physically active before colorectal cancer diagnosis experienced better survival than those who were inactive or minimally active. IMPACT: Our results support existing physical activity recommendations for colorectal cancer patients and suggest that the beneficial effect of activity is not specific to a particular molecular phenotype of colorectal cancer.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Childhood cancers in families with and without Lynch syndrome</a>, Familial cancer, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Heath JA, Reece JC, Buchanan DD, Casey G, Durno CA, Gallinger S, Haile RW, Newcomb PA, Potter JD, Thibodeau SN, Le Marchand L, Lindor NM, Hopper JL, Jenkins MA, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">14</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-015-9810-3</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25963852</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-015-9810-3">http://dx.doi.org/10.1007/s10689-015-9810-3</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Childhood cancer Familial cancer Lynch syndrome Mismatch repair</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Inheritance of a germline mutation in one of the DNA mismatch repair (MMR) genes or the EPCAM gene is associated with an increased risk of colorectal cancer, endometrial cancer, and other adult malignancies (Lynch syndrome). The risk of childhood cancers in Lynch syndrome families, however, is not well studied. Using data from the Colon Cancer Family Registry, we compared the proportion of childhood cancers (diagnosed before 18 years of age) in the first-, second-, and third-degree relatives of 781 probands with a pathogenic mutation in one of the MMR genes; MLH1 (n = 275), MSH2 (n = 342), MSH6 (n = 99), or PMS2 (n = 55) or in EPCAM (n = 10) (Lynch syndrome families), with that of 5073 probands with MMR-deficient colorectal cancer (non-Lynch syndrome families). There was no evidence of a difference in the proportion of relatives with a childhood cancer between Lynch syndrome families (41/17,230; 0.24 %) and non-Lynch syndrome families (179/94,302; 0.19 %; p = 0.19). Incidence rate of all childhood cancers was estimated to be 147 (95 % CI 107-206) per million population per year in Lynch syndrome families and 115 (95 % CI 99.1-134) per million population per year in non-Lynch syndrome families. There was no evidence for a significant increase in the risk of all childhood cancers, hematologic cancers, brain and central nervous system cancers, Lynch syndrome-associated cancers, or other cancers in Lynch syndrome families compared with non-Lynch syndrome families. Larger studies, however, are required to more accurately define the risk of specific individual childhood cancers in Lynch syndrome families.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Consequences of germline variation disrupting the constitutional translational initiation codon start sites of MLH1 and BRCA2: Use of potential alternative start sites and implications for predicting variant pathogenicity</a>, Molecular carcinogenesis, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Parsons MT, Whiley PJ, Beesley J, Drost M, de Wind N, Thompson BA, Marquart L, Hopper JL, Jenkins MA, Australasian Colorectal Cancer Family,  Registry, Brown MA, Tucker K, Warwick L, Buchanan DD, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Molecular carcinogenesis</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">54</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/mc.22116</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24302565</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/mc.22116">http://dx.doi.org/10.1002/mc.22116</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics *Alternative Splicing BRCA2 Protein/*genetics *Codon, Initiator DNA Mismatch Repair Exons Genes, BRCA2 Humans Neoplasms/*genetics Nuclear Proteins/*genetics Protein Biosynthesis Protein Isoforms/genetics cancer syndrome genes in vitro assay unclassified variant</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Variants that disrupt the translation initiation sequences in cancer predisposition genes are generally assumed to be deleterious. However, few studies have validated these assumptions with functional and clinical data. Two cancer syndrome gene variants likely to affect native translation initiation were identified by clinical genetic testing: MLH1:c.1A>G p.(Met1?) and BRCA2:c.67+3A>G. In vitro GFP-reporter assays were conducted to assess the consequences of translation initiation disruption on alternative downstream initiation codon usage. Analysis of MLH1:c.1A>G p.(Met1?) showed that translation was mostly initiated at an in-frame position 103 nucleotides downstream, but also at two ATG sequences downstream. The protein product encoded by the in-frame transcript initiating from position c.103 showed loss of in vitro mismatch repair activity comparable to known pathogenic mutations. BRCA2:c.67+3A>G was shown by mRNA analysis to result in an aberrantly spliced transcript deleting exon 2 and the consensus ATG site. In the absence of exon 2, translation initiated mostly at an out-of-frame ATG 323 nucleotides downstream, and to a lesser extent at an in-frame ATG 370 nucleotides downstream. Initiation from any of the downstream alternative sites tested in both genes would lead to loss of protein function, but further clinical data is required to confirm if these variants are associated with a high cancer risk. Importantly, our results highlight the need for caution in interpreting the functional and clinical consequences of variation that leads to disruption of the initiation codon, since translation may not necessarily occur from the first downstream alternative start site, or from a single alternative start site.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association between molecular subtypes of colorectal cancer and patient survival</a>, Gastroenterology, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Phipps AI, Limburg PJ, Baron JA, Burnett-Hartman AN, Weisenberger DJ, Laird PW, Sinicrope FA, Rosty C, Buchanan DD, Potter JD, Newcomb PA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gastroenterology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">148</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1053/j.gastro.2014.09.038</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25280443</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1053/j.gastro.2014.09.038">http://dx.doi.org/10.1053/j.gastro.2014.09.038</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Biomarkers, Tumor/*genetics Colorectal Neoplasms/*genetics/mortality/pathology/therapy *CpG Islands *DNA Methylation DNA Mutational Analysis Disease-Free Survival Female Genetic Predisposition to Disease Humans Kaplan-Meier Estimate Male *Microsatellite Instability Middle Aged Mutation Phenotype Proportional Hazards Models Proto-Oncogene Proteins/*genetics Proto-Oncogene Proteins B-raf/*genetics Proto-Oncogene Proteins p21(ras) Registries Risk Factors Time Factors Treatment Outcome Washington Young Adult ras Proteins/*genetics Methylation Oncogene Prognostic Factor Serrated Colorectal Cancer</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND AND AIMS: Colorectal cancer (CRC) is a heterogeneous disease that can develop via several pathways. Different CRC subtypes, identified based on tumor markers, have been proposed to reflect these pathways. We evaluated the significance of these previously proposed classifications to survival. METHODS: Participants in the population-based Seattle Colon Cancer Family Registry were diagnosed with invasive CRC from 1998 through 2007 in western Washington State (N = 2706), and followed for survival through 2012. Tumor samples were collected from 2050 participants and classified into 5 subtypes based on combinations of tumor markers: type 1 (microsatellite instability [MSI]-high, CpG island methylator phenotype [CIMP] -positive, positive for BRAF mutation, negative for KRAS mutation); type 2 (microsatellite stable [MSS] or MSI-low, CIMP-positive, positive for BRAF mutation, negative for KRAS mutation); type 3 (MSS or MSI low, non-CIMP, negative for BRAF mutation, positive for KRAS mutation); type 4 (MSS or MSI-low, non-CIMP, negative for mutations in BRAF and KRAS); and type 5 (MSI-high, non-CIMP, negative for mutations in BRAF and KRAS). Multiple imputation was used to impute tumor markers for those missing data on 1-3 markers. We used Cox regression to estimate hazard ratios (HR) and 95% confidence intervals (CI) for associations of subtypes with disease-specific and overall mortality, adjusting for age, sex, body mass, diagnosis year, and smoking history. RESULTS: Compared with participants with type 4 tumors (the most predominant), participants with type 2 tumors had the highest disease-specific mortality (HR = 2.20, 95% CI: 1.47-3.31); subjects with type 3 tumors also had higher disease-specific mortality (HR = 1.32, 95% CI: 1.07-1.63). Subjects with type 5 tumors had the lowest disease-specific mortality (HR = 0.30, 95% CI: 0.14-0.66). Associations with overall mortality were similar to those with disease-specific mortality. CONCLUSIONS: Based on a large, population-based study, CRC subtypes, defined by proposed etiologic pathways, are associated with marked differences in survival. These findings indicate the clinical importance of studies into the molecular heterogeneity of CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Do serrated neoplasms of the small intestine represent a distinct entity? Pathological findings and molecular alterations in a series of 13 cases</a>, Histopathology, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Campbell C, Clendenning M, Bettington M, Buchanan DD, Brown IS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Histopathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">66</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1111/his.12469</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24894811</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1111/his.12469">http://dx.doi.org/10.1111/his.12469</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenocarcinoma/*genetics/*pathology Adenoma/*genetics/*pathology Adult Aged Aged, 80 and over Biomarkers, Tumor/genetics/metabolism DNA Mutational Analysis Female Humans Immunohistochemistry Intestinal Neoplasms/*genetics/*pathology Intestine, Small/pathology Male Middle Aged Braf Cimp Kras serrated adenocarcinoma serrated adenoma small intestine traditional serrated adenoma</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">AIMS: To characterize pathological, immunohistochemical and molecular features of small intestinal serrated neoplasms. METHODS AND RESULTS: We report 13 serrated neoplasms located predominantly in the duodenum (median age, 71 years; male to female ratio, 7:6). The serrated adenomas demonstrated prominent serration, ectopic crypt formations and cytological features reminiscent of colorectal traditional serrated adenomas. Almost half the serrated adenomas demonstrated high-grade dysplasia or were associated with an adenocarcinoma. Immunohistochemical and molecular analysis showed an intestinal (CDX2-positive) phenotype in all tumours, abnormal beta-catenin staining in three cases (23%), abnormal p53 expression in four cases (31%), focal loss of MGMT expression in one case (8%), KRAS mutation in five cases (38%) and CpG island methylator phenotype in six cases (50%). A diffuse pattern of Ki67 expression was present in eight adenomas (62%) and was associated with high-grade dysplasia (P = 0.02). No BRAF(V600E) mutation or loss of MLH1 expression was observed. CONCLUSIONS: To our knowledge, this is the first series reporting serrated adenoma in the small intestine, a rare subtype of adenomas resembling traditional serrated adenoma with aggressive morphological features. The absence of the BRAF(V600E) mutation does not support a role for the serrated neoplasia pathway in the development of these lesions, as in colorectal serrated polyps.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>The intestinal epithelial cell differentiation marker intestinal alkaline phosphatase (ALPi) is selectively induced by histone deacetylase inhibitors (HDACi) in colon cancer cells in a Kruppel-like factor 5 (KLF5)-dependent manner</a>, The Journal of biological chemistry, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Shin J, Carr A, Corner GA, Togel L, Davalos-Salas M, Tran H, Chueh AC, Al-Obaidi S, Chionh F, Ahmed N, Buchanan DD, Young JP, Malo MS, Hodin RA, Arango D, Sieber OM, Augenlicht LH, Dhillon AS, Weber TK, Mariadason JM</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The Journal of biological chemistry</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">290</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">25</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1074/jbc.A114.557546</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26092983</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1074/jbc.A114.557546">http://dx.doi.org/10.1074/jbc.A114.557546</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Shin, Joongho Carr, Azadeh Corner, Georgia A Togel, Lars Davalos-Salas, Mercedes Tran, Hoanh Chueh, Anderly C Al-Obaidi, Sheren Chionh, Fiona Ahmed, Naseem Buchanan, Daniel D Young, Joanne P Malo, Madhu S Hodin, Richard A Arango, Diego Sieber, Oliver M Augenlicht, Leonard H Dhillon, Amardeep S Weber, Thomas K Mariadason, John M eng Published Erratum 2015/06/21 06:00 J Biol Chem. 2015 Jun 19;290(25):15392. doi: 10.1074/jbc.A114.557546.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association of the colorectal CpG island methylator phenotype with molecular features, risk factors, and family history</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Weisenberger DJ, Levine AJ, Long TI, Buchanan DD, Walters R, Clendenning M, Rosty C, Joshi AD, Stern MC, Le Marchand L, Lindor NM, Daftary D, Gallinger S, Selander T, Bapat B, Newcomb PA, Campbell PT, Casey G, Ahnen DJ, Baron JA, Haile RW, Hopper JL, Young JP, Laird PW, Siegmund KD, Registry,  for the Colon Cancer Family</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">24</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-14-1161</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25587051</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-14-1161">http://dx.doi.org/10.1158/1055-9965.EPI-14-1161</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Colorectal Neoplasms/*genetics *CpG Islands *DNA Methylation Family Health Female Genetic Predisposition to Disease Humans Male Middle Aged Phenotype Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: The CpG island methylator phenotype (CIMP) represents a subset of colorectal cancers characterized by widespread aberrant DNA hypermethylation at select CpG islands. The risk factors and environmental exposures contributing to etiologic heterogeneity between CIMP and non-CIMP tumors are not known. METHODS: We measured the CIMP status of 3,119 primary population-based colorectal cancer tumors from the multinational Colon Cancer Family Registry. Etiologic heterogeneity was assessed by a case-case study comparing risk factor frequency of colorectal cancer cases with CIMP and non-CIMP tumors using logistic regression to estimate the case-case odds ratio (ccOR). RESULTS: We found associations between tumor CIMP status and MSI-H (ccOR = 7.6), BRAF V600E mutation (ccOR = 59.8), proximal tumor site (ccOR = 9; all P < 0.0001), female sex [ccOR = 1.8; 95% confidence interval (CI), 1.5-2.1], older age (ccOR = 4.0 comparing over 70 years vs. under 50; 95% CI, 3.0-5.5), and family history of CRC (ccOR = 0.6; 95% CI, 0.5-0.7). While use of NSAIDs varied by tumor CIMP status for both males and females (P = 0.0001 and P = 0.02, respectively), use of multivitamin or calcium supplements did not. Only for female colorectal cancer was CIMP status associated with increased pack-years of smoking (Ptrend < 0.001) and body mass index (BMI; Ptrend = 0.03). CONCLUSIONS: The frequency of several colorectal cancer risk factors varied by CIMP status, and the associations of smoking and obesity with tumor subtype were evident only for females. IMPACT: Differences in the associations of a unique DNA methylation-based subgroup of colorectal cancer with important lifestyle and environmental exposures increase understanding of the molecular pathologic epidemiology of this heavily methylated subset of colorectal cancer. Cancer Epidemiol Biomarkers Prev; 24(3); 512-9. (c)2015 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Role of tumour molecular and pathology features to estimate colorectal cancer risk for first-degree relatives</a>, Gut, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Buchanan DD, Rosty C, MacInnis RJ, Dowty JG, Dite GS, Giles GG, Southey MC, Young JP, Clendenning M, Walsh MD, Walters RJ, Boussioutas A, Smyrk TC, Thibodeau SN, Baron JA, Potter JD, Newcomb PA, Le Marchand L, Haile RW, Gallinger S, Lindor NM, Hopper JL, Ahnen DJ, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gut</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">64</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1136/gutjnl-2013-306567</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24615377</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1136/gutjnl-2013-306567">http://dx.doi.org/10.1136/gutjnl-2013-306567</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Aged Cohort Studies Colorectal Neoplasms/epidemiology/*genetics/*pathology Colorectal Neoplasms, Hereditary Nonpolyposis/genetics/pathology Family Health Female Humans Male Middle Aged Risk Assessment Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">OBJECTIVE: To estimate risk of colorectal cancer (CRC) for first-degree relatives of CRC cases based on CRC molecular subtypes and tumour pathology features. DESIGN: We studied a cohort of 33,496 first-degree relatives of 4853 incident invasive CRC cases (probands) who were recruited to the Colon Cancer Family Registry through population cancer registries in the USA, Canada and Australia. We categorised the first-degree relatives into four groups: 28,156 of 4095 mismatch repair (MMR)-proficient probands, 2302 of 301 MMR-deficient non-Lynch syndrome probands, 1799 of 271 suspected Lynch syndrome probands and 1239 of 186 Lynch syndrome probands. We compared CRC risk for first-degree relatives stratified by the absence or presence of specific tumour molecular pathology features in probands across each of these four groups and for all groups combined. RESULTS: Compared with first-degree relatives of MMR-proficient CRC cases, a higher risk of CRC was estimated for first-degree relatives of CRC cases with suspected Lynch syndrome (HR 2.06, 95% CI 1.59 to 2.67) and with Lynch syndrome (HR 5.37, 95% CI 4.16 to 6.94), but not with MMR-deficient non-Lynch syndrome (HR 1.04, 95% CI 0.82 to 1.31). A greater risk of CRC was estimated for first-degree relatives if CRC cases were diagnosed before age 50 years, had proximal colon cancer or if their tumours had any of the following: expanding tumour margin, peritumoral lymphocytes, tumour-infiltrating lymphocytes or synchronous CRC. CONCLUSIONS: Molecular pathology features are potentially useful to refine screening recommendations for first-degree relatives of CRC cases and to identify which cases are more likely to be caused by genetic or other familial factors.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Genetic variants within the hTERT gene and the risk of colorectal cancer in Lynch syndrome</a>, Genes & cancer, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Clendenning M, Crawford W, Rosty C, Preston SG, Southey MC, Parry S, Giles GG, Macrae FA, Winship IM, Baron JA, Hopper JL, Jenkins MA, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Genes & cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11-12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26807197</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/26807197">https://www.ncbi.nlm.nih.gov/pubmed/26807197</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Lynch syndrome colorectal cancer genetic modifier genetic variant hTERT</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Lynch syndrome is an inherited cancer-predisposing disorder caused by germline mutations in the DNA mismatch repair (MMR) genes but there is a high degree of variability in cancer risk observed among carriers, suggesting the existence of modifying factors. Our aim was to investigate variants within the hTERT gene as a potential colorectal cancer (CRC) risk modifier for MMR gene mutation carriers. We identified 1098 MMR gene mutation carriers (420 MLH1, 481 MSH2, 126 MSH6, 53 PMS2 and 18 EPCAM) from 330 families recruited from either family cancer clinics or population cancer registries of the Australasian Colorectal Cancer Family Registry between 1997 and 2012. Using weighted Cox regression after adjusting for ascertainment bias, we estimated associations between 23 SNPs within the hTERT gene and CRC risk. During 46,836 person-years observation, 392 (36%) carriers were diagnosed with CRC at a mean age of 42.2 (standard deviation 11.4) years. There was no evidence of association between any of the hTERT SNPs and CRC risk, overall and stratified by sex and MMR gene mutated, after adjustment for multiple testing. Our findings suggest no evidence for clinical utility of the SNPs within the hTERT gene in Lynch syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk of colorectal cancer for people with a mutation in both a MUTYH and a DNA mismatch repair gene</a>, Familial cancer, 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Reece JC, Buchanan DD, Clendenning M, Young JP, Cleary SP, Kim H, Cotterchio M, Dowty JG, MacInnis RJ, Tucker KM, Winship IM, Macrae FA, Burnett T, Le Marchand L, Casey G, Haile RW, Newcomb PA, Thibodeau SN, Lindor NM, Hopper JL, Gallinger S, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">14</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-015-9824-x</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26202870</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-015-9824-x">http://dx.doi.org/10.1007/s10689-015-9824-x</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal cancer Lynch syndrome Mutyh Mismatch repair</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The base excision repair protein, MUTYH, functionally interacts with the DNA mismatch repair (MMR) system. As genetic testing moves from testing one gene at a time, to gene panel and whole exome next generation sequencing approaches, understandin g the risk associated with co-existence of germline mutations in these genes will be important for clinical interpretation and management. From the Colon Cancer Family Registry, we identified 10 carriers who had both a MUTYH mutation (6 with c.1187G>A p.(Gly396Asp), 3 with c.821G>A p.(Arg274Gln), and 1 with c.536A>G p.(Tyr179Cys)) and a MMR gene mutation (3 in MLH1, 6 in MSH2, and 1 in PMS2), 375 carriers of a single (monoallelic) MUTYH mutation alone, and 469 carriers of a MMR gene mutation alone. Of the 10 carriers of both gene mutations, 8 were diagnosed with colorectal cancer. Using a weighted cohort analysis, we estimated that risk of colorectal cancer for carriers of both a MUTYH and a MMR gene mutation was substantially higher than that for carriers of a MUTYH mutation alone [hazard ratio (HR) 21.5, 95 % confidence interval (CI) 9.19-50.1; p < 0.001], but not different from that for carriers of a MMR gene mutation alone (HR 1.94, 95 % CI 0.63-5.99; p = 0.25). Within the limited power of this study, there was no evidence that a monoallelic MUTYH gene mutation confers additional risk of colorectal cancer for carriers of a MMR gene mutation alone. Our finding suggests MUTYH mutation testing in MMR gene mutation carriers is not clinically informative.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Germline TP53 Mutations in Patients With Early-Onset Colorectal Cancer in the Colon Cancer Family Registry</a>, , 2015.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Yurgelun MB, Masciari S, Joshi VA, Mercado RC, Lindor NM, Gallinger S, Hopper JL, Jenkins MA, Buchanan DD, Newcomb PA, Potter JD, Haile RW, Kucherlapati R, Syngal S, Colon Cancer Family,  Registry</dd>
                  
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2015</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1001/jamaoncol.2015.0197</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">26086041</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1001/jamaoncol.2015.0197">http://dx.doi.org/10.1001/jamaoncol.2015.0197</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">IMPORTANCE: Li-Fraumeni syndrome, usually characterized by germline TP53 mutations, is associated with markedly elevated lifetime risks of multiple cancers, and has been linked to an increased risk of early-onset colorectal cancer. OBJECTIVE: To examine the frequency of germline TP53 alterations in patients with early-onset colorectal cancer. DESIGN, SETTING, AND PARTICIPANTS: This was a multicenter cross-sectional cohort study of individuals recruited to the Colon Cancer Family Registry (CCFR) from 1998 through 2007 (genetic testing data updated as of January 2015). Both population-based and clinic-based patients in the United States, Canada, Australia, and New Zealand were recruited to the CCFR. Demographic information, clinical history, and family history data were obtained at enrollment. Biospecimens were collected from consenting probands and families, including microsatellite instability and DNA mismatch repair immunohistochemistry results. A total of a 510 individuals diagnosed as having colorectal cancer at age 40 years or younger and lacking a known hereditary cancer syndrome were identified from the CCFR as being potentially eligible. Fifty-three participants were excluded owing to subsequent identification of germline mutations in DNA mismatch repair genes (n = 47) or biallelic MUTYH mutations (n = 6). INTERVENTIONS: Germline sequencing of the TP53 gene was performed. Identified TP53 alterations were assessed for pathogenicity using literature and international mutation database searches and in silico prediction models. MAIN OUTCOMES AND MEASURES: Frequency of nonsynonymous germline TP53 alterations. RESULTS: Among 457 eligible participants (314, population-based; 143, clinic-based; median age at diagnosis, 36 years [range, 15-40 years]), 6 (1.3%; 95% CI, 0.5%-2.8%) carried germline missense TP53 alterations, none of whom met clinical criteria for Li-Fraumeni syndrome. Four of the identified TP53 alterations have been previously described in the literature in probands with clinical features of Li-Fraumeni syndrome, and 2 were novel alterations. CONCLUSIONS AND RELEVANCE: In a large cohort of patients with early-onset colorectal cancer, germline TP53 mutations were detected at a frequency comparable with the published prevalence of germline APC mutations in colorectal cancer. With the increasing use of multigene next-generation sequencing panels in hereditary cancer risk assessment, clinicians will be faced with the challenge of interpreting the biologic and clinical significance of germline TP53 mutations in families whose phenotypes are atypical for Li-Fraumeni syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2014</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>The association of telomere length with colorectal cancer differs by the age of cancer onset</a>, Clinical and translational gastroenterology, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Boardman LA, Litzelman K, Seo S, Johnson RA, Vanderboom RJ, Kimmel GW, Cunningham JM, Gangnon RE, Engelman CD, Riegert-Johnson DL, Potter J, Haile R, Buchanan D, Jenkins MA, Rider DN, Thibodeau SN, Petersen GM, Skinner HG</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Clinical and translational gastroenterology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/ctg.2014.3</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24598784</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/ctg.2014.3">http://dx.doi.org/10.1038/ctg.2014.3</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">OBJECTIVES: Telomeres are nucleoprotein structures that cap the end of chromosomes and shorten with sequential cell divisions in normal aging. Short telomeres are also implicated in the incidence of many cancers, but the evidence is not conclusive for colorectal cancer (CRC). Therefore, the aim of this study was to assess the association of CRC and telomere length. METHODS: In this case-control study, we measured relative telomere length from peripheral blood leukocytes (PBLs) DNA with quantitative PCR in 598 CRC patients and 2,212 healthy controls. RESULTS: Multivariate analysis indicated that telomere length was associated with risk for CRC, and this association varied in an age-related manner; younger individuals (50 years of age) with shortened telomeres (1-10 percentiles) had 2-12 times the risk for CRC. The risk for CRC varies with extremes in telomere length in an age-associated manner. CONCLUSIONS: Younger individuals with longer telomeres or older individuals with shorter telomeres are at higher risk for CRC. These findings indicate that the association of PBL telomere length varies according to the age of cancer onset and that CRC is likely associated with at minimum two different mechanisms of telomere dynamics.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Clinical problems of colorectal cancer and endometrial cancer cases with unknown cause of tumor mismatch repair deficiency (suspected Lynch syndrome)</a>, The application of clinical genetics, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Rosty C, Clendenning M, Spurdle AB, Win AK</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The application of clinical genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">7</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.2147/TACG.S48625</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25328415</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.2147/TACG.S48625">http://dx.doi.org/10.2147/TACG.S48625</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10"> 59% (95% confidence interval [CI]: 55% to 64%) of colorectal cancers and 52% (95% CI: 41% to 62%) of endometrial cancers with MMR deficiency were identified as suspected Lynch syndrome. Recent studies estimated that colorectal cancer risk for relatives of suspected Lynch syndrome cases is lower than for relatives of those with MMR gene mutations</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Carriers of a germline mutation in one of the DNA mismatch repair (MMR) genes have a high risk of developing numerous different cancers, predominantly colorectal cancer and endometrial cancer (known as Lynch syndrome). MMR gene mutation carriers develop tumors with MMR deficiency identified by tumor microsatellite instability or immunohistochemical loss of MMR protein expression. Tumor MMR deficiency is used to identify individuals most likely to carry an MMR gene mutation. However, MMR deficiency can also result from somatic inactivation, most commonly methylation of the MLH1 gene promoter. As tumor MMR testing of all incident colorectal and endometrial cancers (universal screening) is becoming increasingly adopted, a growing clinical problem is emerging for individuals who have tumors that show MMR deficiency who are subsequently found not to carry an MMR gene mutation after genetic testing using the current diagnostic approaches (Sanger sequencing and multiplex ligation-dependent probe amplification) and who also show no evidence of MLH1 methylation. The inability to determine the underlying cause of tumor MMR deficiency in these \Lynch-like\" or \"suspected Lynch syndrome\" cases has significant implications on the clinical management of these individuals and their relatives. When the data from published studies are combined</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Reply to J. Moline et al</a>, Journal of clinical oncology: official journal of the American Society of Clinical Oncology, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Tan YY, Walsh MD, Clendenning M, Metcalf AM, Ferguson K, Arnold ST, Thompson BA, Lose FA, Parsons MT, Walters RJ, Pearson SA, Cummings M, Oehler MK, Blomfield PB, Quinn MA, Kirk JA, Stewart CJ, Obermair A, Young JP, Webb PM, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of clinical oncology: official journal of the American Society of Clinical Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">32</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">21</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1200/JCO.2014.55.8213</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24912891</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1200/JCO.2014.55.8213">http://dx.doi.org/10.1200/JCO.2014.55.8213</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics *DNA Methylation DNA Mismatch Repair/*genetics Endometrial Neoplasms/*genetics Female *Germ-Line Mutation Humans Nuclear Proteins/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Buchanan, Daniel D Tan, Yen Y Walsh, Michael D Clendenning, Mark Metcalf, Alexander M Ferguson, Kaltin Arnold, Sven T Thompson, Bryony A Lose, Felicity A Parsons, Michael T Walters, Rhiannon J Pearson, Sally-Ann Cummings, Margaret Oehler, Martin K Blomfield, Penelope B Quinn, Michael A Kirk, Judy A Stewart, Colin J Obermair, Andreas Young, Joanne P Webb, Penelope M Spurdle, Amanda B eng U01 CA097735/CA/NCI NIH HHS/ Comment Letter Research Support, N.I.H., Extramural Research Support, Non-U.S. Gov't 2014/06/11 06:00 J Clin Oncol. 2014 Jul 20;32(21):2278-9. doi: 10.1200/JCO.2014.55.8213. Epub 2014 Jun 9.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Tumor mismatch repair immunohistochemistry and DNA MLH1 methylation testing of patients with endometrial cancer diagnosed at age younger than 60 years optimizes triage for population-level germline mismatch repair gene mutation testing</a>, Journal of clinical oncology: official journal of the American Society of Clinical Oncology, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Tan YY, Walsh MD, Clendenning M, Metcalf AM, Ferguson K, Arnold ST, Thompson BA, Lose FA, Parsons MT, Walters RJ, Pearson SA, Cummings M, Oehler MK, Blomfield PB, Quinn MA, Kirk JA, Stewart CJ, Obermair A, Young JP, Webb PM, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of clinical oncology: official journal of the American Society of Clinical Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">32</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1200/JCO.2013.51.2129</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24323032</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1200/JCO.2013.51.2129">http://dx.doi.org/10.1200/JCO.2013.51.2129</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics/metabolism Adenosine Triphosphatases/genetics/metabolism Adult Aged Aged, 80 and over Australia Cohort Studies *DNA Methylation DNA Mismatch Repair/*genetics DNA Repair Enzymes/genetics/metabolism DNA-Binding Proteins/genetics/metabolism Early Detection of Cancer/methods Endometrial Neoplasms/diagnosis/*genetics/metabolism Female Genetic Testing/methods *Germ-Line Mutation Humans Immunohistochemistry Middle Aged MutS Homolog 2 Protein/genetics/metabolism Nuclear Proteins/*genetics/metabolism Population Surveillance/methods</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">PURPOSE: Clinicopathologic data from a population-based endometrial cancer cohort, unselected for age or family history, were analyzed to determine the optimal scheme for identification of patients with germline mismatch repair (MMR) gene mutations. PATIENTS AND METHODS: Endometrial cancers from 702 patients recruited into the Australian National Endometrial Cancer Study (ANECS) were tested for MMR protein expression using immunohistochemistry (IHC) and for MLH1 gene promoter methylation in MLH1-deficient cases. MMR mutation testing was performed on germline DNA of patients with MMR-protein deficient tumors. Prediction of germline mutation status was compared for combinations of tumor characteristics, age at diagnosis, and various clinical criteria (Amsterdam, Bethesda, Society of Gynecologic Oncology, ANECS). RESULTS: Tumor MMR-protein deficiency was detected in 170 (24%) of 702 cases. Germline testing of 158 MMR-deficient cases identified 22 truncating mutations (3% of all cases) and four unclassified variants. Tumor MLH1 methylation was detected in 99 (89%) of 111 cases demonstrating MLH1/PMS2 IHC loss; all were germline MLH1 mutation negative. A combination of MMR IHC plus MLH1 methylation testing in women younger than 60 years of age at diagnosis provided the highest positive predictive value for the identification of mutation carriers at 46% versus </= 41% for any other criteria considered. CONCLUSION: Population-level identification of patients with MMR mutation-positive endometrial cancer is optimized by stepwise testing for tumor MMR IHC loss in patients younger than 60 years, tumor MLH1 methylation in individuals with MLH1 IHC loss, and germline mutations in patients exhibiting loss of MSH6, MSH2, or PMS2 or loss of MLH1/PMS2 with absence of MLH1 methylation.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Colorectal cancer cell lines are representative models of the main molecular subtypes of primary cancer</a>, Cancer research, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mouradov D, Sloggett C, Jorissen RN, Love CG, Li S, Burgess AW, Arango D, Strausberg RL, Buchanan D, Wormald S, O'Connor L, Wilding JL, Bicknell D, Tomlinson IP, Bodmer WF, Mariadason JM, Sieber OM</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer research</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">74</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/0008-5472.CAN-14-0013</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24755471</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/0008-5472.CAN-14-0013">http://dx.doi.org/10.1158/0008-5472.CAN-14-0013</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Cell Line, Tumor Chromosome Aberrations Colorectal Neoplasms/*genetics/metabolism DNA Copy Number Variations DNA Mutational Analysis Exome Gene Dosage Gene Frequency Genes, Neoplasm Humans Microsatellite Instability Transcriptome</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Human colorectal cancer cell lines are used widely to investigate tumor biology, experimental therapy, and biomarkers. However, to what extent these established cell lines represent and maintain the genetic diversity of primary cancers is uncertain. In this study, we profiled 70 colorectal cancer cell lines for mutations and DNA copy number by whole-exome sequencing and SNP microarray analyses, respectively. Gene expression was defined using RNA-Seq. Cell line data were compared with those published for primary colorectal cancers in The Cancer Genome Atlas. Notably, we found that exome mutation and DNA copy-number spectra in colorectal cancer cell lines closely resembled those seen in primary colorectal tumors. Similarities included the presence of two hypermutation phenotypes, as defined by signatures for defective DNA mismatch repair and DNA polymerase epsilon proofreading deficiency, along with concordant mutation profiles in the broadly altered WNT, MAPK, PI3K, TGFbeta, and p53 pathways. Furthermore, we documented mutations enriched in genes involved in chromatin remodeling (ARID1A, CHD6, and SRCAP) and histone methylation or acetylation (ASH1L, EP300, EP400, MLL2, MLL3, PRDM2, and TRRAP). Chromosomal instability was prevalent in nonhypermutated cases, with similar patterns of chromosomal gains and losses. Although paired cell lines derived from the same tumor exhibited considerable mutation and DNA copy-number differences, in silico simulations suggest that these differences mainly reflected a preexisting heterogeneity in the tumor cells. In conclusion, our results establish that human colorectal cancer lines are representative of the main subtypes of primary tumors at the genomic level, further validating their utility as tools to investigate colorectal cancer biology and drug responses.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association between hypermethylation of DNA repetitive elements in white blood cell DNA and pancreatic cancer</a>, Cancer epidemiology, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Neale RE, Clark PJ, Fawcett J, Fritschi L, Nagler BN, Risch HA, Walters RJ, Crawford WJ, Webb PM, Whiteman DC, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">38</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.canep.2014.08.006</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25201440</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.canep.2014.08.006">http://dx.doi.org/10.1016/j.canep.2014.08.006</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Pancreatic cancer is a leading cause of cancer-related deaths worldwide. Methylation of DNA may influence risk or be a marker of early disease. The aim of this study was to measure the association between methylation of three DNA repetitive elements in white blood cell (WBC) DNA and pancreatic cancer. DNA from WBCs of pancreatic cancer cases (n=559) and healthy unrelated controls (n=603) were tested for methylation of the LINE-1, Alu and Sat2 DNA repetitive elements using MethyLight quantitative PCR assays. Odds ratios (ORs) and 95% confidence intervals (95%CI) between both continuous measures of percent of methylated sample compared to a reference (PMR) or quintiles of PMR and pancreatic cancer, adjusted for age, sex, smoking, BMI, alcohol and higher education, were estimated. The PMR for each of the three markers was higher in cases than in controls, although only LINE-1 was significantly associated with pancreatic cancer (OR per log unit=1.37, 95%CI=1.16-1.63). The marker methylation score for all three markers combined was significantly associated with pancreatic cancer (p-trend=0.0006). There were no associations between measures of PMR and either presence of metastases, or timing of blood collection in relation to diagnosis, surgery, chemotherapy or death (all p>0.1). We observed an association between methylation of LINE-1 in WBC DNA and risk of pancreatic cancer. Further studies are needed to confirm this association.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Tumour MLH1 promoter region methylation testing is an effective prescreen for Lynch Syndrome (HNPCC)</a>, Journal of medical genetics, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Newton K, Jorgensen NM, Wallace AJ, Buchanan DD, Lalloo F, McMahon RF, Hill J, Evans DG</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of medical genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">51</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1136/jmedgenet-2014-102552</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25280751</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1136/jmedgenet-2014-102552">http://dx.doi.org/10.1136/jmedgenet-2014-102552</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND AND AIMS: Lynch syndrome (LS) patients have DNA mismatch repair deficiency and up to 80% lifetime risk of colorectal cancer (CRC). Screening of mutation carriers reduces CRC incidence and mortality. Selection for constitutional mutation testing relies on family history (Amsterdam and Bethesda Guidelines) and tumour-derived biomarkers. Initial biomarker analysis uses mismatch repair protein immunohistochemistry and microsatellite instability. Abnormalities in either identify mismatch repair deficiency but do not differentiate sporadic epigenetic defects, due to MLH1 promoter region methylation (13% of CRCs) from LS (4% of CRCs). A diagnostic biomarker capable of making this distinction would be valuable. This study compared two biomarkers in tumours with mismatch repair deficiency; quantification of methylation of the MLH1 promoter region using a novel assay and BRAF c.1799T>A, p.(Val600Glu) mutation status in the identification of constitutional mutations. METHODS: Tumour DNA was extracted (formalin fixed, paraffin embedded, FFPE tissue) and pyrosequencing used to test for MLH1 promoter methylation and presence of the BRAF c.1799T>A, p.(Val600Glu) mutation 71 CRCs from individuals with pathogenic MLH1 mutations and 73 CRCs with sporadic MLH1 loss. Specificity and sensitivity was compared. FINDINGSS: Unmethylated MLH1 promoter: sensitivity 94.4% (95% CI 86.2% to 98.4%), specificity 87.7% (95% CI 77.9% to 94.2%), Wild-type BRAF (codon 600): sensitivity 65.8% (95% CI 53.7% to 76.5%), specificity 98.6% (95% CI 92.4% to 100.0%) for the identification of those with pathogenic MLH1 mutations. CONCLUSIONS: Quantitative MLH1 promoter region methylation using pyrosequencing is superior to BRAF codon 600 mutation status in identifying constitutional mutations in mismatch repair deficient tumours.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Pooled analysis of iron-related genes in Parkinson's disease: association with transferrin</a>, Neurobiology of disease, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rhodes SL, Buchanan DD, Ahmed I, Taylor KD, Loriot MA, Sinsheimer JS, Bronstein JM, Elbaz A, Mellick GD, Rotter JI, Ritz B</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Neurobiology of disease</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">62</dd>
              
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.nbd.2013.09.019</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24121126</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.nbd.2013.09.019">http://dx.doi.org/10.1016/j.nbd.2013.09.019</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Aged Case-Control Studies Female Genetic Association Studies Humans Iron/metabolism Male Middle Aged Parkinson Disease/*genetics/metabolism Polymorphism, Single Nucleotide Receptors, Transferrin/genetics Transferrin/*genetics Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Pathologic features of Parkinson's disease (PD) include death of dopaminergic neurons in the substantia nigra, presence of alpha-synuclein containing Lewy bodies, and iron accumulation in PD-related brain regions. The observed iron accumulation may be contributing to PD etiology but it also may be a byproduct of cell death or cellular dysfunction. To elucidate the possible role of iron accumulation in PD, we investigated genetic variation in 16 genes related to iron homeostasis in three case-control studies from the United States, Australia, and France. After screening 90 haplotype tagging single nucleotide polymorphisms (SNPs) within the genes of interest in the US study population, we investigated the five most promising gene regions in two additional independent case-control studies. For the pooled data set (1289 cases, 1391 controls) we observed a protective association (OR=0.83, 95% CI: 0.71-0.96) between PD and a haplotype composed of the A allele at rs1880669 and the T allele at rs1049296 in transferrin (TF; GeneID: 7018). Additionally, we observed a suggestive protective association (OR=0.87, 95% CI: 0.74-1.02) between PD and a haplotype composed of the G allele at rs10247962 and the A allele at rs4434553 in transferrin receptor 2 (TFR2; GeneID: 7036). We observed no associations in our pooled sample for haplotypes in SLC40A1, CYB561, or HFE. Taken together with previous findings in model systems, our results suggest that TF or a TF-TFR2 complex may have a role in the etiology of PD, possibly through iron misregulation or mitochondrial dysfunction within dopaminergic neurons.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>High prevalence of mismatch repair deficiency in prostate cancers diagnosed in mismatch repair gene mutation carriers from the colon cancer family registry</a>, Familial cancer, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Walsh MD, Lindor NM, Thibodeau SN, Mundt E, Gallinger S, Aronson M, Pollett A, Baron JA, Pearson S, Clendenning M, Walters RJ, Nagler BN, Crawford WJ, Young JP, Winship I, Win AK, Hopper JL, Jenkins MA, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">13</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-014-9744-1</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25117503</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-014-9744-1">http://dx.doi.org/10.1007/s10689-014-9744-1</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The question of whether prostate cancer is part of the Lynch syndrome spectrum of tumors is unresolved. We investigated the mismatch repair (MMR) status and pathologic features of prostate cancers diagnosed in MMR gene mutation carriers. Prostate cancers (mean age at diagnosis = 62 +/- SD = 8 years) from 32 MMR mutation carriers (23 MSH2, 5 MLH1 and 4 MSH6) enrolled in the Australasian, Mayo Clinic and Ontario sites of the Colon Cancer Family Registry were examined for clinico-pathologic features and MMR-deficiency (immunohistochemical loss of MMR protein expression and high levels of microsatellite instability; MSI-H). Tumor MMR-deficiency was observed for 22 cases [69 %; 95 % confidence interval (CI) 50-83 %], with the highest prevalence of MMR-deficiency in tumors from MSH2 mutation carriers (19/23, 83 %) compared with MLH1 and MSH6 carriers combined (3/9, 33 %; p = 0.01). MMR-deficient tumors had increased levels of tumor infiltrating lymphocytes compared with tumors without MMR-deficiency (p = 0.04). Under the assumption that tumour MMR-deficiency occurred only because the cancer was caused by the germline mutation, mutation carriers are at 3.2-fold (95 % CI 2.0-6.3) increased risk of prostate cancer, and when assessed by gene, the relative risk was greatest for MSH2 carriers (5.8, 95 % CI 2.6-20.9). Prostate cancer was the first or only diagnosed tumor in 37 % of carriers. MMR gene mutation carriers have at least a twofold or greater increased risk of developing MMR-deficient prostate cancer where the risk is highest for MSH2 mutation carriers. MMR IHC screening of prostate cancers will aid in identifying MMR gene mutation carriers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Re: Microsatellite instability and BRAF mutation testing in colorectal cancer prognostication</a>, Journal of the National Cancer Institute, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Williamson EJ, Clendenning M, Walters RJ, Walsh MD, Win AK, Jenkins MA, Hopper JL, Winship I, Southey MC, Giles GG, English DR, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">106</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/jnci/dju180</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25114271</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/jnci/dju180">http://dx.doi.org/10.1093/jnci/dju180</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal Neoplasms/*diagnosis/*genetics *CpG Islands *DNA Methylation Female Humans Male *Microsatellite Instability *Mutation Proto-Oncogene Proteins B-raf/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Rosty, Christophe Williamson, Elizabeth J Clendenning, Mark Walters, Rhiannon J Walsh, Michael D Win, Aung K Jenkins, Mark A Hopper, John L Winship, Ingrid Southey, Melissa C Giles, Graham G English, Dallas R Buchanan, Daniel D eng Comment Letter Research Support, Non-U.S. Gov't 2014/08/13 06:00 J Natl Cancer Inst. 2014 Aug 11;106(8). pii: dju180. doi: 10.1093/jnci/dju180. Print 2014 Aug.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Should the grading of colorectal adenocarcinoma include microsatellite instability status?</a>, Human pathology, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Williamson EJ, Clendenning M, Walters RJ, Win AK, Jenkins MA, Hopper JL, Winship IM, Southey MC, Giles GG, English DR, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">45</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.humpath.2014.06.020</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25149551</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.humpath.2014.06.020">http://dx.doi.org/10.1016/j.humpath.2014.06.020</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenocarcinoma/mortality/*pathology Adult Aged Colorectal Neoplasms/mortality/*pathology DNA Repair Enzymes/*analysis/biosynthesis Female Humans Immunohistochemistry Kaplan-Meier Estimate Male *Microsatellite Instability Neoplasm Grading/*methods Prognosis Proportional Hazards Models</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Adenocarcinomas of the colon and rectum are graded using a 2-tiered system into histologic low-grade and high-grade tumors based on the proportion of gland formation. The current grading system does not apply to subtypes of carcinomas associated with a high frequency of microsatellite instability (MSI), such as mucinous and medullary carcinomas. We investigated the combined effect of histologic grade and MSI status on survival for 738 patients with colorectal carcinoma (48% female; mean age at diagnosis 68.2 years). The proportion of high-grade adenocarcinoma was 18%. MSI was observed in 59 adenocarcinomas (9%), with higher frequency in high-grade tumors compared with low-grade tumors (20% versus 6%; P < .001). Using Cox regression models, adjusting for sex and age at diagnosis and stratifying by the American Joint Committee on Cancer stage, microsatellite stable (MSS) high-grade tumors were associated with increased hazard of all-cause and colorectal cancer-specific mortality: hazard ratio 2.09 (95% confidence interval [CI], 1.58-2.77) and 2.54 (95% CI, 1.86-3.47), respectively, both P < .001. A new grading system separating adenocarcinoma into low grade (all histologic low grade and MSI high grade) and high grade (MSS histologic high grade) gave a lower Akaike information criterion value when compared with the current grading system and thus represented a better model fit to stratify patients according to survival. We found that patients with a high-grade adenocarcinoma had significantly shorter survival than patients with low-grade adenocarcinoma only if the tumor was MSS, suggesting that the grading of colorectal adenocarcinoma with high-grade histologic features should be made according to the MSI status of the tumor.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>A novel colorectal cancer risk locus at 4q32.2 identified from an international genome-wide association study</a>, Carcinogenesis, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Schmit SL, Schumacher FR, Edlund CK, Conti DV, Raskin L, Lejbkowicz F, Pinchev M, Rennert HS, Jenkins MA, Hopper JL, Buchanan DD, Lindor NM, Le Marchand L, Gallinger S, Haile RW, Newcomb PA, Huang SC, Rennert G, Casey G, Gruber SB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Carcinogenesis</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">35</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/carcin/bgu148</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">25023989</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/carcin/bgu148">http://dx.doi.org/10.1093/carcin/bgu148</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Only a fraction of colorectal cancer heritability is explained by known risk-conferring genetic variation. This study was designed to identify novel risk alleles in Europeans. We conducted a genome-wide association study (GWAS) meta-analysis of colorectal cancer in participants from a population-based case-control study in Israel (n = 1616 cases, 1329 controls) and a consortium study from the Colon Cancer Family Registry (n = 1977 cases, 999 controls). We used a two-stage (discovery-replication) GWAS design, followed by a joint meta-analysis. A combined analysis identified a novel susceptibility locus that reached genome-wide significance on chromosome 4q32.2 [rs35509282, risk allele = A (minor allele frequency = 0.09); odds ratio (OR) per risk allele = 1.53; P value = 8.2 x 10(-9); nearest gene = FSTL5]. The direction of the association was consistent across studies. In addition, we confirmed that 14 of 29 previously identified susceptibility variants were significantly associated with risk of colorectal cancer in this study. Genetic variation on chromosome 4q32.2 is significantly associated with risk of colorectal cancer in Ashkenazi Jews and Europeans in this study.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Characterisation of familial colorectal cancer Type X, Lynch syndrome, and non-familial colorectal cancer</a>, British journal of cancer, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Shiovitz S, Copeland WK, Passarelli MN, Burnett-Hartman AN, Grady WM, Potter JD, Gallinger S, Buchanan DD, Rosty C, Win AK, Jenkins M, Thibodeau SN, Haile R, Baron JA, Marchand LL, Newcomb PA, Lindor NM, Colon Cancer Family,  Registry</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">British journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">111</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/bjc.2014.309</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24918813</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/bjc.2014.309">http://dx.doi.org/10.1038/bjc.2014.309</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Colorectal Neoplasms, Hereditary Nonpolyposis/*epidemiology/pathology Comorbidity Female Humans Logistic Models Male Middle Aged Neoplasms, Cystic, Mucinous, and Serous/*epidemiology/pathology Odds Ratio Questionnaires Registries</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Familial Colorectal Cancer Type X (FCCTX) is defined as individuals with colorectal cancer (CRC) who families meet Amsterdam Criteria-1 (AC1), but whose tumours are DNA-mismatch-repair-proficient, unlike Lynch syndrome (LS). FCCTX does not have an increased risk of extra-colonic cancers. This analysis compares epidemiologic and clinicopathologic features among FCCTX, LS, and 'non-familial' (non-AC1) CRC cases. METHODS: From the Colon Cancer Family Registry, FCCTX (n=173), LS (n=303), and non-AC1 (n=9603) CRC cases were identified. Questionnaire-based epidemiologic information and CRC pathologic features were compared across case groups using polytomous logistic regression. RESULTS: Compared with LS, FCCTX cases were less likely to be current (vs never) smokers; have a proximal subsite (vs rectal) tumour; or have mucinous histology, poor differentiation, or tumour-infiltrating lymphocytes. There were no observed differences in co-morbidities or medication usage. CONCLUSIONS: FCCTX were less likely to be current tobacco users; other exposures were similar between these groups. Histopathologic differences highly suggestive of LS CRCs do not appear to be shared by FCCTX.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Application of a 5-tiered scheme for standardized classification of 2,360 unique mismatch repair gene variants in the InSiGHT locus-specific database</a>, Nature genetics, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Thompson BA, Spurdle AB, Plazzer JP, Greenblatt MS, Akagi K, Al-Mulla F, Bapat B, Bernstein I, Capella G, den Dunnen JT, du Sart D, Fabre A, Farrell MP, Farrington SM, Frayling IM, Frebourg T, Goldgar DE, Heinen CD, Holinski-Feder E, Kohonen-Corish M, Robinson KL, Leung SY, Martins A, Moller P, Morak M, Nystrom M, Peltomaki P, Pineda M, Qi M, Ramesar R, Rasmussen LJ, Royer-Pokora B, Scott RJ, Sijmons R, Tavtigian SV, Tops CM, Weber T, Wijnen J, Woods MO, Macrae F, Genuardi M, InSiGht</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Nature genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">46</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/ng.2854</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24362816</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/ng.2854">http://dx.doi.org/10.1038/ng.2854</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Classification/*methods DNA Mismatch Repair/*genetics *Databases, Genetic Disease Management Gastrointestinal Neoplasms/*genetics Genetic Variation/*genetics Humans</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The clinical classification of hereditary sequence variants identified in disease-related genes directly affects clinical management of patients and their relatives. The International Society for Gastrointestinal Hereditary Tumours (InSiGHT) undertook a collaborative effort to develop, test and apply a standardized classification scheme to constitutional variants in the Lynch syndrome-associated genes MLH1, MSH2, MSH6 and PMS2. Unpublished data submission was encouraged to assist in variant classification and was recognized through microattribution. The scheme was refined by multidisciplinary expert committee review of the clinical and functional data available for variants, applied to 2,360 sequence alterations, and disseminated online. Assessment using validated criteria altered classifications for 66% of 12,006 database entries. Clinical recommendations based on transparent evaluation are now possible for 1,370 variants that were not obviously protein truncating from nomenclature. This large-scale endeavor will facilitate the consistent management of families suspected to have Lynch syndrome and demonstrates the value of multidisciplinary collaboration in the curation and classification of variants in public locus-specific databases.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis</a>, Human molecular genetics, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Whiffin N, Hosking FJ, Farrington SM, Palles C, Dobbins SE, Zgaga L, Lloyd A, Kinnersley B, Gorman M, Tenesa A, Broderick P, Wang Y, Barclay E, Hayward C, Martin L, Buchanan DD, Win AK, Hopper J, Jenkins M, Lindor NM, Newcomb PA, Gallinger S, Conti D, Schumacher F, Casey G, Liu T, Swedish Low-Risk Colorectal Cancer Study,  Group, Campbell H, Lindblom A, Houlston RS, Tomlinson IP, Dunlop MG</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human molecular genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">23</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">17</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/hmg/ddu177</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24737748</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/hmg/ddu177">http://dx.doi.org/10.1093/hmg/ddu177</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">To identify common variants influencing colorectal cancer (CRC) risk, we performed a meta-analysis of five genome-wide association studies, comprising 5626 cases and 7817 controls of European descent. We conducted replication of top ranked single nucleotide polymorphisms (SNPs) in additional series totalling 14 037 cases and 15 937 controls, identifying a new CRC risk locus at 10q24.2 [rs1035209; odds ratio (OR) = 1.13, P = 4.54 x 10(-11)]. We also performed meta-analysis of our studies, with previously published data, of several recently purported CRC risk loci. We failed to find convincing evidence for a previously reported genome-wide association at rs11903757 (2q32.3). Of the three additional loci for which evidence of an association in Europeans has been previously described we failed to show an association between rs59336 (12q24.21) and CRC risk. However, for the other two SNPs, our analyses demonstrated new, formally significant associations with CRC. These are rs3217810 intronic in CCND2 (12p13.32; OR = 1.19, P = 2.16 x 10(-10)) and rs10911251 near LAMC1 (1q25.3; OR = 1.09, P = 1.75 x 10(-8)). Additionally, we found some evidence to support a relationship between, rs647161, rs2423297 and rs10774214 and CRC risk originally identified in East Asians in our European datasets. Our findings provide further insights into the genetic and biological basis of inherited genetic susceptibility to CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk of colorectal cancer for carriers of mutations in MUTYH, with and without a family history of cancer</a>, Gastroenterology, 2014.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Dowty JG, Cleary SP, Kim H, Buchanan DD, Young JP, Clendenning M, Rosty C, MacInnis RJ, Giles GG, Boussioutas A, Macrae FA, Parry S, Goldblatt J, Baron JA, Burnett T, Le Marchand L, Newcomb PA, Haile RW, Hopper JL, Cotterchio M, Gallinger S, Lindor NM, Tucker KM, Winship IM, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gastroenterology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">146</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2014</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1053/j.gastro.2014.01.022</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24444654</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1053/j.gastro.2014.01.022">http://dx.doi.org/10.1053/j.gastro.2014.01.022</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Aged, 80 and over Australia Colorectal Neoplasms/*genetics DNA Glycosylases/*genetics Female Genetic Predisposition to Disease Heredity Heterozygote Humans Male Middle Aged *Mutation New Zealand North America Pedigree Phenotype Registries Risk Assessment Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">We studied 2332 individuals with monoallelic mutations in MUTYH among 9504 relatives of 264 colorectal cancer (CRC) cases with a MUTYH mutation. We estimated CRC risks through 70 years of age of 7.2% for male carriers of monoallelic mutations (95% confidence interval [CI], 4.6%-11.3%) and 5.6% for female carriers of monoallelic mutations (95% CI, 3.6%-8.8%), irrespective of family history. For monoallelic MUTYH mutation carriers with a first-degree relative with CRC diagnosed by 50 years of age who does not have the MUTYH mutation, risks of CRC were 12.5% for men (95% CI, 8.6%-17.7%) and 10% for women (95% CI, 6.7%-14.4%). Risks of CRC for carriers of monoallelic mutations in MUTYH with a first-degree relative with CRC are sufficiently high to warrant more intensive screening than for the general population.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2013</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Germline HOXB13 p.Gly84Glu mutation and risk of colorectal cancer</a>, Cancer epidemiology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Akbari MR, Anderson LN, Buchanan DD, Clendenning M, Jenkins MA, Win AK, Hopper JL, Giles GG, Nam R, Narod S, Gallinger S, Cleary SP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">37</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.canep.2013.03.003</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23541221</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.canep.2013.03.003">http://dx.doi.org/10.1016/j.canep.2013.03.003</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Aged Aged, 80 and over Australia Canada Case-Control Studies Colorectal Neoplasms/*genetics Female *Genetic Predisposition to Disease Genotype Germ-Line Mutation Homeodomain Proteins/*genetics Humans Male Middle Aged Registries Risk Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">INTRODUCTION: The HOXB13 pGly84Glu mutation has recently been associated with an increased risk of prostate cancer but the association of other cancer sites with this allele has not been assessed. Data has suggested that HOXB13 expression levels are decreased in colorectal cancer (CRC) cell lines indicating this gene may be involved in colorectal tumourigenesis. METHODS: To evaluate a potential association of this mutation with CRC, we genotyped the mutation in 2695 CRC cases and 4593 controls from population-based registries in Canada and Australia. RESULTS: The HOXB13 pGly84Glu mutation was more common in CRC cases than controls (0.48% vs. 0.17%, P=0.02) indicating a significant association between the HOXB13 variant and CRC risk (OR=2.8; 95%CI: 1.2-6.8). This association was attenuated but remained significant with the inclusion of previously published and publicly available genotype data. Pedigree analysis of cases and controls revealed that 7/21 HOXB13 mutation carriers had a family history of prostate cancer. DISCUSSION: This report is the first to suggest a risk of CRC associated with mutations in the HOXB13 gene. These findings require further validation but may be of importance in the screening and genetic counseling of families known to carry the HOXB13 pGly84Glu mutation.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Family history of colorectal cancer in BRAF p.V600E-mutated colorectal cancer cases</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Win AK, Walsh MD, Walters RJ, Clendenning M, Nagler B, Pearson SA, Macrae FA, Parry S, Arnold J, Winship I, Giles GG, Lindor NM, Potter JD, Hopper JL, Rosty C, Young JP, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">22</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-12-1211</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23462926</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-12-1211">http://dx.doi.org/10.1158/1055-9965.EPI-12-1211</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal Neoplasms/*genetics/pathology DNA Mismatch Repair Family Health Female Humans Male Middle Aged Proto-Oncogene Proteins B-raf/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Previous reports suggest that relatives of colorectal cancer (CRC)-affected probands carrying the BRAF p.V600E mutation are at an increased risk of CRC and extracolonic cancers (ECC). In this study, we estimated the association between a family history of either CRC or ECC and risk of CRC with a BRAF p.V600E mutation. METHODS: Population-based CRC cases (probands, ages 18-59 years at diagnosis), recruited irrespective of family cancer history, were characterized for BRAF p.V600E mutation and mismatch repair (MMR) status. ORs and 95% confidence intervals (CI) were estimated using multivariable logistic regression. RESULTS: The 690 eligible probands showed a mean age at CRC diagnosis of 46.9 +/- 7.8 years, with 313 (47.9%) reporting a family history of CRC and 53 (7.7%) that were BRAF-mutated. Probands with BRAF-mutated, MMR-proficient CRCs were less likely to have a family history of CRC than probands that were BRAF wild-type (OR, 0.46; 95% CI, 0.24-0.91; P = 0.03). For probands with a BRAF-mutated CRC, the mean age at diagnosis was greater for those with a CRC-affected first- or second-degree relative (49.3 +/- 6.4 years) compared with those without a family history (43.8 +/- 10.2 years; P = 0.04). The older the age at diagnosis of CRC with the BRAF p.V600E mutation, the more likely these probands were to show a family history of CRC (OR, 1.09 per year of age; 95% CI, 1.00-1.18; P = 0.04). CONCLUSIONS: Probands with early-onset, BRAF-mutated, and MMR-proficient CRC were less likely to have a family history of CRC than probands that were BRAF-wild-type. IMPACT: These findings provide useful insights for cancer risk assessment in families and suggest that familial or inherited factors are more important in early-onset, BRAF-wild-type CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Absence of PMS2 mutations in colon-CFR participants whose colorectal cancers demonstrate unexplained loss of MLH1 expression</a>, Clinical genetics, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Clendenning M, Macrae FA, Walsh MD, Walters RJ, Thibodeau SN, Gunawardena SR, Potter JD, Haile RW, Gallinger S, Colorectal Cancer Family,  Registry, Hopper JL, Jenkins MA, Rosty C, Young JP, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Clinical genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">83</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1111/cge.12011</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23017166</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1111/cge.12011">http://dx.doi.org/10.1111/cge.12011</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics/metabolism Adenosine Triphosphatases/*genetics/metabolism Chromatography, High Pressure Liquid/methods Cohort Studies Colorectal Neoplasms/*genetics/metabolism DNA Mutational Analysis/methods DNA Repair Enzymes/*genetics/metabolism DNA-Binding Proteins/*genetics/metabolism Humans Immunohistochemistry *Mutation Nuclear Proteins/*genetics/metabolism Polymorphism, Genetic Registries</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Clendenning, M Macrae, F A Walsh, M D Walters, R J Thibodeau, S N Gunawardena, S R Potter, J D Haile, R W Gallinger, S Hopper, J L Jenkins, M A Rosty, C Young, J P Buchanan, D D eng CA-95-011/CA/NCI NIH HHS/ Letter Research Support, N.I.H., Extramural Research Support, Non-U.S. Gov't Denmark 2012/09/29 06:00 Clin Genet. 2013 Jun;83(6):591-3. doi: 10.1111/cge.12011. Epub 2012 Sep 27.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Detection of large scale 3' deletions in the PMS2 gene amongst Colon-CFR participants: have we been missing anything?</a>, Familial cancer, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Clendenning M, Walsh MD, Gelpi JB, Thibodeau SN, Lindor N, Potter JD, Newcomb P, LeMarchand L, Haile R, Gallinger S, Colorectal Cancer Family,  Registry, Hopper JL, Jenkins MA, Rosty C, Young JP, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-012-9597-4</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23288611</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-012-9597-4">http://dx.doi.org/10.1007/s10689-012-9597-4</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Current screening practices have been able to identify PMS2 mutations in 78 % of cases of colorectal cancer from the Colorectal Cancer Family Registry (Colon CFR) which showed solitary loss of the PMS2 protein. However the detection of large-scale deletions in the 3' end of the PMS2 gene has not been possible due to technical difficulties associated with pseudogene sequences. Here, we utilised a recently described MLPA/long-range PCR-based approach to screen the remaining 22 % (n = 16) of CRC-affected probands for mutations in the 3' end of the PMS2 gene. No deletions encompassing any or all of exons 12 through 15 were identified; therefore, our results suggest that 3' deletions in PMS2 are not a frequent occurrence in such families.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Germline Mutations in the Polyposis-Associated Genes are not Common in Individuals with Serrated Polyposis Syndrome</a>, PloS one, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Clendenning M, Young JP, Walsh MD, Woodall S, Arnold J, Jenkins M, Win AK, Hopper JL, Sweet K, Gallinger S, Rosty C, Parry S, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/journal.pone.0066705</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23805267</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pone.0066705">http://dx.doi.org/10.1371/journal.pone.0066705</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Recent reports have observed that individuals with serrated polyps, some of whom meet the clinical diagnostic criteria for Serrated Polyposis Syndrome (SPS), are among those who carry germline mutations in genes associated with polyposis syndromes including; (1) genes known to underlie hamartomatous polyposes (SMAD4, BMPR1A, and PTEN), (2) MUTYH-associated polyposis and (3) GREM1 in Hereditary Mixed Polyposis Syndrome (HMPS). The aim of this study was to characterise individuals fulfilling the current WHO criteria for SPS for germline mutations in these polyposis-associated genes. METHODS: A total of 65 individuals with SPS (fulfilling WHO criteria 1 or 3), were recruited to the Genetics of Serrated Neoplasia study between 2000 and 2012, through multiple Genetics or Family Cancer Clinics within Australia, or from the New Zealand Familial Gastrointestinal Cancer Service. Individuals with SPS were tested for coding mutations and large deletions in the PTEN, SMAD4, and BMPR1A genes, for the MUTYH variants in exons 7 (Y179C) and 13 (G396D), and for the duplication upstream of GREM1. RESULTS: We found no variants that were likely to be deleterious germline mutations in the SPS cases in the PTEN, SMAD4, and BMPR1A genes. A novel variant in intron 2 (c.164+223T>C) of PTEN was identified in one individual and was predicted by in silico analysis to have no functional consequences. One further individual with SPS was found to be mono-allelic for the MUTYH G396D mutation. No individuals carried the recently reported duplication within GREM1. CONCLUSIONS: Genes involved in the gastrointestinal hamartomatous polyposis, Hereditary Mixed Polyposis Syndrome and MUTYH-associated polyposis syndromes are not commonly altered in individuals with SPS.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Telomere length varies by DNA extraction method: implications for epidemiologic research</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Cunningham JM, Johnson RA, Litzelman K, Skinner HG, Seo S, Engelman CD, Vanderboom RJ, Kimmel GW, Gangnon RE, Riegert-Johnson DL, Baron JA, Potter JD, Haile R, Buchanan DD, Jenkins MA, Rider DN, Thibodeau SN, Petersen GM, Boardman LA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">22</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-13-0409</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24019396</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-13-0409">http://dx.doi.org/10.1158/1055-9965.EPI-13-0409</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Both shorter and longer telomeres in peripheral blood leukocyte (PBL) DNA have been associated with cancer risk. However, associations remain inconsistent across studies of the same cancer type. This study compares DNA preparation methods to determine telomere length from patients with colorectal cancer. METHODS: We examined PBL relative telomere length (RTL) measured by quantitative PCR (qPCR) in 1,033 patients with colorectal cancer and 2,952 healthy controls. DNA was extracted with phenol/chloroform, PureGene, or QIAamp. RESULTS: We observed differences in RTL depending on DNA extraction method (P < 0.001). Phenol/chloroform-extracted DNA had a mean RTL (T/S ratio) of 0.78 (range 0.01-6.54) compared with PureGene-extracted DNA (mean RTL of 0.75; range 0.00-12.33). DNA extracted by QIAamp yielded a mean RTL of 0.38 (range 0.02-3.69). We subsequently compared RTL measured by qPCR from an independent set of 20 colorectal cancer cases and 24 normal controls in PBL DNA extracted by each of the three extraction methods. The range of RTL measured by qPCR from QIAamp-extracted DNA (0.17-0.58) was less than from either PureGene or phenol/chloroform (ranges, 0.04-2.67 and 0.32-2.81, respectively). CONCLUSIONS: RTL measured by qPCR from QIAamp-extracted DNA was less than from either PureGene or phenol/chloroform (P < 0.001). IMPACT: Differences in DNA extraction method may contribute to the discrepancies between studies seeking to find an association between the risk of cancer or other diseases and RTL.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Identification of novel variants in colorectal cancer families by high-throughput exome sequencing</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">DeRycke MS, Gunawardena SR, Middha S, Asmann YW, Schaid DJ, McDonnell SK, Riska SM, Eckloff BW, Cunningham JM, Fridley BL, Serie DJ, Bamlet WR, Cicek MS, Jenkins MA, Duggan DJ, Buchanan D, Clendenning M, Haile RW, Woods MO, Gallinger SN, Casey G, Potter JD, Newcomb PA, Le Marchand L, Lindor NM, Thibodeau SN, Goode EL</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">22</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-12-1226</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23637064</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-12-1226">http://dx.doi.org/10.1158/1055-9965.EPI-12-1226</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Aged, 80 and over Colorectal Neoplasms/*genetics Computational Biology *Exome Female Genome-Wide Association Study High-Throughput Nucleotide Sequencing/*methods Humans Male Middle Aged Polymorphism, Single Nucleotide Registries</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Colorectal cancer (CRC) in densely affected families without Lynch Syndrome may be due to mutations in undiscovered genetic loci. Familial linkage analyses have yielded disparate results; the use of exome sequencing in coding regions may identify novel segregating variants. METHODS: We completed exome sequencing on 40 affected cases from 16 multicase pedigrees to identify novel loci. Variants shared among all sequenced cases within each family were identified and filtered to exclude common variants and single-nucleotide variants (SNV) predicted to be benign. RESULTS: We identified 32 nonsense or splice-site SNVs, 375 missense SNVs, 1,394 synonymous or noncoding SNVs, and 50 indels in the 16 families. Of particular interest are two validated and replicated missense variants in CENPE and KIF23, which are both located within previously reported CRC linkage regions, on chromosomes 1 and 15, respectively. CONCLUSIONS: Whole-exome sequencing identified DNA variants in multiple genes. Additional sequencing of these genes in additional samples will further elucidate the role of variants in these regions in CRC susceptibility. IMPACT: Exome sequencing of familial CRC cases can identify novel rare variants that may influence disease risk.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Cancer risks for MLH1 and MSH2 mutation carriers</a>, Human mutation, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Dowty JG, Win AK, Buchanan DD, Lindor NM, Macrae FA, Clendenning M, Antill YC, Thibodeau SN, Casey G, Gallinger S, Marchand LL, Newcomb PA, Haile RW, Young GP, James PA, Giles GG, Gunawardena SR, Leggett BA, Gattas M, Boussioutas A, Ahnen DJ, Baron JA, Parry S, Goldblatt J, Young JP, Hopper JL, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human mutation</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">34</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/humu.22262</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23255516</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/humu.22262">http://dx.doi.org/10.1002/humu.22262</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics Adult Age Factors Aged Aged, 80 and over Carrier Proteins/genetics/metabolism Colorectal Neoplasms/*genetics Colorectal Neoplasms, Hereditary Nonpolyposis/*genetics Endometrial Neoplasms/*genetics Female *Germ-Line Mutation Heterozygote Humans Male Middle Aged MutS Homolog 2 Protein/*genetics Nuclear Proteins/*genetics Penetrance Questionnaires Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">We studied 17,576 members of 166 MLH1 and 224 MSH2 mutation-carrying families from the Colon Cancer Family Registry. Average cumulative risks of colorectal cancer (CRC), endometrial cancer (EC), and other cancers for carriers were estimated using modified segregation analysis conditioned on ascertainment criteria. Heterogeneity in risks was investigated using a polygenic risk modifier. Average CRC cumulative risks at the age of 70 years (95% confidence intervals) for MLH1 and MSH2 mutation carriers, respectively, were estimated to be 34% (25%-50%) and 47% (36%-60%) for male carriers and 36% (25%-51%) and 37% (27%-50%) for female carriers. Corresponding EC risks were 18% (9.1%-34%) and 30% (18%-45%). A high level of CRC risk heterogeneity was observed (P < 0.001), with cumulative risks at the age of 70 years estimated to follow U-shaped distributions. For example, 17% of male MSH2 mutation carriers have estimated lifetime risks of 0%-10% and 18% have risks of 90%-100%. Therefore, average risks are similar for the two genes but there is so much individual variation about the average that large proportions of carriers have either very low or very high lifetime cancer risks. Our estimates of CRC and EC cumulative risks for MLH1 and MSH2 mutation carriers are the most precise currently available.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>KRAS-mutation status in relation to colorectal cancer survival: the joint impact of correlated tumour markers</a>, British journal of cancer, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Phipps AI, Buchanan DD, Makar KW, Win AK, Baron JA, Lindor NM, Potter JD, Newcomb PA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">British journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">108</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/bjc.2013.118</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23511557</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/bjc.2013.118">http://dx.doi.org/10.1038/bjc.2013.118</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Colorectal Neoplasms/epidemiology/*genetics/mortality Female *Genes, ras Humans Male Middle Aged *Mutation SEER Program Survival Rate Washington/epidemiology Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Mutations in the Kirsten Ras (KRAS) oncogene are common in colorectal cancer (CRC). The role of KRAS-mutation status as a prognostic factor, however, is unclear. We evaluated the relationship between KRAS-mutation status and CRC survival, considering heterogeneity in this association by tumour and patient characteristics. METHODS: The population-based study included individuals diagnosed with CRC between 1998-2007 in Western Washington State. Tumour specimens were tested for KRAS exon 2 mutations, the BRAF p.V600E mutation, and microsatellite instability (MSI). We used Cox regression to estimate hazard ratios (HR) and 95% confidence intervals (CI) for the association between KRAS-mutation status and disease-specific and overall survival. Stratified analyses were conducted by age, sex, tumour site, stage, and MSI. We conducted additional analyses combining KRAS-mutation, BRAF-mutation, and MSI status. RESULTS: Among 1989 cases, 31% had KRAS-mutated CRC. Kirsten Ras (KRAS)-mutated CRC was associated with poorer disease-specific survival (HR=1.37, 95% CI: 1.13-1.66). This association was not evident in cases who presented with distant-stage CRC. Cases with KRAS-wild-type/BRAF-wild-type/MSI-high CRC had the most favourable prognosis; those with CRC exhibiting a KRAS- or BRAF-mutation and no MSI had the poorest prognosis. Patterns were similar for overall survival. CONCLUSION: Kirsten Ras (KRAS)-mutated CRC was associated with statistically significantly poorer survival after diagnosis than KRAS-wild-type CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Multiplicity and molecular heterogeneity of colorectal carcinomas in individuals with serrated polyposis</a>, The American journal of surgical pathology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Walsh MD, Walters RJ, Clendenning M, Pearson SA, Jenkins MA, Win AK, Hopper JL, Sweet K, Frankel WL, Aronson M, Gallinger S, Goldblatt J, Tucker K, Greening S, Gattas MR, Woodall S, Arnold J, Walker NI, Parry S, Young JP, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The American journal of surgical pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">37</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1097/PAS.0b013e318270f748</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23211288</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1097/PAS.0b013e318270f748">http://dx.doi.org/10.1097/PAS.0b013e318270f748</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenocarcinoma/*complications/genetics/pathology Adolescent Adult Aged Colonic Polyps/*complications/genetics/pathology Colorectal Neoplasms/*complications/genetics/pathology Female Humans Immunohistochemistry Male Middle Aged Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Serrated polyposis (SP) is a clinically defined syndrome characterized by the occurrence of multiple serrated polyps in the large intestine. Individuals with SP and their relatives are at increased risk of colorectal carcinoma (CRC). We aimed to determine the pathologic and molecular profiles of CRCs in individuals fulfilling World Health Organization criteria for SP. A total of 45 CRCs were obtained from 38 individuals with SP (27 female and 11 male patients; median age at CRC diagnosis, 58.5 y) attending genetics clinics. Tumor samples were pathologically reviewed, screened for somatic BRAF and KRAS mutations, and analyzed immunohistochemically for mismatch repair protein (MMR) expression. Tumors were spread throughout the large intestine, with 64% located in the proximal colon. Mutations in BRAF and KRAS and immunohistochemical evidence of MMR deficiency were found in 46%, 5%, and 38%, respectively. Nearly half of CRCs were BRAF/KRAS wild type, and these were associated with distal location (63%) and MMR proficiency (84%). Overexpression of p53 and/or evidence of beta-catenin activation were identified in 13 CRCs. Ten patients (26%) had synchronous or metachronous CRCs. In conclusion, the majority of CRCs arising in individuals with SP do not harbor molecular hallmarks of serrated pathway CRCs but show a diverse range of molecular profiles. The high proportion of multiple CRCs suggests that individuals with SP would benefit from frequent colonoscopic surveillance and from a consideration of a more extensive colectomy at the time of CRC diagnosis.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>PIK3CA activating mutation in colorectal carcinoma: associations with molecular features and survival</a>, PloS one, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Young JP, Walsh MD, Clendenning M, Sanderson K, Walters RJ, Parry S, Jenkins MA, Win AK, Southey MC, Hopper JL, Giles GG, Williamson EJ, English DR, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/journal.pone.0065479</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23785428</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pone.0065479">http://dx.doi.org/10.1371/journal.pone.0065479</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Aged, 80 and over Colorectal Neoplasms/*genetics/*mortality/pathology CpG Islands DNA Methylation DNA Modification Methylases/genetics DNA Repair Enzymes/genetics Exons Female Follow-Up Studies Humans Male Middle Aged *Mutation Phosphatidylinositol 3-Kinases/*genetics Prospective Studies Proto-Oncogene Proteins/genetics Proto-Oncogene Proteins B-raf/genetics *Transcriptional Activation Tumor Suppressor Proteins/genetics ras Proteins/genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Mutations in PIK3CA are present in 10 to 15% of colorectal carcinomas. We aimed to examine how PIK3CA mutations relate to other molecular alterations in colorectal carcinoma, to pathologic phenotype and survival. PIK3CA mutation testing was carried out using direct sequencing on 757 incident tumors from the Melbourne Collaborative Cohort Study. The status of O-6-methylguanine-DNA methyltransferase (MGMT) was assessed using both immunohistochemistry and methyLight techniques. Microsatellite instability, CpG island phenotype (CIMP), KRAS and BRAF V600E mutation status, and pathology review features were derived from previous reports. PIK3CA mutation was observed in 105 of 757 (14%) of carcinomas, characterized by location in the proximal colon (54% vs. 34%; P<0.001) and an increased frequency of KRAS mutation (48% vs. 25%; P<0.001). High-levels of CIMP were more frequently found in PIK3CA-mutated tumors compared with PIK3CA wild-type tumors (22% vs. 11%; P = 0.004). There was no difference in the prevalence of BRAF V600E mutation between these two tumor groups. PIK3CA-mutated tumors were associated with loss of MGMT expression (35% vs. 20%; P = 0.001) and the presence of tumor mucinous differentiation (54% vs. 32%; P<0.001). In patients with wild-type BRAF tumors, PIK3CA mutation was associated with poor survival (HR 1.51 95% CI 1.04-2.19, P = 0.03). In summary, PIK3CA-mutated colorectal carcinomas are more likely to develop in the proximal colon, to demonstrate high levels of CIMP, KRAS mutation and loss of MGMT expression. PIK3CA mutation also contributes to significantly decreased survival for patients with wild-type BRAF tumors.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Colorectal carcinomas with KRAS mutation are associated with distinctive morphological and molecular features</a>, Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Young JP, Walsh MD, Clendenning M, Walters RJ, Pearson S, Pavluk E, Nagler B, Pakenas D, Jass JR, Jenkins MA, Win AK, Southey MC, Parry S, Hopper JL, Giles GG, Williamson E, English DR, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">26</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/modpathol.2012.240</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23348904</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/modpathol.2012.240">http://dx.doi.org/10.1038/modpathol.2012.240</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Carcinoma/*genetics/mortality/*pathology Cell Differentiation Chi-Square Distribution Colonic Polyps/*genetics/mortality/*pathology Colorectal Neoplasms/chemistry/*genetics/mortality/*pathology DNA Methylation DNA Modification Methylases/analysis/genetics DNA Mutational Analysis DNA Repair Enzymes/analysis/genetics Female Genetic Predisposition to Disease Humans Immunohistochemistry Kaplan-Meier Estimate Male Microsatellite Instability Middle Aged *Mutation Phenotype Prognosis Proportional Hazards Models Prospective Studies Proto-Oncogene Proteins/*genetics Proto-Oncogene Proteins B-raf/genetics Risk Factors Time Factors Tumor Markers, Biological/analysis/genetics Tumor Suppressor Proteins/analysis/genetics Victoria ras Proteins/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">KRAS-mutated carcinomas comprise 35-40% of all colorectal carcinomas but little is known about their characteristics. The aim of this study was to examine the pathological and molecular features of KRAS-mutated colorectal carcinomas and to compare them with other carcinoma subgroups. KRAS mutation testing was performed in 776 incident tumors from the Melbourne Collaborative Cohort Study. O(6)-methylguanine DNA methyltransferase (MGMT) status was assessed using both immunohistochemistry and MethyLight techniques. Microsatellite instability (MSI) phenotype and BRAF V600E mutation status were derived from earlier studies. Mutation in KRAS codon 12 or codon 13 was present in 28% of colorectal carcinomas. Compared with KRAS wild-type carcinomas, KRAS-mutated carcinomas were more frequently observed in contiguity with a residual polyp (38 vs 21%; P<0.001), demonstrated mucinous differentiation (46 vs 31%; P=0.001) and were associated with different MSI status (P<0.001) and with MGMT methylation (47 vs 21%; P=0.001). Compared with tumors demonstrating neither BRAF nor KRAS mutation, KRAS-mutated carcinomas showed more frequent location in the proximal colon (41 vs 27%; P=0.001), mucinous differentiation (46 vs 25%; P<0.001), presence of a contiguous polyp (38 vs 22%; P<0.001), MGMT methylation (47 vs 26%; P=0.01) and loss of MGMT immunohistochemical expression (27 vs 19%; P=0.02). KRAS-mutated carcinomas were distributed in a bimodal pattern along the proximal-distal axis of the colorectum. Compared with male subjects, female subjects were more likely to have KRAS-mutated carcinoma in the transverse colon and descending colon (39 vs 15%; P=0.02). No difference in overall survival was observed in patients according to their tumor KRAS mutation status. In summary, KRAS-mutated carcinomas frequently develop in contiguity with a residual polyp and show molecular features distinct from other colorectal carcinomas, in particular from tumors with neither BRAF nor KRAS mutation.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Immunophenotypic analysis of ovarian endometrioid adenocarcinoma: correlation with KRAS mutation and the presence of endometriosis</a>, Pathology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Stewart CJ, Walsh MD, Budgeon CA, Crook ML, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">45</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1097/PAT.0b013e3283650ad7</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">24018808</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1097/PAT.0b013e3283650ad7">http://dx.doi.org/10.1097/PAT.0b013e3283650ad7</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">AIMS: The relationship between endometriosis and ovarian endometrioid adenocarcinoma (OEC) is well recognised but it is unclear whether endometriosis positive and negative OECs develop via similar pathogenetic mechanisms. MATERIALS: Sixty-seven low grade OECs (35 associated with endometriosis) were stained immunohistochemically for beta-catenin, cyclin D1, BAF250a, PTEN, p53, WT1 and the mismatch repair (MMR) proteins MLH1, PMS2, MSH2 and MSH6. The results were correlated with KRAS mutation analysis and the presence of concurrent endometriosis. RESULTS: Abnormal beta-catenin, cyclin D1, BAF250a, PTEN, p53 and MMR protein expression was identified in 61.2%, 50.7%, 19.4%, 23.9%, 9.0%, and 6.0% of cases, respectively; these changes were equally common in endometriosis positive and negative tumours. WT1 expression was restricted to endometriosis negative EOC (8/32, 25%) and four WT1 positive cases showed sertoliform/spindle cell histological patterns. Abnormal beta-catenin expression correlated with cyclin D1 overexpression but was inversely related to KRAS mutation. Immunophenotypic abnormalities were present in four of 17 histologically benign endometriotic lesions. CONCLUSIONS: Most immunophenotypic alterations were equally common in endometriosis associated and independent OECs but only the latter were associated with abnormal WT1 expression. The inverse relationship between abnormal beta-catenin expression and KRAS mutation merits further study. Histologically benign endometriotic epithelium may show immunophenotypic abnormalities similar to those present in associated carcinomas.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Improving identification of lynch syndrome patients: a comparison of research data with clinical records</a>, International journal of cancer. Journal international du cancer, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Tan YY, McGaughran J, Ferguson K, Walsh MD, Buchanan DD, Young JP, Webb PM, Obermair A, Spurdle AB, Group,  Anecs</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">132</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.27978</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23225370</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.27978">http://dx.doi.org/10.1002/ijc.27978</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Aged Colorectal Neoplasms, Hereditary Nonpolyposis/*diagnosis/*genetics DNA Mismatch Repair Endometrial Neoplasms/genetics Female Germ-Line Mutation Humans Medical Records Middle Aged Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Current evidence suggests poor identification and referral of Lynch syndrome patients. This study evaluated the strategies by which patients with endometrial cancer were referred to genetics services. Data from clinic-based patients with endometrial cancer enrolled through the Australian National Endometrial Cancer population-based research study with detailed family history information were analyzed. The Amsterdam II criteria, the revised Bethesda guidelines, and criteria adapted for this study was assessed using personal/family history information. The percentages of patients referred and who could have been referred to genetics services, and the performance of each criterion for identifying possible mismatch-repair (MMR) gene mutation carriers, based on tumor MMR immunohistochemistry (IHC), were determined. Research data indicated that 236/397(59%) of patients with endometrial cancer had family/personal history of cancer, including 14 (4%) who fulfilled Amsterdam II criteria. Family history information was noted in the hospital records for only 61(15%) patients, including 7/14 (50%) of patients meeting Amsterdam criteria, and always less extensively than that recorded in the research setting. Only 13 patients (two meeting Amsterdam criteria) were referred for genetic assessment. Of 58 patients with tumor MMR protein-IHC loss, the Amsterdam criteria and Bethesda guidelines identified only three and 34% of these possible germline mutation carriers, respectively. Greater sensitivity (60%) was obtained using a single criterion proposed by our study, >/=2 first-degree or second-degree relatives reporting Lynch cancers. Hospital records indicate poor recognition of family history. Application of research methods show improved identification and may facilitate appropriate referrals of endometrial cancer patients with possible Lynch syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>A multifactorial likelihood model for MMR gene variant classification incorporating probabilities based on sequence bioinformatics and tumor characteristics: a report from the Colon Cancer Family Registry</a>, Human mutation, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Thompson BA, Goldgar DE, Paterson C, Clendenning M, Walters R, Arnold S, Parsons MT, Michael DW, Gallinger S, Haile RW, Hopper JL, Jenkins MA, Lemarchand L, Lindor NM, Newcomb PA, Thibodeau SN, Colon Cancer Family,  Registry, Young JP, Buchanan DD, Tavtigian SV, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human mutation</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">34</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/humu.22213</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22949379</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/humu.22213">http://dx.doi.org/10.1002/humu.22213</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/genetics Alternative Splicing/genetics Colonic Neoplasms/*genetics Computational Biology/classification/*methods/statistics & numerical data DNA Mismatch Repair/*genetics DNA Mutational Analysis/methods/statistics & numerical data DNA-Binding Proteins/genetics Family Health Humans Likelihood Functions Microsatellite Instability Microsatellite Repeats/genetics MutS Homolog 2 Protein/genetics *Mutation Nuclear Proteins/genetics Proto-Oncogene Proteins B-raf/genetics Registries/classification/statistics & numerical data</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Mismatch repair (MMR) gene sequence variants of uncertain clinical significance are often identified in suspected Lynch syndrome families, and this constitutes a challenge for both researchers and clinicians. Multifactorial likelihood model approaches provide a quantitative measure of MMR variant pathogenicity, but first require input of likelihood ratios (LRs) for different MMR variation-associated characteristics from appropriate, well-characterized reference datasets. Microsatellite instability (MSI) and somatic BRAF tumor data for unselected colorectal cancer probands of known pathogenic variant status were used to derive LRs for tumor characteristics using the Colon Cancer Family Registry (CFR) resource. These tumor LRs were combined with variant segregation within families, and estimates of prior probability of pathogenicity based on sequence conservation and position, to analyze 44 unclassified variants identified initially in Australasian Colon CFR families. In addition, in vitro splicing analyses were conducted on the subset of variants based on bioinformatic splicing predictions. The LR in favor of pathogenicity was estimated to be ~12-fold for a colorectal tumor with a BRAF mutation-negative MSI-H phenotype. For 31 of the 44 variants, the posterior probabilities of pathogenicity were such that altered clinical management would be indicated. Our findings provide a working multifactorial likelihood model for classification that carefully considers mode of ascertainment for gene testing.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>BRAFV600E immunohistochemistry facilitates universal screening of colorectal cancers for Lynch syndrome</a>, The American journal of surgical pathology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Toon CW, Walsh MD, Chou A, Capper D, Clarkson A, Sioson L, Clarke S, Mead S, Walters RJ, Clendenning M, Rosty C, Young JP, Win AK, Hopper JL, Crook A, von Deimling A, Jenkins MA, Buchanan DD, Gill AJ</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The American journal of surgical pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">37</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1097/PAS.0b013e31828f233d</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23797718</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1097/PAS.0b013e31828f233d">http://dx.doi.org/10.1097/PAS.0b013e31828f233d</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">*Algorithms Colorectal Neoplasms, Hereditary Nonpolyposis/*diagnosis/*genetics/metabolism DNA Mismatch Repair Female Humans Immunohistochemistry Male Mass Screening/*methods Microsatellite Instability Multiplex Polymerase Chain Reaction Mutation Proto-Oncogene Proteins B-raf/*genetics Spectrometry, Mass, Matrix-Assisted Laser Desorption-Ionization Tissue Array Analysis</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BRAFV600E mutation in microsatellite-unstable (MSI) colorectal carcinomas (CRCs) virtually excludes Lynch syndrome (LS). In microsatellite-stable (MSS) CRCs it predicts poor prognosis. We propose a universal CRC LS screening algorithm using concurrent reflex immunohistochemistry (IHC) for BRAFV600E and mismatch-repair (MMR) proteins. We compared BRAFV600E IHC with multiplex polymerase chain reaction (PCR) and matrix-assisted laser desorption/ionization-time of flight mass spectrometry in 216 consecutive CRCs from 2011. Discordant cases were resolved with real-time PCR. BRAFV600E IHC was performed on 51 CRCs from the Australasian Colorectal Cancer Family Registry (ACCFR), which were fully characterized for BRAF mutation by allele-specific PCR, MMR status (MMR IHC and MSI), MLH1 promoter methylation, and germline MLH1 mutation. We then assessed MMR and BRAFV600E IHC on 1403 consecutive CRCs. By matrix-assisted laser desorption/ionization-time of flight mass spectrometry 15 cases did not yield a BRAF result, whereas 38/201 (19%) were positive. By IHC 45/216 (20%) were positive. Of the 7 discordant cases, real-time PCR confirmed the IHC result in 6. In the 51 CRCs from the ACCFR, IHC was concordant with allele-specific PCR in 50 cases. BRAFV600E and MSI IHC on 1403 CRCs demonstrated the following phenotypes: BRAF/MSS (1029 cases, 73%), BRAF/MSS (98, 7%), BRAF/MSI (183, 13%), and BRAF/MSI (93, 7%). All 11/1403 cancers associated with proven LS were BRAF/MSI. We conclude that BRAF IHC is highly concordant with 2 commonly used PCR-based BRAFV600E assays; it performed well in identifying MLH1 mutation carriers from the ACCFR and identified all cases of proven LS among the 1403 CRCs. Reflex BRAFV600E and MMR IHC are simple cheap tests that facilitate universal LS screening and identify the poor prognosis of the BRAFV600E-mutant MSS CRC phenotype.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Expression of MUC2, MUC5AC, MUC5B, and MUC6 mucins in colorectal cancers and their association with the CpG island methylator phenotype</a>, Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walsh MD, Clendenning M, Williamson E, Pearson SA, Walters RJ, Nagler B, Packenas D, Win AK, Hopper JL, Jenkins MA, Haydon AM, Rosty C, English DR, Giles GG, McGuckin MA, Young JP, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">26</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/modpathol.2013.101</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23807779</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/modpathol.2013.101">http://dx.doi.org/10.1038/modpathol.2013.101</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Mucinous differentiation is associated with both CpG island methylator phenotype and microsatellite instability in colorectal cancer. The mucinous phenotype derives from abundant expression of the colonic goblet cell mucin, MUC2, and de novo expression of gastric foveolar mucin, MUC5AC. We, therefore, investigated the protein expression levels of MUC2 and MUC5AC, as well as MUC5B and MUC6, in molecular subtypes of colorectal cancer. Seven-hundred and twenty-two incident colorectal carcinomas occurring in 702 participants of the Melbourne Collaborative Cohort Study were characterized for methylator status, MLH1 methylation, somatic BRAF and KRAS mutations, microsatellite-instability status, MLH1, MSH2, MSH6, and PMS2 mismatch repair, and p53 protein expression, and their histopathology was reviewed. Protein expression levels of MUC2, MUC5AC, MUC5B, MUC6, and the putative mucin regulator CDX2 were compared with molecular and clinicopathological features of colorectal cancers using odds ratios and corresponding 95% confidence intervals. MUC2 overexpression (>25% positive tumor cells) was observed in 33% colorectal cancers, MUC5B expression in 53%, and de novo MUC5AC and MUC6 expression in 50% and 39%, respectively. Co-expression of two or more of the mucins was commonly observed. Expression of MUC2, MUC5AC and MUC6 was strongly associated with features associated with tumorigenesis via the serrated neoplasia pathway, including methylator positivity, somatic BRAF p.V600E mutation, and mismatch repair deficiency, as well as proximal location, poor differentiation, lymphocytic response, and increased T stage (all P<0.001). Overexpression was observed in tumors with and without mucinous differentiation. There were inverse associations between expression of all four mucins and p53 overexpression. CDX2 expression was inversely associated with MUC2, MUC5AC and MUC6 expression. Our results suggest that, in methylator-positive tumors, mucin genes on chromosome 11p15.5 region undergo increased expression via mechanisms other than direct regulation by CDX2.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lynch syndrome-associated breast cancers do not overexpress chromosome 11-encoded mucins</a>, Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walsh MD, Cummings MC, Pearson SA, Clendenning M, Walters RJ, Nagler B, Hopper JL, Jenkins MA, Suthers GK, Goldblatt J, Tucker K, Gattas MR, Arnold JL, Parry S, Macrae FA, McGuckin MA, Young JP, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">26</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/modpathol.2012.232</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23370770</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/modpathol.2012.232">http://dx.doi.org/10.1038/modpathol.2012.232</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Mismatch repair-deficient breast cancers may be identified in Lynch syndrome mutation carriers, and have clinicopathological features in common with mismatch repair-deficient colorectal and endometrial cancers such as tumour-infiltrating lymphocytes and poor differentiation. Mismatch repair-deficient colorectal cancers frequently show mucinous differentiation associated with upregulation of chromosome 11 mucins. The aim of this study was to compare the protein expression of these mucins in mismatch repair-deficient and -proficient breast cancers. Cases of breast cancer (n=100) were identified from families where (1) both breast and colon cancer co-occurred and (2) families met either modified Amsterdam criteria or had at least one early-onset (<50 years) colorectal cancer. Tumour sections were stained for the epithelial mucins, MUC2, MUC5AC, MUC5B and MUC6, and the homeobox protein CDX2, a regulator of MUC2 expression. In all, 16 mismatch repair-deficient Lynch syndrome breast cancers and 84 non-Lynch breast cancers were assessed for altered mucin expression. No significant difference in the expression of MUC2, MUC5AC or MUC6 was observed between the mismatch repair-deficient and mismatch repair-proficient breast cancers; however, there was a trend for mismatch repair-deficient tumours to express high levels of MUC5B less frequently (P=0.07, OR=0.2 (0.0-1.0)). Co-expression of two or more gel-forming mucins was common. Ectopic expression of CDX2 was associated with expression of MUC2 (P=0.035, OR=8.7 (1.3-58.4)). Mismatch repair-deficient breast cancers do not show differential expression of the mucins genes on chromosome 11 when compared with mismatch repair-proficient breast cancers, in contrast with mismatch repair-deficient colorectal and endometrial cancers, which frequently have increased mucin protein expression when compared with their mismatch repair-proficient counterparts. In addition, ectopic CDX2 expression is positively associated with de novo MUC2 expression.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association between hypermethylation of DNA repetitive elements in white blood cell DNA and early-onset colorectal cancer</a>, Epigenetics: official journal of the DNA Methylation Society, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walters RJ, Williamson EJ, English DR, Young JP, Rosty C, Clendenning M, Walsh MD, Parry S, Ahnen DJ, Baron JA, Win AK, Giles GG, Hopper JL, Jenkins MA, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Epigenetics: official journal of the DNA Methylation Society</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.4161/epi.25178</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23804018</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.4161/epi.25178">http://dx.doi.org/10.4161/epi.25178</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Changes in the methylation levels of DNA from white blood cells (WBCs) are putatively associated with an elevated risk for several cancers. The aim of this study was to investigate the association between colorectal cancer (CRC) and the methylation status of three DNA repetitive elements in DNA from peripheral blood. WBC DNA from 539 CRC cases diagnosed before 60 years of age and 242 sex and age frequency-matched healthy controls from the Australasian Colorectal Cancer Family Registry were assessed for methylation across DNA repetitive elements Alu, LINE-1 and Sat2 using MethyLight. The percentage of methylated reference (PMR) of cases and controls was calculated for each marker. Odds ratios (ORs) and 95% confidence intervals (CIs) were estimated using multivariable logistic regression adjusted for potential confounders. CRC cases demonstrated a significantly higher median PMR for LINE-1 (p < 0.001), Sat2 (p < 0.001) and Alu repeats (p = 0.02) when compared with controls. For each of the DNA repetitive elements, individuals with PMR values in the highest quartile were significantly more likely to have CRC compared with those in the lowest quartile (LINE-1 OR = 2.34, 95%CI = 1.48-3.70; p < 0.001, Alu OR = 1.83, 95%CI = 1.17-2.86; p = 0.01, Sat2 OR = 1.72, 95%CI = 1.10-2.71; p = 0.02). When comparing the OR for the PMR of each marker across subgroups of CRC, only the Alu marker showed a significant difference in the 5-fluoruracil treated and nodal involvement subgroups (both p = 0.002). This association between increasing methylation levels of three DNA repetitive elements in WBC DNA and early-onset CRC is novel and may represent a potential epigenetic biomarker for early CRC detection.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Are the common genetic variants associated with colorectal cancer risk for DNA mismatch repair gene mutation carriers?</a>, European journal of cancer, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Hopper JL, Buchanan DD, Young JP, Tenesa A, Dowty JG, Giles GG, Goldblatt J, Winship I, Boussioutas A, Young GP, Parry S, Baron JA, Duggan D, Gallinger S, Newcomb PA, Haile RW, Le Marchand L, Lindor NM, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">European journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">49</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1016/j.ejca.2013.01.029</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23434150</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1016/j.ejca.2013.01.029">http://dx.doi.org/10.1016/j.ejca.2013.01.029</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/genetics Adenosine Triphosphatases/genetics Adult Alleles Colorectal Neoplasms/*genetics DNA Mismatch Repair/*genetics DNA Repair Enzymes/genetics DNA-Binding Proteins/genetics Family Health Female Gene Frequency Genetic Predisposition to Disease/genetics Genotype *Germ-Line Mutation Heterozygote Humans Male Middle Aged MutS Homolog 2 Protein/genetics Nuclear Proteins/genetics *Polymorphism, Single Nucleotide Proportional Hazards Models Risk Assessment/statistics & numerical data Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Genome-wide association studies have identified at least 15 independent common genetic variants associated with colorectal cancer (CRC) risk. The aim of this study was to investigate whether 11 of these variants are associated with CRC risk for carriers of germline mutations in DNA mismatch repair (MMR) genes. METHODS: A total of 927 MMR gene mutation carriers (360 MLH1, 442 MSH2, 85 MSH6 and 40 PMS2) from 315 families enrolled in the Colon Cancer Family Registry, were genotyped for the single nucleotide polymorphisms (SNPs): rs16892766 (8q23.3), rs6983267 (8q24.21), rs719725 (9p24), rs10795668 (10p14), rs3802842 (11q23.1), rs4444235 (14q22.2), rs4779584 (15q13.3), rs9929218 (16q22.1), rs4939827 (18q21.1), rs10411210 (19q13.1) and rs961253 (20p12.3). We used a weighted Cox regression to estimate CRC risk for homozygous and heterozygous carriers of the risk allele compared with homozygous non-carriers as well as for an additive per allele model (on the log scale). RESULTS: Over a total of 40,978 person-years observation, 426 (46%) carriers were diagnosed with CRC at a mean age of 44.3 years. For all carriers combined, we found no evidence of an association between CRC risk and the total number of risk alleles (hazard ratio [HR] per risk allele=0.97, 95% confidence interval [CI]=0.88-1.07, p=0.52). CONCLUSIONS: We found no evidence that the SNPs associated with CRC in the general population are modifiers of the risk for MMR gene mutation carriers overall, and therefore any evidence of proven clinical utility in Lynch syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risks of colorectal and other cancers after endometrial cancer for women with Lynch syndrome</a>, Journal of the National Cancer Institute, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Lindor NM, Winship I, Tucker KM, Buchanan DD, Young JP, Rosty C, Leggett B, Giles GG, Goldblatt J, Macrae FA, Parry S, Kalady MF, Baron JA, Ahnen DJ, Marchand LL, Gallinger S, Haile RW, Newcomb PA, Hopper JL, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">105</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1093/jnci/djs525</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23385444</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1093/jnci/djs525">http://dx.doi.org/10.1093/jnci/djs525</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/genetics Adenosine Triphosphatases/genetics Adult Aged Breast Neoplasms/epidemiology Colorectal Neoplasms/*epidemiology/genetics Colorectal Neoplasms, Hereditary Nonpolyposis/*epidemiology/genetics DNA Mismatch Repair/*genetics DNA Repair Enzymes/genetics DNA-Binding Proteins/genetics Endometrial Neoplasms/*epidemiology/genetics Female *Germ-Line Mutation *Heterozygote Humans Incidence Kaplan-Meier Estimate Middle Aged MutS Homolog 2 Protein/genetics Neoplasms/epidemiology Nuclear Proteins/genetics Risk Skin Neoplasms/epidemiology Urologic Neoplasms/epidemiology/genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Lynch syndrome is an autosomal dominantly inherited disorder caused by germline mutations in DNA mismatch repair (MMR) genes. Previous studies have shown that MMR gene mutation carriers are at increased risk of colorectal, endometrial, and several other cancers following an initial diagnosis of colorectal cancer. We estimated cancer risks following an endometrial cancer diagnosis for women carrying MMR gene mutations. METHODS: We obtained data from the Colon Cancer Family Registry for a cohort of 127 women who had a diagnosis of endometrial cancer and who carried a mutation in one of four MMR genes (30 carried a mutation in MLH1, 72 in MSH2, 22 in MSH6, and 3 in PMS2). We used the Kaplan-Meier method to estimate 10- and 20-year cumulative risks for each cancer. We estimated the age-, country-, and calendar period-specific standardized incidence ratios (SIRs) for each cancer, compared with the general population. RESULTS: Following endometrial cancer, women carrying MMR gene mutations had the following 20-year risks of other cancer cancers: colorectal cancer (48%, 95% confidence interval [CI] = 35% to 62%); cancer of the kidney, renal pelvis, or ureter (11%, 95% CI = 3% to 20%); urinary bladder cancer (9%, 95% CI = 2% to 17%); and breast cancer (11%, 95% CI = 4% to 19%). Compared with the general population, these women were at statistically significantly elevated risks of colorectal cancer (SIR = 39.9, 95% CI = 27.2 to 58.3), cancer of the kidney, renal pelvis, or ureter (SIR = 28.3, 95% CI = 11.9 to 48.6), urinary bladder cancer (SIR = 24.3, 95% CI = 8.56 to 42.9), and breast cancer (SIR = 2.51, 95% CI = 1.17 to 4.14). CONCLUSIONS: Women with Lynch syndrome who are diagnosed with endometrial cancer have increased risks of several cancers, including breast cancer.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk of metachronous colon cancer following surgery for rectal cancer in mismatch repair gene mutation carriers</a>, Annals of surgical oncology, 2013.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Parry S, Parry B, Kalady MF, Macrae FA, Ahnen DJ, Young GP, Lipton L, Winship I, Boussioutas A, Young JP, Buchanan DD, Arnold J, Le Marchand L, Newcomb PA, Haile RW, Lindor NM, Gallinger S, Hopper JL, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Annals of surgical oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">20</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2013</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1245/s10434-012-2858-5</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23358792</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1245/s10434-012-2858-5">http://dx.doi.org/10.1245/s10434-012-2858-5</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/genetics Adenosine Triphosphatases/genetics Adolescent Adult Aged Australia/epidemiology Canada/epidemiology *Carrier State Colonic Neoplasms/*epidemiology/*genetics/pathology DNA Repair Enzymes/genetics DNA-Binding Proteins/genetics Female Humans Incidence Kaplan-Meier Estimate Male Middle Aged MutS Homolog 2 Protein/genetics Mutation Neoplasms, Second Primary/*epidemiology/*genetics/pathology New Zealand/epidemiology Nuclear Proteins/genetics Proportional Hazards Models Rectal Neoplasms/*genetics/surgery Retrospective Studies Risk Factors United States/epidemiology Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Despite regular surveillance colonoscopy, the metachronous colorectal cancer risk for mismatch repair (MMR) gene mutation carriers after segmental resection for colon cancer is high and total or subtotal colectomy is the preferred option. However, if the index cancer is in the rectum, management decisions are complicated by considerations of impaired bowel function. We aimed to estimate the risk of metachronous colon cancer for MMR gene mutation carriers who underwent a proctectomy for index rectal cancer. METHODS: This retrospective cohort study comprised 79 carriers of germline mutation in a MMR gene (18 MLH1, 55 MSH2, 4 MSH6, and 2 PMS2) from the Colon Cancer Family Registry who had had a proctectomy for index rectal cancer. Cumulative risks of metachronous colon cancer were calculated using the Kaplan-Meier method. RESULTS: During median 9 years (range 1-32 years) of observation since the first diagnosis of rectal cancer, 21 carriers (27 %) were diagnosed with metachronous colon cancer (incidence 24.25, 95 % confidence interval [CI] 15.81-37.19 per 1,000 person-years). Cumulative risk of metachronous colon cancer was 19 % (95 % CI 9-31 %) at 10 years, 47 (95 % CI 31-68 %) at 20 years, and 69 % (95 % CI 45-89 %) at 30 years after surgical resection. The frequency of surveillance colonoscopy was 1 colonoscopy per 1.16 years (95 % CI 1.01-1.31 years). The AJCC stages of the metachronous cancers, where available, were 72 % stage I, 22 % stage II, and 6 % stage III. CONCLUSIONS: Given the high metachronous colon cancer risk for MMR gene mutation carriers diagnosed with an index rectal cancer, proctocolectomy may need to be considered.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2012</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Colorectal Cancer Linkage on Chromosomes 4q21, 8q13, 12q24, and 15q22</a>, PloS one, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Cicek MS, Cunningham JM, Fridley BL, Serie DJ, Bamlet WR, Diergaarde B, Haile RW, Le Marchand L, Krontiris TG, Younghusband HB, Gallinger S, Newcomb PA, Hopper JL, Jenkins MA, Casey G, Schumacher F, Chen Z, Derycke MS, Templeton AS, Winship I, Green RC, Green JS, Macrae FA, Parry S, Young GP, Young JP, Buchanan D, Thomas DC, Bishop DT, Lindor NM, Thibodeau SN, Potter JD, Goode EL</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/journal.pone.0038175 PONE-D-12-07558 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22675446</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pone.0038175 PONE-D-12-07558 [pii]">http://dx.doi.org/10.1371/journal.pone.0038175 PONE-D-12-07558 [pii]</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">A substantial proportion of familial colorectal cancer (CRC) is not a consequence of known susceptibility loci, such as mismatch repair (MMR) genes, supporting the existence of additional loci. To identify novel CRC loci, we conducted a genome-wide linkage scan in 356 white families with no evidence of defective MMR (i.e., no loss of tumor expression of MMR proteins, no microsatellite instability (MSI)-high tumors, or no evidence of linkage to MMR genes). Families were ascertained via the Colon Cancer Family Registry multi-site NCI-supported consortium (Colon CFR), the City of Hope Comprehensive Cancer Center, and Memorial University of Newfoundland. A total of 1,612 individuals (average 5.0 per family including 2.2 affected) were genotyped using genome-wide single nucleotide polymorphism linkage arrays; parametric and non-parametric linkage analysis used MERLIN in a priori-defined family groups. Five lod scores greater than 3.0 were observed assuming heterogeneity. The greatest were among families with mean age of diagnosis less than 50 years at 4q21.1 (dominant HLOD = 4.51, alpha = 0.84, 145.40 cM, rs10518142) and among all families at 12q24.32 (dominant HLOD = 3.60, alpha = 0.48, 285.15 cM, rs952093). Among families with four or more affected individuals and among clinic-based families, a common peak was observed at 15q22.31 (101.40 cM, rs1477798; dominant HLOD = 3.07, alpha = 0.29; dominant HLOD = 3.03, alpha = 0.32, respectively). Analysis of families with only two affected individuals yielded a peak at 8q13.2 (recessive HLOD = 3.02, alpha = 0.51, 132.52 cM, rs1319036). These previously unreported linkage peaks demonstrate the continued utility of family-based data in complex traits and suggest that new CRC risk alleles remain to be elucidated.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Correction: Colorectal Cancer Linkage on Chromosomes 4q21, 8q13, 12q24, and 15q22</a>, PloS one, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Cicek MS, Cunningham JM, Fridley BL, Serie DJ, Bamlet WR, Diergaarde B, Haile RW, Le Marchand L, Krontiris TG, Younghusband HB, Gallinger S, Newcomb PA, Hopper JL, Jenkins MA, Casey G, Schumacher F, Chen Z, Derycke MS, Templeton AS, Winship I, Green RC, Green JS, Macrae FA, Parry S, Young GP, Young JP, Buchanan D, Thomas DC, Bishop DT, Lindor NM, Thibodeau SN, Potter JD, Goode EL</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/annotation/1ba2f5e3-8aef-4a12-909b-23f95a889325</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23284602</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/annotation/1ba2f5e3-8aef-4a12-909b-23f95a889325">http://dx.doi.org/10.1371/annotation/1ba2f5e3-8aef-4a12-909b-23f95a889325</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">[This corrects the article on p. e38175 in vol. 7.].</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Using tumour pathology to identify people at high genetic risk of breast and colorectal cancers</a>, Pathology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Hopper JL, Jenkins MA, Dowty JG, Dite GS, Apicella C, Keogh L, Win AK, Young JP, Buchanan D, Walsh MD, Rosty C, Baglietto L, Severi G, Phillips KA, Wong EM, Dobrovic A, Waring P, Winship I, Ramus SJ, Giles GG, Southey MC</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">44</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1097/PAT.0b013e32834e8e5b</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22198256</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1097/PAT.0b013e32834e8e5b">http://dx.doi.org/10.1097/PAT.0b013e32834e8e5b</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">SUMMARY: Genes have been identified for which germline mutations are associated with high lifetime risks of breast, colorectal and other cancers. Identification of mutation carriers through genetic testing is important as it could help lower cancer incidence and mortality. The translation of genetic information into better health outcomes is expensive because of the costs of genetic counselling as well as laboratory testing. Approaches to triage for mutation screening of known genes which rely on cancer family history are not necessarily sensitive and specific or the most cost-effective. Recent population-based research has shown that the cancers and precancerous lesions arising in mutation carriers have specific molecular and morphological characteristics. People with colorectal cancer, especially those diagnosed at a young age, whose tumours exhibit microsatellite instability and some specific pathology and immunohistochemically-defined features are more likely to carry a germline mutation in one of four mismatch repair genes. Some morphological and immunohistochemically-defined features are associated with breast cancers arising in women who carry BRCA1 or BRCA2 germline mutations, especially if at a young age. Screening paradigms based on molecular and morphological features that predict mutation status, especially if focused on early-onset disease, have the potential to identify mutation carriers with greater sensitivity and specificity, and in a more cost-effective way, than those based on family history alone. Genetic testing results could help inform treatment if those affected are tested soon after diagnosis using pathology-led selection strategies to identify cases most likely to carry germline mutations. We propose how this new approach could be undertaken by having genetic testing and counselling prioritised to those with the greatest probability of carrying a germline mutation in these known cancer predisposition genes.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Body size and risk for colorectal cancers showing BRAF mutations or microsatellite instability: a pooled analysis</a>, International journal of epidemiology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Hughes LA, Williamson EJ, van Engeland M, Jenkins MA, Giles GG, Hopper JL, Southey MC, Young JP, Buchanan DD, Walsh MD, van den Brandt PA, Alexandra Goldbohm R, Weijenberg MP, English DR</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of epidemiology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">41</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22531127</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/22531127">https://www.ncbi.nlm.nih.gov/pubmed/22531127</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Anthropometry Australia/epidemiology Body Size/*genetics Colorectal Neoplasms/epidemiology/*genetics Female Humans Kaplan-Meier Estimate Male *Microsatellite Instability Middle Aged Netherlands/epidemiology Proportional Hazards Models Prospective Studies Proto-Oncogene Proteins B-raf/*genetics Questionnaires Registries Risk</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: How body size influences risk of molecular subtypes of colorectal cancer (CRC) is unclear. We investigated whether measures of anthropometry differentially influence risk of tumours according to BRAF c.1799T>A p.V600E mutation (BRAF) and microsatellite instability (MSI) status. METHODS: Data from The Netherlands Cohort Study (n = 120,852) and Melbourne Collaborative Cohort Study (n = 40,514) were pooled and included 734 and 717 colorectal cancer cases from each study, respectively. Hazard ratios (HRs) and 95% confidence intervals (CIs) for body mass index (BMI), waist measurement and height were calculated and compared for subtypes defined by BRAF mutation and MSI status, measured from archival tissue. RESULTS: Results were consistent between studies. When pooled, BMI modelled in 5 kg/m(2) increments was positively associated with BRAF wild-type (HR: 1.16, 95% CI: 1.08-1.26) and MS-stable tumours (HR: 1.15, 95% CI: 1.06-1.24). Waist measurement was also associated with BRAF wild-type (highest vs lowest quartile, HR: 1.59, 95% CI: 1.33-1.90) and MS-stable tumours (highest vs lowest quartile HR: 1.68, 95% CI: 1.31-2.15). The HRs for BRAF mutation tumours and MSI tumours were smaller and non-significant, but differences between the HRs by tumour subtypes were not significant. Height, modelled per 5-cm increase, was positively associated with BRAF wild-type and BRAF mutation tumours, but the HR was greater for tumours with a BRAF mutation than BRAF wild-type (HR: 1.23, 95% CI: 1.11-1.37, P(heterogeneity) = 0.03). Similar associations were observed with respect to height and MSI tumours (HR: 1.26, 95% CI: 1.13-1.40, P(heterogeneity) = 0.02). CONCLUSIONS: Generally, overweight increases the risk of CRC. Taller individuals have an increased risk of developing a tumour with a BRAF mutation or MSI.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Genome-wide search for gene-gene interactions in colorectal cancer</a>, PloS one, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jiao S, Hsu L, Berndt S, Bezieau S, Brenner H, Buchanan D, Caan BJ, Campbell PT, Carlson CS, Casey G, Chan AT, Chang-Claude J, Chanock S, Conti DV, Curtis KR, Duggan D, Gallinger S, Gruber SB, Harrison TA, Hayes RB, Henderson BE, Hoffmeister M, Hopper JL, Hudson TJ, Hutter CM, Jackson RD, Jenkins MA, Kantor ED, Kolonel LN, Kury S, Le Marchand L, Lemire M, Newcomb PA, Potter JD, Qu C, Rosse SA, Schoen RE, Schumacher FR, Seminara D, Slattery ML, Ulrich CM, Zanke BW, Peters U</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/journal.pone.0052535 PONE-D-12-29750 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23300701</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pone.0052535 PONE-D-12-29750 [pii]">http://dx.doi.org/10.1371/journal.pone.0052535 PONE-D-12-29750 [pii]</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Genome-wide association studies (GWAS) have successfully identified a number of single-nucleotide polymorphisms (SNPs) associated with colorectal cancer (CRC) risk. However, these susceptibility loci known today explain only a small fraction of the genetic risk. Gene-gene interaction (GxG) is considered to be one source of the missing heritability. To address this, we performed a genome-wide search for pair-wise GxG associated with CRC risk using 8,380 cases and 10,558 controls in the discovery phase and 2,527 cases and 2,658 controls in the replication phase. We developed a simple, but powerful method for testing interaction, which we term the Average Risk Due to Interaction (ARDI). With this method, we conducted a genome-wide search to identify SNPs showing evidence for GxG with previously identified CRC susceptibility loci from 14 independent regions. We also conducted a genome-wide search for GxG using the marginal association screening and examining interaction among SNPs that pass the screening threshold (p<10(-4)). For the known locus rs10795668 (10p14), we found an interacting SNP rs367615 (5q21) with replication p = 0.01 and combined p = 4.19x10(-8). Among the top marginal SNPs after LD pruning (n = 163), we identified an interaction between rs1571218 (20p12.3) and rs10879357 (12q21.1) (nominal combined p = 2.51x10(-6); Bonferroni adjusted p = 0.03). Our study represents the first comprehensive search for GxG in CRC, and our results may provide new insight into the genetic etiology of CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Cancer Risks for the Relatives of Colorectal Cancer Cases with a Methylated MLH1 Promoter Region: Data from the Colorectal Cancer Family Registry</a>, Cancer prevention research , 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Levine AJ, Win AK, Buchanan DD, Jenkins MA, Baron JA, Young JP, Long TI, Weisenberger DJ, Laird PW, McCall RL, Duggan DJ, Haile RW</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer prevention research </dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22144422</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/22144422">https://www.ncbi.nlm.nih.gov/pubmed/22144422</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Methylation of the MLH1 gene promoter region is an underlying cause of colorectal cancer (CRC) with high microsatellite instability (MSI-H) diagnosed in persons without a germ line mutation in a mismatch repair (MMR) gene (non-Lynch Syndrome CRC). It is unclear whether relatives of CRC cases with MLH1 methylation have an increased risk of colorectal or other cancers. In this retrospective cohort study, we assessed risk of CRC and other cancers for the first- and second-degree relatives of CRC cases with a methylated MLH1 gene, by comparing observed numbers of cases with those expected on the basis of age-, sex-, and country-specific cancer incidences (standardized incidence ratios). The cohort consisted of 3,128 first- and second-degree relatives of the 233 MLH1-methylated CRC cases with no MMR or MUTYH gene mutations. The standardized incidence ratio (SIR) for CRC was 1.60 [95% confidence interval (CI), 1.22-2.16] for first-degree relatives and 1.08 (0.74-1.60) for second-degree relatives. The SIR for gastric cancer was 2.58 (1.52-4.71) for first-degree relatives and 4.52 (2.23-10.61) for second-degree relatives and, for ovarian cancer, it was 2.16 (1.29-3.86) for first-degree relatives. The risk of liver cancer was also increased significantly in first-degree relatives but the estimate was on the basis of only two cases. These data imply that relatives of CRC cases with MLH1 methylation may be at increased risk of CRC and stomach cancer and possibly ovarian and liver cancer, suggesting that there may be a heritable factor for CRC and other cancers associated with MLH1 methylation in non-Lynch syndrome CRCs. Cancer Prev Res; 5(2); 328-35. (c)2011 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Identification of Lynch syndrome among patients with colorectal cancer</a>, JAMA: the journal of the American Medical Association, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Moreira L, Balaguer F, Lindor N, de la Chapelle A, Hampel H, Aaltonen LA, Hopper JL, Le Marchand L, Gallinger S, Newcomb PA, Haile R, Thibodeau SN, Gunawardena S, Jenkins MA, Buchanan DD, Potter JD, Baron JA, Ahnen DJ, Moreno V, Andreu M, Ponz de Leon M, Rustgi AK, Castells A</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">JAMA: the journal of the American Medical Association</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">308</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">15</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1001/jama.2012.13088 1383232 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">23073952</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1001/jama.2012.13088 1383232 [pii]">http://dx.doi.org/10.1001/jama.2012.13088 1383232 [pii]</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Cohort Studies Colorectal Neoplasms, Hereditary Nonpolyposis/*diagnosis/*genetics DNA Mismatch Repair/*genetics DNA Repair Enzymes/*genetics Family Female Finland Genetic Testing/*standards Humans Male Middle Aged Multivariate Analysis Ohio Registries/statistics & numerical data Sensitivity and Specificity</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">CONTEXT: Lynch syndrome is the most common form of hereditary colorectal cancer (CRC) and is caused by germline mutations in DNA mismatch repair (MMR) genes. Identification of gene carriers currently relies on germline analysis in patients with MMR-deficient tumors, but criteria to select individuals in whom tumor MMR testing should be performed are unclear. OBJECTIVE: To establish a highly sensitive and efficient strategy for the identification of MMR gene mutation carriers among CRC probands. DESIGN, SETTING, AND PATIENTS: Pooled-data analysis of 4 large cohorts of newly diagnosed CRC probands recruited between 1994 and 2010 (n = 10,206) from the Colon Cancer Family Registry, the EPICOLON project, the Ohio State University, and the University of Helsinki examining personal, tumor-related, and family characteristics, as well as microsatellite instability, tumor MMR immunostaining, and germline MMR mutational status data. MAIN OUTCOME: Performance characteristics of selected strategies (Bethesda guidelines, Jerusalem recommendations, and those derived from a bivariate/multivariate analysis of variables associated with Lynch syndrome) were compared with tumor MMR testing of all CRC patients (universal screening). RESULTS: Of 10,206 informative, unrelated CRC probands, 312 (3.1%) were MMR gene mutation carriers. In the population-based cohorts (n = 3671 probands), the universal screening approach (sensitivity, 100%; 95% CI, 99.3%-100%; specificity, 93.0%; 95% CI, 92.0%-93.7%; diagnostic yield, 2.2%; 95% CI, 1.7%-2.7%) was superior to the use of Bethesda guidelines (sensitivity, 87.8%; 95% CI, 78.9%-93.2%; specificity, 97.5%; 95% CI, 96.9%-98.0%; diagnostic yield, 2.0%; 95% CI, 1.5%-2.4%; P < .001), Jerusalem recommendations (sensitivity, 85.4%; 95% CI, 77.1%-93.6%; specificity, 96.7%; 95% CI, 96.0%-97.2%; diagnostic yield, 1.9%; 95% CI, 1.4%-2.3%; P < .001), and a selective strategy based on tumor MMR testing of cases with CRC diagnosed at age 70 years or younger and in older patients fulfilling the Bethesda guidelines (sensitivity, 95.1%; 95% CI, 89.8%-99.0%; specificity, 95.5%; 95% CI, 94.7%-96.1%; diagnostic yield, 2.1%; 95% CI, 1.6%-2.6%; P < .001). This selective strategy missed 4.9% of Lynch syndrome cases but resulted in 34.8% fewer cases requiring tumor MMR testing and 28.6% fewer cases undergoing germline mutational analysis than the universal approach. CONCLUSION: Universal tumor MMR testing among CRC probands had a greater sensitivity for the identification of Lynch syndrome compared with multiple alternative strategies, although the increase in the diagnostic yield was modest.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Correlation of tumour BRAF mutations and MLH1 methylation with germline mismatch repair (MMR) gene mutation status: a literature review assessing utility of tumour features for MMR variant classification</a>, Journal of medical genetics, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Parsons MT, Buchanan DD, Thompson B, Young JP, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of medical genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">49</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22368298</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/22368298">https://www.ncbi.nlm.nih.gov/pubmed/22368298</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Colorectal cancer (CRC) that demonstrates microsatellite instability (MSI) is caused by either germline mismatch repair (MMR) gene mutations, or 'sporadic' somatic tumour MLH1 promoter methylation. MLH1 promoter methylation is reportedly correlated with tumour BRAF V600E mutation status. No systematic review has been undertaken to assess the value of BRAF V600E mutation and MLH1 promoter methylation tumour markers as negative predictors of germline MMR mutation status. A literature review of CRC cohorts tested for MMR mutations, and tumour BRAF V600E mutation and/or MLH1 promoter methylation was conducted using PubMed. Studies were assessed for tumour features, stratified by tumour MMR status based on immunohistochemistry or MSI where possible. Pooled frequencies and 95% CIs were calculated using a random effects model. BRAF V600E results for 4562 tumours from 35 studies, and MLH1 promoter methylation results for 2975 tumours from 43 studies, were assessed. In 550 MMR mutation carriers, the BRAF V600E mutation frequency was 1.40% (95% CI 0.06% to 3%). In MMR mutation-negative cases, the BRAF V600E mutation frequency was 5.00% (95% CI 4% to 7%) in 1623 microsatellite stable (MSS) cases and 63.50% (95% CI 47% to 79%) in 332 cases demonstrating MLH1 methylation or MLH1 expression loss. MLH1 promoter methylation of the 'A region' was reported more frequently than the 'C region' in MSS CRCs (17% vs 0.06%, p<0.0001) and in MLH1 mutation carriers (42% vs 6%, p<0.0001), but not in MMR mutation-negative MSI-H CRCs (40% vs 47%, p=0.12). Methylation of the 'C region' was a predictor of MMR mutation-negative status in MSI-H CRC cases (47% vs 6% in MLH1 mutation carriers, p<0.0001). This review demonstrates that tumour BRAF V600E mutation, and MLH1 promoter 'C region' methylation specifically, are strong predictors of negative MMR mutation status. It is important to incorporate these features in multifactorial models aimed at predicting MMR mutation status.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>BRAF Mutation Status and Survival after Colorectal Cancer Diagnosis According to Patient and Tumor Characteristics</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Phipps AI, Buchanan DD, Makar KW, Burnett-Hartman AN, Coghill AE, Passarelli MN, Baron JA, Ahnen DJ, Win AK, Potter JD, Newcomb PA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">21</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1158/1055-9965.EPI-12-0674 1055-9965.EPI-12-0674 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22899730</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1158/1055-9965.EPI-12-0674 1055-9965.EPI-12-0674 [pii]">http://dx.doi.org/10.1158/1055-9965.EPI-12-0674 1055-9965.EPI-12-0674 [pii]</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: BRAF mutations in colorectal cancer (CRC) are disproportionately observed in tumors exhibiting microsatellite instability (MSI) and are associated with other prognostic factors. The independent association between BRAF mutation status and CRC survival, however, remains unclear. METHODS: We evaluated the association between the BRAF c.1799T>A (p.V600E) mutation and survival in individuals with incident invasive CRC diagnosed between 1997 and 2007 in Western Washington State. Tumor specimens were tested for this BRAF mutation and MSI status. We used Cox regression to estimate HRs and 95% confidence intervals (CI) for the association between BRAF mutation status and disease-specific and overall survival. Stratified analyses were conducted by age, sex, tumor site, stage, and MSI status. RESULTS: Among 1,980 cases tested, 12% were BRAF c.1799T>A (p.V600E) mutation-positive (n = 247). BRAF-mutated CRC was associated with poorer disease-specific survival adjusting for age, sex, time from diagnosis to enrollment, stage, and MSI status (HR, 1.43; 95% CI, 1.05-1.95). This association was limited to cases diagnosed at ages A (p.V600E) mutation is associated with significantly poorer prognosis after CRC diagnosis among subgroups of patients. Cancer Epidemiol Biomarkers Prev; 21(10); 1792-8. (c)2012 AACR.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Phenotype and polyp landscape in serrated polyposis syndrome: a series of 100 patients from genetics clinics</a>, The American journal of surgical pathology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Buchanan DD, Walsh MD, Pearson SA, Pavluk E, Walters RJ, Clendenning M, Spring KJ, Jenkins MA, Win AK, Hopper JL, Sweet K, Frankel WL, Aronson M, Gallinger S, Goldblatt J, Woodall S, Arnold J, Walker NI, Jass JR, Parry S, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The American journal of surgical pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">36</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1097/PAS.0b013e31824e133f</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22510757</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1097/PAS.0b013e31824e133f">http://dx.doi.org/10.1097/PAS.0b013e31824e133f</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/metabolism Adenoma/genetics/metabolism/*pathology Adenosine Triphosphatases/metabolism Adolescent Adult Aged Colectomy Colon/*pathology Colonic Polyps/genetics/metabolism/*pathology Colorectal Neoplasms/genetics/metabolism/*pathology DNA Mutational Analysis DNA Repair Enzymes/metabolism DNA, Neoplasm/genetics DNA-Binding Proteins/metabolism Female Humans Hyperplasia/genetics/metabolism/pathology Male Middle Aged Mutation Nuclear Proteins/metabolism Phenotype Proto-Oncogene Proteins/genetics Proto-Oncogene Proteins B-raf/genetics Syndrome Tumor Markers, Biological/genetics/metabolism Young Adult ras Proteins/genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Serrated polyposis syndrome (SPS), also known as hyperplastic polyposis, is a syndrome of unknown genetic basis defined by the occurrence of multiple serrated polyps in the large intestine and associated with an increased risk of colorectal cancer (CRC). There are a variety of SPS presentations, which may encompass a continuum of phenotypes modified by environmental and genetic factors. To explore the phenotype of SPS, we recorded the histologic and molecular characteristics of multiple colorectal polyps in patients with SPS recruited between 2000 and 2010 from genetics clinics in Australia, New Zealand, Canada, and the United States. Three specialist gastrointestinal pathologists reviewed the polyps, which they classified into conventional adenomas or serrated polyps, with various subtypes, according to the current World Health Organization criteria. Mutations in BRAF and KRAS and mismatch repair protein expression were determined in a subset of polyps. A total of 100 patients were selected for the study, of whom 58 were female and 42 were male. The total polyp count per patient ranged from 6 to 150 (median 30). The vast majority of patients (89%) had polyposis affecting the entire large intestine. From this cohort, 406 polyps were reviewed. Most of the polyps (83%) were serrated polyps: microvesicular hyperplastic polyps (HP) (n=156), goblet cell HP (n=25), sessile serrated adenoma/polyps (SSA/P) (n=110), SSA/P with cytologic dysplasia (n=28), and traditional serrated adenomas (n=18). A further 69 polyps were conventional adenomas. BRAF mutation was mainly detected in SSA/P with dysplasia (95%), SSA/P (85%), microvesicular HP (76%), and traditional serrated adenoma (54%), whereas KRAS mutation was present mainly in goblet cell HP (50%) and in tubulovillous adenoma (45%). Four of 6 SSA/Ps with high-grade dysplasia showed loss of MLH1/PMS2 expression. CRC was diagnosed in 39 patients who were more often found to have a conventional adenoma compared with patients without CRC (P=0.003). Patients with SPS referred to genetics clinics had a pancolonic disease with a high polyp burden and a high rate of BRAF mutation. The occurrence of CRC was associated with the presence of conventional adenoma.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>KRAS mutations in ovarian low-grade endometrioid adenocarcinoma: association with concurrent endometriosis</a>, Human pathology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Stewart CJ, Leung Y, Walsh MD, Walters RJ, Young JP, Buchanan DD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">43</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22305241</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/22305241">https://www.ncbi.nlm.nih.gov/pubmed/22305241</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The association between ovarian endometrioid adenocarcinoma and endometriosis is well established. However, not all endometrioid adenocarcinomas are directly related to endometriosis, and it has been suggested that there may be clinicopathologic differences between endometriosis-positive and endometriosis-negative tumors. Molecular alterations in endometrioid adenocarcinoma include KRAS and BRAF mutations, but the incidence of these abnormalities in previous reports has been highly variable (0%-36% and 0%-24%, respectively). This may be explained by relatively small sample sizes in earlier studies but could also reflect difficulties in accurately classifying high-grade ovarian malignancies. In the current study, we investigated KRAS and BRAF mutations in 78 low-grade (FIGO grade 1 and 2) endometrioid adenocarcinomas and compared the results with the presence of endometriosis in the tumor-associated ovary and/or in other pelvic sites. KRAS mutations were identified in 12 (29%) of 42 endometriosis-associated endometrioid adenocarcinomas with satisfactory analysis but in only 1 (3%) of 29 tumors in which endometriosis was not identified. BRAF mutation was identified only in a single endometriosis-associated case. These findings support the hypothesis that endometriosis-associated and independent endometrioid adenocarcinoma may develop via different molecular pathways and that KRAS mutations have an important role only in the former tumors. In contrast, BRAF mutations do not appear to have a significant role in either endometrioid adenocarcinoma subgroup. This may be relevant to future targeted therapies in patients with high-stage or recurrent disease and indicate that histopathologists should carefully examine endometrioid adenocarcinoma specimens, including nonneoplastic tissues, for the presence of endometriosis.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Immunohistochemical testing of conventional adenomas for loss of expression of mismatch repair proteins in Lynch syndrome mutation carriers: a case series from the Australasian site of the colon cancer family registry</a>, Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walsh MD, Buchanan DD, Pearson SA, Clendenning M, Jenkins MA, Win AK, Walters RJ, Spring KJ, Nagler B, Pavluk E, Arnold ST, Goldblatt J, George J, Suthers GK, Phillips K, Hopper JL, Jass JR, Baron JA, Ahnen DJ, Thibodeau SN, Lindor N, Parry S, Walker NI, Rosty C, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Modern pathology: an official journal of the United States and Canadian Academy of Pathology, Inc</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">25</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/modpathol.2011.209 modpathol2011209 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22322191</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/modpathol.2011.209 modpathol2011209 [pii]">http://dx.doi.org/10.1038/modpathol.2011.209 modpathol2011209 [pii]</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Debate continues as to the usefulness of assessing adenomas for loss of mismatch repair protein expression to identify individuals with suspected Lynch syndrome. We tested 109 polyps from 69 proven mutation carriers (35 females and 34 males) belonging to 49 Lynch syndrome families. All polyps were tested by immunohistochemistry for four mismatch repair proteins MLH1, MSH2, MSH6 and PMS2. Detailed pathology review was performed by specialist gastrointestinal pathologists. The majority of polyps (86%) were conventional adenomas (n=94), with 65 tubular and 28 tubulovillous adenomas and a single villous adenoma. The remaining 15 lesions (14%) were serrated polyps. Overall, loss of mismatch repair expression was noted for 78/109 (72%) of polyps. Loss of mismatch repair expression was seen in 74 of 94 (79%) conventional adenomas, and 4 of 15 (27%) serrated polyps from mismatch repair gene mutation carriers. In all instances, loss of expression was consistent with the underlying germline mutation. Mismatch repair protein expression was lost in 27 of 29 adenomas with a villous component compared with 47 of 65 adenomas without this feature (93 vs 73%; P=0.028). A strong trend was observed for high-grade dysplasia. Mismatch repair deficiency was observed in 12 of 12 conventional adenomas with high-grade dysplasia compared with 60 of 79 with low-grade dysplasia (100 vs 76%; P=0.065). We were unable to demonstrate a significant association between conventional adenoma size or site and mismatch repair deficiency. All (4/4 or 100%) of the serrated polyps demonstrating mismatch repair deficiency were traditional serrated adenomas from a single family. Diagnostic testing of adenomas in suspected Lynch syndrome families is a useful alternative in cases where cancers are unavailable. The overwhelming majority of conventional adenomas from mutation carriers show loss of mismatch repair protein expression concordant with the underlying germline mutation.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risks of primary extracolonic cancers following colorectal cancer in lynch syndrome</a>, Journal of the National Cancer Institute, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Lindor NM, Young JP, Macrae FA, Young GP, Williamson E, Parry S, Goldblatt J, Lipton L, Winship I, Leggett B, Tucker KM, Giles GG, Buchanan DD, Clendenning M, Rosty C, Arnold J, Levine AJ, Haile RW, Gallinger S, Le Marchand L, Newcomb PA, Hopper JL, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">104</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">18</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22933731</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/22933731">https://www.ncbi.nlm.nih.gov/pubmed/22933731</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Lynch syndrome is a highly penetrant cancer predisposition syndrome caused by germline mutations in DNA mismatch repair (MMR) genes. We estimated the risks of primary cancers other than colorectal cancer following a diagnosis of colorectal cancer in mutation carriers. METHODS: We obtained data from the Colon Cancer Family Registry for 764 carriers of an MMR gene mutation (316 MLH1, 357 MSH2, 49 MSH6, and 42 PMS2), who had a previous diagnosis of colorectal cancer. The Kaplan-Meier method was used to estimate their cumulative risk of cancers 10 and 20 years after colorectal cancer. We estimated the age-, sex-, country- and calendar period-specific standardized incidence ratios (SIRs) of cancers following colorectal cancer, compared with the general population. RESULTS: Following colorectal cancer, carriers of MMR gene mutations had the following 10-year risk of cancers in other organs: kidney, renal pelvis, ureter, and bladder (2%, 95% confidence interval [CI] = 1% to 3%); small intestine, stomach, and hepatobiliary tract (1%, 95% CI = 0.2% to 2%); prostate (3%, 95% CI = 1% to 5%); endometrium (12%, 95% CI = 8% to 17%); breast (2%, 95% CI = 1% to 4%); and ovary (1%, 95% CI = 0% to 2%). They were at elevated risk compared with the general population: cancers of the kidney, renal pelvis, and ureter (SIR = 12.54, 95% CI = 7.97 to 17.94), urinary bladder (SIR = 7.22, 95% CI = 4.08 to 10.99), small intestine (SIR = 72.68, 95% CI = 39.95 to 111.29), stomach (SIR = 5.65, 95% CI = 2.32 to 9.69), and hepatobiliary tract (SIR = 5.94, 95% CI = 1.81 to 10.94) for both sexes; cancer of the prostate (SIR = 2.05, 95% CI = 1.23 to 3.01), endometrium (SIR = 40.23, 95% CI = 27.91 to 56.06), breast (SIR = 1.76, 95% CI = 1.07 to 2.59), and ovary (SIR = 4.19, 95% CI = 1.28 to 7.97). CONCLUSION: Carriers of MMR gene mutations who have already had a colorectal cancer are at increased risk of a greater range of cancers than the recognized spectrum of Lynch syndrome cancers, including breast and prostate cancers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Cancer risks for relatives of patients with serrated polyposis</a>, The American journal of gastroenterology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Walters RJ, Buchanan DD, Jenkins MA, Sweet K, Frankel WL, de la Chapelle A, McKeone DM, Walsh MD, Clendenning M, Pearson SA, Pavluk E, Nagler B, Hopper JL, Gattas MR, Goldblatt J, George J, Suthers GK, Phillips KD, Woodall S, Arnold J, Tucker K, Field M, Greening S, Gallinger S, Aronson M, Perrier R, Woods MO, Green JS, Walker N, Rosty C, Parry S, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The American journal of gastroenterology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">107</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/ajg.2012.52 ajg201252 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22525305</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/ajg.2012.52 ajg201252 [pii]">http://dx.doi.org/10.1038/ajg.2012.52 ajg201252 [pii]</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenocarcinoma/genetics Adenoma/genetics Colonic Polyps/*genetics/pathology Colorectal Neoplasms/genetics Female Humans Male Middle Aged Neoplasms/*genetics Pancreatic Neoplasms/genetics Risk</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">OBJECTIVES: Serrated polyposis (hyperplastic polyposis) is characterized by multiple polyps with serrated architecture in the colorectum. Although patients with serrated polyposis are known to be at increased risk of colorectal cancer (CRC) and possibly extracolonic cancers, cancer risk for their relatives has not been widely explored. The aim of this study was to estimate the risks of CRC and extracolonic cancers for relatives of patients with serrated polyposis. METHODS: A cohort of the 1,639 first- and second-degree relatives of 100 index patients with serrated polyposis recruited regardless of a family history of polyps or cancer from genetic clinics in Australia, New Zealand, Canada, and the USA, were retrospectively analyzed to estimate the country-, age-, and sex-specific standardized incidence ratios (SIRs) for relatives compared with the general population. RESULTS: A total of 102 CRCs were observed in first- and second-relatives (SIR 2.25, 95% confidence interval (CI) 1.75-2.93; P<0.001), with 54 in first-degree relatives (SIR 5.16, 95% CI 3.70-7.30; P<0.001) and 48 in second-degree relatives (SIR 1.38, 95% CI 1.01-1.91; P=0.04). Six pancreatic cancers were observed in first-degree relatives (SIR 3.64, 95% CI 1.70-9.21; P=0.003). There was no statistical evidence of increased risk for cancer of the stomach, brain, breast, or prostate. CONCLUSIONS: Our finding that relatives of serrated polyposis patients are at significantly increased risk of colorectal and pancreatic cancer adds to the accumulating evidence that serrated polyposis has an inherited component.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Colorectal and other cancer risks for carriers and noncarriers from families with a DNA mismatch repair gene mutation: a prospective cohort study</a>, Journal of clinical oncology: official journal of the American Society of Clinical Oncology, 2012.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Young JP, Lindor NM, Tucker KM, Ahnen DJ, Young GP, Buchanan DD, Clendenning M, Giles GG, Winship I, Macrae FA, Goldblatt J, Southey MC, Arnold J, Thibodeau SN, Gunawardena SR, Bapat B, Baron JA, Casey G, Gallinger S, Le Marchand L, Newcomb PA, Haile RW, Hopper JL, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of clinical oncology: official journal of the American Society of Clinical Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">30</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">9</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2012</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">22331944</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/22331944">https://www.ncbi.nlm.nih.gov/pubmed/22331944</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/genetics Adenosine Triphosphatases/genetics Adult Aged Aged, 80 and over Australia/epidemiology Colorectal Neoplasms/epidemiology/*etiology/mortality DNA Mismatch Repair/*genetics DNA Repair Enzymes/genetics DNA-Binding Proteins/genetics Female Follow-Up Studies *Genetic Predisposition to Disease Heterozygote Humans Incidence Male Middle Aged MutS Homolog 2 Protein/genetics Mutation/*genetics Neoplasms/epidemiology/*etiology/mortality Nuclear Proteins/genetics Prognosis Prospective Studies Risk Factors Survival Rate</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">PURPOSE: To determine whether cancer risks for carriers and noncarriers from families with a mismatch repair (MMR) gene mutation are increased above the risks of the general population. PATIENTS AND METHODS: We prospectively followed a cohort of 446 unaffected carriers of an MMR gene mutation (MLH1, n = 161; MSH2, n = 222; MSH6, n = 47; and PMS2, n = 16) and 1,029 their unaffected relatives who did not carry a mutation every 5 years at recruitment centers of the Colon Cancer Family Registry. For comparison of cancer risk with the general population, we estimated country-, age-, and sex-specific standardized incidence ratios (SIRs) of cancer for carriers and noncarriers. RESULTS: Over a median follow-up of 5 years, mutation carriers had an increased risk of colorectal cancer (CRC; SIR, 20.48; 95% CI, 11.71 to 33.27; P < .001), endometrial cancer (SIR, 30.62; 95% CI, 11.24 to 66.64; P < .001), ovarian cancer (SIR, 18.81; 95% CI, 3.88 to 54.95; P < .001), renal cancer (SIR, 11.22; 95% CI, 2.31 to 32.79; P < .001), pancreatic cancer (SIR, 10.68; 95% CI, 2.68 to 47.70; P = .001), gastric cancer (SIR, 9.78; 95% CI, 1.18 to 35.30; P = .009), urinary bladder cancer (SIR, 9.51; 95% CI, 1.15 to 34.37; P = .009), and female breast cancer (SIR, 3.95; 95% CI, 1.59 to 8.13; P = .001). We found no evidence of their noncarrier relatives having an increased risk of any cancer, including CRC (SIR, 1.02; 95% CI, 0.33 to 2.39; P = .97). CONCLUSION: We confirmed that carriers of an MMR gene mutation were at increased risk of a wide variety of cancers, including some cancers not previously recognized as being a result of MMR mutations, and found no evidence of an increased risk of cancer for their noncarrier relatives.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2011</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Quality assessment and correlation of microsatellite instability and immunohistochemical markers among population- and clinic-based colorectal tumors results from the Colon Cancer Family Registry</a>, The Journal of molecular diagnostics: JMD, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Cicek MS, Lindor NM, Gallinger S, Bapat B, Hopper JL, Jenkins MA, Young J, Buchanan D, Walsh MD, Le Marchand L, Burnett T, Newcomb PA, Grady WM, Haile RW, Casey G, Plummer SJ, Krumroy LA, Baron JA, Thibodeau SN</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">The Journal of molecular diagnostics: JMD</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">13</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21497289</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/21497289">https://www.ncbi.nlm.nih.gov/pubmed/21497289</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adolescent Adult Age Factors Aged Aged, 80 and over Colorectal Neoplasms/diagnosis/*genetics/*metabolism Female Genetic Association Studies Humans Male *Microsatellite Instability Microsatellite Repeats/genetics Middle Aged Phenotype Quality Assurance, Health Care Reference Values *Registries Reproducibility of Results Sensitivity and Specificity Sex Factors Tumor Markers, Biological/*genetics/*metabolism Young Adult</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The detection of defective mismatch repair (MMR), as assessed by the presence of tumor microsatellite instability (MSI) and/or loss of MMR protein expression by IHC, has been useful for risk assessment, prognosis, and prediction of treatment in patients with colorectal cancer. We analyzed tumors for the presence of defective MMR from 5927 Colorectal Cancer Family Registry patients recruited at six international consortium sites. We evaluated the appropriate percentage instability cutoff used to distinguish the three MSI phenotypes [ie, stable (MSS), low instability (MSI-L), and high instability (MSI-H)]; the sensitivity, specificity, and performance characteristics of individual markers; and the concordance between MSI and IHC phenotypes. Guided by the results of the IHC testing, our findings indicate that the distinction between an MSI-H phenotype from a low-instability or MSS phenotype can best be accomplished by using a cutoff of 30% or greater of the markers showing instability. The sensitivity and specificity of the mononucleotide markers were higher than those of the dinucleotide markers. Specifically, BAT26 and BAT25 had the highest sensitivity (94%) and specificity (98%), and the use of mononucleotide markers alone identified 97% of the MSI-H cases correctly. As expected, the presence of MSI-H correlated with an older age of diagnosis, the presence of tumor in the proximal colon, and female sex.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Mutation deep within an intron of MSH2 causes Lynch syndrome</a>, Familial cancer, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Clendenning M, Buchanan DD, Walsh MD, Nagler B, Rosty C, Thompson B, Spurdle AB, Hopper JL, Jenkins MA, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-011-9427-0</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21360204</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-011-9427-0">http://dx.doi.org/10.1007/s10689-011-9427-0</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Lynch syndrome, a heritable form of cancer predisposition, is caused by germline mutations within genes of the DNA mismatch repair family, and can be rapidly identified in young onset cancer patients through the detection of loss of expression of at least one of these genes in tumour samples. To date, such causative mutations have only been identified within exonic and splice site regions. Though this approach has been successful in the majority of families, a considerable number remain in which no mutation has been found. To address this situation, we used an alternative mutation discovery procedure which involved haplotype analysis of the locus containing the gene lost in the tumour and delineation of segregating haplotypes, followed by an investigation of splicing aberrations to uncover cryptic splice sites which lay outside the genomic regions routinely examined for mutations. In this report, we show that an intronic mutation 478 bp upstream of exon 2 in the MSH2 gene causes Lynch syndrome through creation of a novel splice donor site with subsequent pseudoexon activation, thus highlighting the need for more extensive sequencing approaches in families where routine procedures fail to find a mutation.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Promoter methylation of Wnt antagonists DKK1 and SFRP1 is associated with opposing tumor subtypes in two large populations of colorectal cancer patients</a>, Carcinogenesis, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rawson JB, Manno M, Mrkonjic M, Daftary D, Dicks E, Buchanan DD, Younghusband HB, Parfrey PS, Young JP, Pollett A, Green RC, Gallinger S, McLaughlin JR, Knight JA, Bapat B</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Carcinogenesis</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">32</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21304055</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/21304055">https://www.ncbi.nlm.nih.gov/pubmed/21304055</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Aberrant activation of canonical Wnt signaling is a hallmark event in colorectal carcinogenesis. The Dickkopf-1 (DKK1) and Secreted Frizzled Related Protein 1 (SFRP1) genes encode extracellular inhibitors of Wnt signaling that are frequently silenced by promoter hypermethylation in colorectal cancer (CRC). These methylation events have been identified as prognostic markers of patient outcome and tumor subtype in several cancers but similar roles in CRC have not been comprehensively examined. In CRC, the microsatellite instability (MSI) subtype associates with favorable disease outcome but the molecular events that are responsible remain poorly understood. Consequently, we quantified promoter methylation status of the Wnt antagonist genes DKK1 and SFRP1 in a large population-based cohort of CRCs from Ontario (n = 549) and Newfoundland (n = 696) stratified by MSI status. We examined the association between methylation status and clinicopathlogical features including tumor MSI status and patient survival. DKK1 and SFRP1 were methylated in 13 and 95% of CRCs, respectively. In Ontario, DKK1 methylation was strongly associated with MSI tumors after adjustment for age, sex and tumor location [odds ratio (OR) = 13.7, 95% confidence interval (CI) = 7.8-24.2, P < 0.001]. Conversely, SFRP1 methylation was inversely associated with MSI tumors after these adjustments (OR = 0.3, 95% CI = 0.1-0.9, P = 0.009). Similar results were obtained in Newfoundland. There were no independent associations with recurrence-free survival. This is the first large study to identify associations between Wnt antagonist promoter hypermethylation and CRC MSI subtype. These events provide insight into subtype-specific epigenetic mediation of Wnt signaling in CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Promoter methylation of Wnt5a is associated with microsatellite instability and BRAF V600E mutation in two large populations of colorectal cancer patients</a>, British journal of cancer, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rawson JB, Mrkonjic M, Daftary D, Dicks E, Buchanan DD, Younghusband HB, Parfrey PS, Young JP, Pollett A, Green RC, Gallinger S, McLaughlin JR, Knight JA, Bapat B</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">British journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">104</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/bjc.2011.165 bjc2011165 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21587258</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/bjc.2011.165 bjc2011165 [pii]">http://dx.doi.org/10.1038/bjc.2011.165 bjc2011165 [pii]</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Background:In colorectal cancer (CRC), tumour microsatellite instability (MSI) status and CpG island methylator phenotype (CIMP) status are indicators of patient outcome, but the molecular events that give rise to these outcomes remain largely unknown. Wnt5a is a critical regulator of non-canonical Wnt activity and promoter hypermethylation of this gene has emerging prognostic roles in CRC; however the frequency and prognostic significance of this epigenetic event have not been explored in the context of colorectal tumour subtype. Consequently, we investigated the frequency and prognostic significance of Wnt5a methylation in a large cohort of MSI-stratified CRCs.Methods:Methylation was quantified in a large cohort of 1232 colorectal carcinomas from two clinically distinct populations from Canada. Associations were examined between methylation status and clinicopathlogical features, including tumour MSI status, BRAF V600E mutation, and patient survival.Results:In Ontario, Wnt5a methylation was strongly associated with MSI tumours after adjustment for age, sex, and tumour location (odds ratio (OR)=4.2, 95% confidence interval (CI)=2.4-7.4, P<10(-6)) and with BRAF V600E mutation, a marker of CIMP (OR=12.3, 95% CI=6.9-21.7, P<10(-17)), but was not associated with patient survival. Concordant results were obtained in Newfoundland.Conclusion:Methylation of Wnt5a is associated with distinct tumour subtypes, strengthening the evidence of an epigenetic-mediated Wnt bias in CRC.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Linkage to chromosome 2q32.2-q33.3 in familial serrated neoplasia (Jass syndrome)</a>, Familial cancer, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Roberts A, Nancarrow D, Clendenning M, Buchanan DD, Jenkins MA, Duggan D, Taverna D, McKeone D, Walters R, Walsh MD, Young BW, Jass JR, Rosty C, Gattas M, Pelzer E, Hopper JL, Goldblatt J, George J, Suthers GK, Phillips K, Parry S, Woodall S, Arnold J, Tucker K, Muir A, Drini M, Macrae F, Newcomb P, Potter JD, Pavluk E, Lindblom A, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-010-9408-8</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21165777</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-010-9408-8">http://dx.doi.org/10.1007/s10689-010-9408-8</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Chromosome Mapping *Chromosomes, Human, Pair 2 Colorectal Neoplasms/*genetics Female *Genetic Linkage *Genetic Predisposition to Disease Genome-Wide Association Study Haplotypes Humans Lod Score Male Middle Aged Syndrome</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Causative genetic variants have to date been identified for only a small proportion of familial colorectal cancer (CRC). While conditions such as Familial Adenomatous Polyposis and Lynch syndrome have well defined genetic causes, the search for variants underlying the remainder of familial CRC is plagued by genetic heterogeneity. The recent identification of families with a heritable predisposition to malignancies arising through the serrated pathway (familial serrated neoplasia or Jass syndrome) provides an opportunity to study a subset of familial CRC in which heterogeneity may be greatly reduced. A genome-wide linkage screen was performed on a large family displaying a dominantly-inherited predisposition to serrated neoplasia genotyped using the Affymetrix GeneChip Human Mapping 10 K SNP Array. Parametric and nonparametric analyses were performed and resulting regions of interest, as well as previously reported CRC susceptibility loci at 3q22, 7q31 and 9q22, were followed up by finemapping in 10 serrated neoplasia families. Genome-wide linkage analysis revealed regions of interest at 2p25.2-p25.1, 2q24.3-q37.1 and 8p21.2-q12.1. Finemapping linkage and haplotype analyses identified 2q32.2-q33.3 as the region most likely to harbour linkage, with heterogeneity logarithm of the odds (HLOD) 2.09 and nonparametric linkage (NPL) score 2.36 (P = 0.004). Five primary candidate genes (CFLAR, CASP10, CASP8, FZD7 and BMPR2) were sequenced and no segregating variants identified. There was no evidence of linkage to previously reported loci on chromosomes 3, 7 and 9.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Hyperplastic polyp of the duodenum: a report of 9 cases with immunohistochemical and molecular findings</a>, Human pathology, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Rosty C, Buchanan DD, Walters RJ, Carr NJ, Bothman JW, Young JP, Brown IS</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human pathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">42</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">12</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21733555</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/21733555">https://www.ncbi.nlm.nih.gov/pubmed/21733555</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged DNA, Neoplasm/genetics Diagnosis, Differential Duodenal Diseases/genetics/*pathology Duodenum/pathology Female Humans Hyperplasia Immunohistochemistry Intestinal Polyps/genetics/*pathology Male Middle Aged Mucins/genetics/*metabolism Mutation Proto-Oncogene Proteins/*genetics Proto-Oncogene Proteins B-raf/*genetics Young Adult ras Proteins/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Benign serrated polyps are commonly found in the colorectum but have rarely been described in other parts of the gastrointestinal tract. We report a series of 9 serrated polyps arising in the duodenum with clinicopathologic features, immunohistochemical expression profile of mucins (MUC2, MUC5AC, MUC6), and molecular analysis for BRAF and KRAS. The polyps were diagnosed as incidental endoscopy findings in 9 different patients, comprising 3 male and 6 female patients, with a mean age of 52.2 years (range, 21-72 years). The second part of the duodenum was the most common site (n = 5), followed by the ampulla (n = 1) and the distal duodenum (n = 1), with the location of the 2 remaining polyps unspecified. Other upper gastrointestinal tract pathology features included Barrett esophagus for 5 patients, Helicobacter gastritis for 1 patient, and mild chronic gastritis for 1 patient. The histologic appearance of the polyps was similar to microvesicular hyperplastic polyp in the colorectum. Immunostaining for mucins showed MUC6 expression in the crypt bases of all polyps, MUC5AC expression in 8 cases (89%), and mucin 2 expression in 6 cases (67%). Molecular testing was successful in 6 polyps, showing BRAF mutation (V600E) in 2 polyps, KRAS mutation in 2 polyps, and no mutation for either gene in 2 polyps. Colonoscopy reports were available for 6 patients, of whom 4 were diagnosed with hyperplastic polyps or sessile serrated polyps in the colorectum. However, no patient met the criteria for serrated polyposis. Although probably rare and of uncertain malignant potential, hyperplastic polyp should be considered in the differential diagnosis of benign duodenal polyp.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Cancer risks for monoallelic MUTYH mutation carriers with a family history of colorectal cancer</a>, International journal of cancer. Journal international du cancer, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Cleary SP, Dowty JG, Baron JA, Young JP, Buchanan DD, Southey MC, Burnett T, Parfrey PS, Green RC, Marchand LL, Newcomb PA, Haile RW, Lindor NM, Hopper JL, Gallinger S, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of cancer. Journal international du cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">129</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">9</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ijc.25870</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21171015</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ijc.25870">http://dx.doi.org/10.1002/ijc.25870</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Cancer risks for a person who has inherited a MUTYH mutation from only one parent (monoallelic mutation carrier) are uncertain. Using the Colon Cancer Family Registry and Newfoundland Familial Colon Cancer Registry, we identified 2,179 first- and second-degree relatives of 144 incident colorectal cancer (CRC) cases who were monoallelic or biallelic mutation carriers ascertained by sampling population complete cancer registries in the United States, Canada and Australia. Using Cox regression weighted to adjust for sampling on family history, we estimated that the country-, age- and sex-specific standardized incidence ratios (SIRs) for monoallelic mutation carriers, compared to the general population, were: 2.04 (95% confidence interval, CI 1.56-2.70; p < 0.001) for CRC, 3.24 (95%CI 2.18-4.98; p < 0.001) for gastric cancer, 3.09 (95%CI 1.07-12.25; p = 0.07) for liver cancer and 2.33 (95%CI 1.18-5.08; p = 0.02) for endometrial cancer. Age-specific cumulative risks to age 70 years, estimated using the SIRs and US population incidences, were: for CRC, 6% (95%CI 5-8%) for men and 4% (95%CI 3-6%) for women; for gastric cancer, 2% (95%CI 1-3%) for men and 0.7% (95%CI 0.5-1%) for women; for liver cancer, 1% (95%CI 0.3-3%) for men and 0.3% (95%CI 0.1-1%) for women and for endometrial cancer, 4% (95%CI 2-8%). There was no evidence of increased risks for cancers of the brain, pancreas, kidney, lung, breast or prostate. Monoallelic MUTYH mutation carriers with a family history of CRC, such as those identified from screening multiple-case CRC families, are at increased risk of colorectal, gastric, endometrial and possibly liver cancers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Body mass index in early adulthood and colorectal cancer risk for carriers and non-carriers of germline mutations in DNA mismatch repair genes</a>, British journal of cancer, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Dowty JG, English DR, Campbell PT, Young JP, Winship I, Macrae FA, Lipton L, Parry S, Young GP, Buchanan DD, Martinez ME, Jacobs ET, Ahnen DJ, Haile RW, Casey G, Baron JA, Lindor NM, Thibodeau SN, Newcomb PA, Potter JD, Le Marchand L, Gallinger S, Hopper JL, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">British journal of cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">105</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1038/bjc.2011.172 bjc2011172 [pii]</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21559014</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1038/bjc.2011.172 bjc2011172 [pii]">http://dx.doi.org/10.1038/bjc.2011.172 bjc2011172 [pii]</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Carriers of germline mutations in DNA mismatch repair (MMR) genes have a high risk of colorectal cancer (CRC), but the modifiers of this risk are not well established. We estimated an association between body mass index (BMI) in early adulthood and subsequent risk of CRC for carriers and, as a comparison, estimated the association for non-carriers. METHODS: A weighted Cox regression was used to analyse height and weight at 20 years reported by 1324 carriers of MMR gene mutations (500 MLH1, 648 MSH2, 117 MSH6 and 59 PMS2) and 1219 non-carriers from the Colon Cancer Family Registry. RESULTS: During 122,304 person-years of observation, we observed diagnoses of CRC for 659 carriers (50%) and 36 non-carriers (3%). For carriers, the risk of CRC increased by 30% for each 5 kg m(-2) increment in BMI in early adulthood (hazard ratio, HR: 1.30; 95% confidence interval, CI: 1.08-1.58; P=0.01), and increased by 64% for non-carriers (HR: 1.64; 95% CI: 1.02-2.64; P=0.04) after adjusting for sex, country, cigarette smoking and alcohol drinking (and the MMR gene that was mutated in carriers). The difference in HRs for carriers and non-carriers was not statistically significant (P=0.50). For MLH1 and PMS2 (MutLalpha heterodimer) mutation carriers combined, the corresponding increase was 36% (HR: 1.36; 95% CI: 1.05-1.76; P=0.02). For MSH2 and MSH6 (MutSalpha heterodimer) mutation carriers combined, the HR was 1.26 (95% CI: 0.96-1.65; P=0.09). There was no significant difference between the HRs for MutLalpha and MutSalpha heterodimer carriers (P=0.56). CONCLUSION: Body mass index in early adulthood is positively associated with risk of CRC for MMR gene mutation carriers and non-carriers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Determining the frequency of de novo germline mutations in DNA mismatch repair genes</a>, Journal of medical genetics, 2011.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Win AK, Jenkins MA, Buchanan DD, Clendenning M, Young JP, Giles GG, Goldblatt J, Leggett BA, Hopper JL, Thibodeau SN, Lindor NM</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of medical genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">48</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2011</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">21636617</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/21636617">https://www.ncbi.nlm.nih.gov/pubmed/21636617</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Background Carriers of a germline mutation in a DNA mismatch repair (MMR) gene-that is, persons with Lynch syndrome-have substantially high risks of colorectal (CRC), endometrial, and several other cancers. The proportion of carriers who have de novo mutations (not inherited from either parent) is not known. This study reports a case series of de novo mutations in MMR genes and estimates the frequency of de novo mutation in MMR genes using the Colon Cancer Family Registry. Methods Screening for germline MLH1, MSH2, MSH6, and PMS2 mutations was performed for all incident CRC cases recruited from cancer registries (population based probands) displaying microsatellite instability (MSI) or loss of expression of MMR genes by immunohistochemistry (IHC) and probands with CRC in multi-case families recruited from clinics (clinic based probands), regardless of MSI or IHC status. All relatives of probands with a pathogenic mutation who donated a blood sample underwent testing for the mutation identified in the proband. Results Of 261 probands (202 clinic based, 59 population based) with MMR gene mutations for whom it was possible to determine the origin of the mutation, six (2.3%, 95% CI 0.9% to 5.0%) were confirmed as de novo, and the remaining 255 (97.7%, 95% CI 95.0% to 99.1%) were inherited. Of the de novo mutation carriers, three were clinic based probands (1.5%, 95% CI 0.3% to 4.5%) and three were population based probands (5.1%, 95% CI 1.2% to 14.5%). Two were in MLH1, three in MSH2, and one in MSH6. Conclusion De novo MMR gene mutations are uncommon causes of Lynch syndrome.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2010</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risks of Lynch syndrome cancers for MSH6 mutation carriers</a>, Journal of the National Cancer Institute, 2010.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Baglietto L, Lindor NM, Dowty JG, White DM, Wagner A, Gomez Garcia EB, Vriends AH, Cartwright NR, Barnetson RA, Farrington SM, Tenesa A, Hampel H, Buchanan D, Arnold S, Young J, Walsh MD, Jass J, Macrae F, Antill Y, Winship IM, Giles GG, Goldblatt J, Parry S, Suthers G, Leggett B, Butz M, Aronson M, Poynter JN, Baron JA, Le Marchand L, Haile R, Gallinger S, Hopper JL, Potter J, de la Chapelle A, Vasen HF, Dunlop MG, Thibodeau SN, Jenkins MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of the National Cancer Institute</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">102</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2010</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">20028993</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/20028993">https://www.ncbi.nlm.nih.gov/pubmed/20028993</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Age Distribution Age Factors Aged Aged, 80 and over Australia/epidemiology Canada/epidemiology Colorectal Neoplasms, Hereditary Nonpolyposis/*epidemiology/*genetics DNA-Binding Proteins/*genetics Endometrial Neoplasms/epidemiology/genetics Europe/epidemiology Female Gene Deletion *Germ-Line Mutation *Heterozygote Humans Incidence Male Middle Aged Mutagenesis, Insertional Neoplasms/epidemiology/genetics New Zealand/epidemiology Registries Risk Assessment Risk Factors Sex Distribution Sex Factors United States/epidemiology</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Germline mutations in MSH6 account for 10%-20% of Lynch syndrome colorectal cancers caused by hereditary DNA mismatch repair gene mutations. Because there have been only a few studies of mutation carriers, their cancer risks are uncertain. METHODS: We identified 113 families of MSH6 mutation carriers from five countries that we ascertained through family cancer clinics and population-based cancer registries. Mutation status, sex, age, and histories of cancer, polypectomy, and hysterectomy were sought from 3104 of their relatives. Age-specific cumulative risks for carriers and hazard ratios (HRs) for cancer risks of carriers, compared with those of the general population of the same country, were estimated by use of a modified segregation analysis with appropriate conditioning depending on ascertainment. RESULTS: For MSH6 mutation carriers, the estimated cumulative risks to ages 70 and 80 years, respectively, were as follows: for colorectal cancer, 22% (95% confidence interval [CI] = 14% to 32%) and 44% (95% CI = 28% to 62%) for men and 10% (95% CI = 5% to 17%) and 20% (95% CI = 11% to 35%) for women; for endometrial cancer, 26% (95% CI = 18% to 36%) and 44% (95% CI = 30% to 58%); and for any cancer associated with Lynch syndrome, 24% (95% CI = 16% to 37%) and 47% (95% CI = 32% to 66%) for men and 40% (95% CI = 32% to 52%) and 65% (95% CI = 53% to 78%) for women. Compared with incidence for the general population, MSH6 mutation carriers had an eightfold increased incidence of colorectal cancer (HR = 7.6, 95% CI = 5.4 to 10.8), which was independent of sex and age. Women who were MSH6 mutation carriers had a 26-fold increased incidence of endometrial cancer (HR = 25.5, 95% CI = 16.8 to 38.7) and a sixfold increased incidence of other cancers associated with Lynch syndrome (HR = 6.0, 95% CI = 3.4 to 10.7). CONCLUSION: We have obtained precise and accurate estimates of both absolute and relative cancer risks for MSH6 mutation carriers.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lessons from Lynch syndrome: a tumor biology-based approach to familial colorectal cancer</a>, Future oncology , 2010.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Roberts A, Walsh MD, Parry S, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Future oncology </dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2010</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.2217/fon.10.16</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">20373868</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.2217/fon.10.16">http://dx.doi.org/10.2217/fon.10.16</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenoma/pathology Colorectal Neoplasms, Hereditary Nonpolyposis/*genetics/*pathology Female *Genetic Predisposition to Disease Humans Male</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Colorectal cancer (CRC) develops within precursor lesions in the single-celled epithelial lining of the gut. The two most common epithelial lesions are the adenoma and the serrated polyp. CRC is also one of the most familial of the common cancers, and just as there are syndromes associated with increased risk of CRC arising in adenomas, there are also syndromes with increased CRC risk associated with serrated polyps. In this article, we describe the features of such a syndrome, familial serrated neoplasia, which distinguish it from the well-characterized condition Lynch syndrome (or hereditary nonpolyposis CRC), and show that the molecular pathology of tumors forms the basis for this distinction. Lynch syndrome CRC arises almost exclusively within adenomatous precursor lesions, in contrast with familial serrated neoplasia where at least half of the cancers develop in serrated polyps. Finally, rare families exist in which both conditions segregate independently, producing a difficult diagnostic picture.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Risk factors for colorectal cancer in patients with multiple serrated polyps: a cross-sectional case series from genetics clinics</a>, PloS one, 2010.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Sweet K, Drini M, Jenkins MA, Win AK, English DR, Walsh MD, Clendenning M, McKeone DM, Walters RJ, Roberts A, Pearson SA, Pavluk E, Hopper JL, Gattas MR, Goldblatt J, George J, Suthers GK, Phillips KD, Woodall S, Arnold J, Tucker K, Muir A, Field M, Greening S, Gallinger S, Perrier R, Baron JA, Potter JD, Haile R, Frankel W, de la Chapelle A, Macrae F, Rosty C, Walker NI, Parry S, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">PloS one</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2010</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1371/journal.pone.0011636</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">20661287</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1371/journal.pone.0011636">http://dx.doi.org/10.1371/journal.pone.0011636</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Colonic Polyps/*complications/*epidemiology Colorectal Neoplasms/*epidemiology/*etiology Female Humans Logistic Models Male Middle Aged Odds Ratio Risk Factors Sex Factors Smoking/adverse effects</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Patients with multiple serrated polyps are at an increased risk for developing colorectal cancer (CRC). Recent reports have linked cigarette smoking with the subset of CRC that develops from serrated polyps. The aim of this work therefore was to investigate the association between smoking and the risk of CRC in high-risk genetics clinic patients presenting with multiple serrated polyps. METHODS AND FINDINGS: We identified 151 Caucasian individuals with multiple serrated polyps including at least 5 outside the rectum, and classified patients into non-smokers, current or former smokers at the time of initial diagnosis of polyposis. Cases were individuals with multiple serrated polyps who presented with CRC. Controls were individuals with multiple serrated polyps and no CRC. Multivariate logistic regression was performed to estimate associations between smoking and CRC with adjustment for age at first presentation, sex and co-existing traditional adenomas, a feature that has been consistently linked with CRC risk in patients with multiple serrated polyps. CRC was present in 56 (37%) individuals at presentation. Patients with at least one adenoma were 4 times more likely to present with CRC compared with patients without adenomas (OR = 4.09; 95%CI 1.27 to 13.14; P = 0.02). For females, the odds of CRC decreased by 90% in current smokers as compared to never smokers (OR = 0.10; 95%CI 0.02 to 0.47; P = 0.004) after adjusting for age and adenomas. For males, there was no relationship between current smoking and CRC. There was no statistical evidence of an association between former smoking and CRC for both sexes. CONCLUSION: A decreased odds for CRC was identified in females with multiple serrated polyps who currently smoke, independent of age and the presence of a traditional adenoma. Investigations into the biological basis for these observations could lead to non-smoking-related therapies being developed to decrease the risk of CRC and colectomy in these patients.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Phenotypic diversity in patients with multiple serrated polyps: a genetics clinic study</a>, International journal of colorectal disease, 2010.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Sweet K, Drini M, Jenkins MA, Win AK, Gattas M, Walsh MD, Clendenning M, McKeone D, Walters R, Roberts A, Young A, Hampel H, Hopper JL, Goldblatt J, George J, Suthers GK, Phillips K, Young GP, Chow E, Parry S, Woodall S, Tucker K, Muir A, Field M, Greening S, Gallinger S, Green J, Woods MO, Spaetgens R, de la Chapelle A, Macrae F, Walker NI, Jass JR, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">International journal of colorectal disease</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">25</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2010</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s00384-010-0907-8</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">20213458</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s00384-010-0907-8">http://dx.doi.org/10.1007/s00384-010-0907-8</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colonic Polyps/*genetics/*pathology Female Humans Male Middle Aged Phenotype Regression Analysis Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">OBJECTIVE: Hyperplastic polyposis is a colonic polyposis condition of unknown aetiology. The purpose of this study was to examine the spectrum of phenotypic variation in patients with multiple serrated polyps as a basis for gene discovery. METHODS: One hundred and twenty-six patients with multiple (> or = 5) serrated polyps were recruited to the study. Polyp counts were extracted from histology and colonoscopy reports. Ethnicity was self-reported. Family history of cancer data were derived from pedigrees. Ascertainment status was classified as either index case or identified by screening. RESULTS: The average reported polyp count was 39. Patients with highest polyp numbers were more likely to be male (P = 0.02). Colorectal cancer (CRC) was identified in 49 of 119 patients (41%) and 28% of these patients had multiple CRC. Young onset patients had higher polyp numbers (P = 0.03) and were more likely to have their CRC in the distal colon (P = 0.02). CRC was significantly associated with the presence of adenomas (P = 0.03). Patients were divided into moderate polyposis (5-79 serrated polyps) and dense polyposis (80 or more) categories. The dense polyposis category was associated with a lack of family history for CRC (P = 0.034) and male gender (P = 0.014), independent of ascertainment status and recruitment site. CONCLUSION: Multiple serrated polyps were associated with an increased personal risk of CRC. A subset of patients with the highest polyp numbers was more likely to be male and to have no family history of CRC. This result suggests heterogeneous modes of inheritance and has implications for studies investigating the genetic basis of multiple serrated polyps.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Confirmation of linkage to and localization of familial colon cancer risk haplotype on chromosome 9q22</a>, Cancer research, 2010.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Gray-McGuire C, Guda K, Adrianto I, Lin CP, Natale L, Potter JD, Newcomb P, Poole EM, Ulrich CM, Lindor N, Goode EL, Fridley BL, Jenkins R, Le Marchand L, Casey G, Haile R, Hopper J, Jenkins M, Young J, Buchanan D, Gallinger S, Adams M, Lewis S, Willis J, Elston R, Markowitz SD, Wiesner GL</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer research</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">70</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">13</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2010</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">20551049</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/20551049">https://www.ncbi.nlm.nih.gov/pubmed/20551049</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">*Chromosomes, Human, Pair 9 Colonic Neoplasms/*genetics Genetic Linkage Genetic Predisposition to Disease Haplotypes Humans Polymorphism, Single Nucleotide</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Genetic risk factors are important contributors to the development of colorectal cancer. Following the definition of a linkage signal at 9q22-31, we fine mapped this region in an independent collection of colon cancer families. We used a custom array of single-nucleotide polymorphisms (SNP) densely spaced across the candidate region, performing both single-SNP and moving-window association analyses to identify a colon neoplasia risk haplotype. Through this approach, we isolated the association effect to a five-SNP haplotype centered at 98.15 Mb on chromosome 9q. This haplotype is in strong linkage disequilibrium with the haplotype block containing HABP4 and may be a surrogate for the effect of this CD30 Ki-1 antigen. It is also in close proximity to GALNT12, also recently shown to be altered in colon tumors. We used a predictive modeling algorithm to show the contribution of this risk haplotype and surrounding candidate genes in distinguishing between colon cancer cases and healthy controls. The ability to replicate this finding, the strength of the haplotype association (odds ratio, 3.68), and the accuracy of our prediction model (approximately 60%) all strongly support the presence of a locus for familial colon cancer on chromosome 9q.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lynch syndrome-associated breast cancers: clinicopathologic characteristics of a case series from the colon cancer family registry</a>, Clinical cancer research: an official journal of the American Association for Cancer Research, 2010.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walsh MD, Buchanan DD, Cummings MC, Pearson SA, Arnold ST, Clendenning M, Walters R, McKeone DM, Spurdle AB, Hopper JL, Jenkins MA, Phillips KD, Suthers GK, George J, Goldblatt J, Muir A, Tucker K, Pelzer E, Gattas MR, Woodall S, Parry S, Macrae FA, Haile RW, Baron JA, Potter JD, Le Marchand L, Bapat B, Thibodeau SN, Lindor NM, McGuckin MA, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Clinical cancer research: an official journal of the American Association for Cancer Research</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">16</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2010</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">20215533</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/20215533">https://www.ncbi.nlm.nih.gov/pubmed/20215533</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Aged, 80 and over Breast Neoplasms/complications/*genetics/metabolism/pathology Breast Neoplasms, Male/genetics/metabolism Carcinoma/complications/*genetics/metabolism/pathology Case-Control Studies Colonic Neoplasms/complications/*genetics/metabolism/pathology Colorectal Neoplasms, Hereditary Nonpolyposis/complications/*genetics/metabolism/pathology DNA Mismatch Repair/genetics Family Female Genetic Predisposition to Disease Heterozygote Detection Humans Male Middle Aged Mutation *Registries</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">PURPOSE: The recognition of breast cancer as a spectrum tumor in Lynch syndrome remains controversial. The aim of this study was to explore features of breast cancers arising in Lynch syndrome families. EXPERIMENTAL DESIGN: This observational study involved 107 cases of breast cancer identified from the Colorectal Cancer Family Registry (Colon CFR) from 90 families in which (a) both breast and colon cancer co-occurred, (b) families met either modified Amsterdam criteria, or had at least one early-onset (<50 years) colorectal cancer, and (c) breast tissue was available within the biospecimen repository for mismatch repair (MMR) testing. Eligibility criteria for enrollment in the Colon CFR are available online. Breast cancers were reviewed by one pathologist. Tumor sections were stained for MLH1, PMS2, MSH2, and MSH6, and underwent microsatellite instability testing. RESULTS: Breast cancer arose in 35 mutation carriers, and of these, 18 (51%) showed immunohistochemical absence of MMR protein corresponding to the MMR gene mutation segregating the family. MMR-deficient breast cancers were more likely to be poorly differentiated (P = 0.005) with a high mitotic index (P = 0.002), steroid hormone receptor-negative (estrogen receptor, P = 0.031; progesterone receptor, P = 0.022), and to have peritumoral lymphocytes (P = 0.015), confluent necrosis (P = 0.002), and growth in solid sheets (P < 0.001) similar to their colorectal counterparts. No difference in age of onset was noted between the MMR-deficient and MMR-intact groups. CONCLUSIONS: MMR deficiency was identified in 51% of breast cancers arising in known mutation carriers. Breast cancer therefore may represent a valid tissue option for the detection of MMR deficiency in which spectrum tumors are lacking.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2009</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Classifying MLH1 and MSH2 variants using bioinformatic prediction, splicing assays, segregation, and tumor characteristics</a>, Human mutation, 2009.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Arnold S, Buchanan DD, Barker M, Jaskowski L, Walsh MD, Birney G, Woods MO, Hopper JL, Jenkins MA, Brown MA, Tavtigian SV, Goldgar DE, Young JP, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Human mutation</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">30</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2009</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/humu.20936</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">19267393</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/humu.20936">http://dx.doi.org/10.1002/humu.20936</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics Aged Biological Assay Chromosome Segregation/*genetics Colonic Neoplasms/*genetics *Computational Biology DNA, Complementary/genetics Female Heterozygote Humans Male Middle Aged MutS Homolog 2 Protein/*genetics Mutation/*genetics Nuclear Proteins/*genetics Polymerase Chain Reaction RNA Splicing/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Reliable methods for predicting functional consequences of variants in disease genes would be beneficial in the clinical setting. This study was undertaken to predict, and confirm in vitro, splicing aberrations associated with mismatch repair (MMR) variants identified in familial colon cancer patients. Six programs were used to predict the effect of 13 MLH1 and 6 MSH2 gene variants on pre-mRNA splicing. mRNA from cycloheximide-treated lymphoblastoid cell lines of variant carriers was screened for splicing aberrations. Tumors of variant carriers were tested for microsatellite instability and MMR protein expression. Variant segregation in families was assessed using Bayes factor causality analysis. Amino acid alterations were examined for evolutionary conservation and physicochemical properties. Splicing aberrations were detected for 10 variants, including a frameshift as a minor cDNA product, and altered ratio of known alternate splice products. Loss of splice sites was well predicted by splice-site prediction programs SpliceSiteFinder (90%) and NNSPLICE (90%), but consequence of splice site loss was less accurately predicted. No aberrations correlated with ESE predictions for the nine exonic variants studied. Seven of eight missense variants had normal splicing (88%), but only one was a substitution considered neutral from evolutionary/physicochemical analysis. Combined with information from tumor and segregation analysis, and literature review, 16 of 19 variants were considered clinically relevant. Bioinformatic tools for prediction of splicing aberrations need improvement before use without supporting studies to assess variant pathogenicity. Classification of mismatch repair gene variants is assisted by a comprehensive approach that includes in vitro, tumor pathology, clinical, and evolutionary conservation data.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>A perspective on bi-allelic MUTYH mutations in patients with hyperplastic polyposis syndrome</a>, Gastroenterology, 2009.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan D, Young J</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gastroenterology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">136</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2009</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=19406141">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=19406141</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenomatous Polyposis Coli/epidemiology/*genetics/*pathology Alleles Case-Control Studies DNA Glycosylases/*genetics DNA Mutational Analysis Female Gene Expression Regulation, Neoplastic Genetic Predisposition to Disease/*epidemiology Heterozygote Humans Incidence Male Reference Values Risk Assessment Sensitivity and Specificity</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Buchanan, Daniel Young, Joanne Comment Comparative Study Letter United States Gastroenterology Gastroenterology. 2009 Jun;136(7):2407-8. Epub 2009 May 3.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Screening PARK genes for mutations in early-onset Parkinson's disease patients from Queensland, Australia</a>, Parkinsonism & related disorders, 2009.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mellick GD, Siebert GA, Funayama M, Buchanan DD, Li Y, Imamichi Y, Yoshino H, Silburn PA, Hattori N</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Parkinsonism & related disorders</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">15</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2009</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">18486522</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/18486522">https://www.ncbi.nlm.nih.gov/pubmed/18486522</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Age of Onset Aged DNA Mutational Analysis/methods Exons/genetics Female Genetic Testing/methods Humans Intracellular Signaling Peptides and Proteins/genetics Male Middle Aged Mutation/*genetics Oncogene Proteins/genetics Parkinson Disease/*genetics Protein Kinases/*genetics Protein-Serine-Threonine Kinases/genetics Queensland Ubiquitin-Protein Ligases/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">A family history of Parkinson's disease (PD) is the most commonly reported risk factor after age, suggesting a genetic component to the disease in a sub-group of patients. Mutations in at least six genes have been identified that can lead to monogenic forms of PD. We screened a sample of 74 early-onset PD cases out of a cohort of 950 patients (onset <50 years) for genetic abnormalities in known familial Parkinsonism genes. A self-reported family history of PD existed for 30 patients (40.5%). Of these, 13 each had a first- or a second-degree relative with PD and four reported a more distant relative with PD. The entire coding region of the PRKN (MIM 602544), DJ-1 (MIM 602533) and PINK1 (MIM 698309) genes, and exon 41 of the LRRK2 gene (MIM 609007) were screened by direct sequencing. All exons of PRKN were examined for gene-dosage abnormalities. Screening identified five patients with putative genetic disease: two patients carried PRKN mutations (p.G12R heterozygous and p.G430D homozygous), one patient carried a p.G411S heterozygous amino acid change in the PINK1 gene and two individuals were heterozygous for the common p.G2019S mutation in LRRK2. No alpha-synuclein or DJ-1 variants were observed. Our data suggest that approximately 7% of early-onset PD cases seen in Queensland movement disorders clinics have mutations involving known PARK genes.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Analysis of families with Lynch syndrome complicated by advanced serrated neoplasia: the importance of pathology review and pedigree analysis</a>, Familial cancer, 2009.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walsh MD, Buchanan DD, Walters R, Roberts A, Arnold S, McKeone D, Clendenning M, Ruszkiewicz AR, Jenkins MA, Hopper JL, Goldblatt J, George J, Suthers GK, Phillips K, Young GP, Macrae F, Drini M, Woods MO, Parry S, Jass JR, Young JP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Familial cancer</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">8</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">4</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2009</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10689-009-9238-8</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">19241144</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10689-009-9238-8">http://dx.doi.org/10.1007/s10689-009-9238-8</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10"> we describe two Lynch syndrome families who demonstrated several such inconsistencies. In one family</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The identification of Lynch syndrome has been greatly assisted by the advent of tumour immunohistochemistry (IHC) for mismatch repair (MMR) proteins, and by the recognition of the role of acquired somatic BRAF mutation in sporadic MMR-deficient colorectal cancer (CRC). However, somatic BRAF mutation may also be present in the tumours in families with a predisposition to develop serrated polyps in the colorectum. In a subgroup of affected members in these families, CRCs emerge which demonstrate clear evidence of MMR deficiency with absent MLH1 staining and high-level microsatellite instability (MSI). This may result in these families being erroneously classified as Lynch syndrome, or conversely, an individual is considered \sporadic\" due to the presence of a somatic BRAF mutation in a tumour. In this report</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2008</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>A frame-shift mutation of PMS2 is a widespread cause of Lynch syndrome</a>, Journal of medical genetics, 2008.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Clendenning M, Senter L, Hampel H, Robinson KL, Sun S, Buchanan D, Walsh MD, Nilbert M, Green J, Potter J, Lindblom A, de la Chapelle A</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of medical genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">45</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2008</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=18178629">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=18178629</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenosine Triphosphatases/*genetics Adult Aged Base Sequence Colorectal Neoplasms, Hereditary Nonpolyposis/*genetics DNA Mutational Analysis DNA Repair Enzymes/*genetics DNA, Neoplasm/genetics DNA-Binding Proteins/*genetics Female Frameshift Mutation/*genetics Genetic Screening Genome, Human/genetics Haplotypes Humans Male Middle Aged Molecular Sequence Data</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: When compared to the other mismatch repair genes involved in Lynch syndrome, the identification of mutations within PMS2 has been limited (10 000 carriers of this mutation in the USA alone. The identification of both the mutation and the common haplotype in one Swedish control sample (n = 225), along with evidence that Lynch syndrome associated cancers are rarer than expected in the probands' families, would suggest that this is a prevalent mutation with reduced penetrance.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Ethnicity and risk for colorectal cancers showing somatic BRAF V600E mutation or CpG island methylator phenotype</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2008.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">English DR, Young JP, Simpson JA, Jenkins MA, Southey MC, Walsh MD, Buchanan DD, Barker MA, Haydon AM, Royce SG, Roberts A, Parry S, Hopper JL, Jass JJ, Giles GG</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">17</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2008</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">18628431</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/18628431">https://www.ncbi.nlm.nih.gov/pubmed/18628431</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Aged Colorectal Neoplasms/*ethnology/genetics CpG Islands/*genetics DNA, Neoplasm/*genetics *Ethnic Groups Female Follow-Up Studies Humans Incidence Male Middle Aged *Mutation Phenotype Prognosis Prospective Studies Proto-Oncogene Proteins B-raf/*genetics Risk Factors Victoria/epidemiology</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Colorectal cancers arising from serrated polyps are characterized by the CpG island methylator phenotype (CIMP) and somatic mutation (V600E) in the BRAF proto-oncogene. Few epidemiologic studies have investigated risk factors for these tumors. We conducted a cohort study of 41,328 residents of Melbourne, Australia that included 9,939 participants of southern European origin and 31,389 of Anglo-Celtic origin. Colorectal adenocarcinomas were identified from population-based cancer registries. BRAF V600E mutation in tumors was determined using a PCR-based allelic discrimination method. Tumors were classified as CIMP positive when at least three of five markers (RUNX3, CACNA1G, SOCS1, NEUROG1, and IGF2) were methylated according to MethyLight analysis. Hazard ratios (HR) and 95% confidence intervals (95% CI) were estimated by Cox regression with adjustment for risk factors for colorectal cancer. During follow-up, 718 participants were diagnosed with colorectal cancer. CIMP assays were done for 579 and BRAF V600E mutation testing for 582. After adjustment for other risk factors, when compared with people of Anglo-Celtic origin, those of southern European origin had lower incidence of colorectal cancer that had CIMP (HR, 0.32; 95% CI, 0.16-0.67) or BRAF mutations (HR, 0.30; 95% CI, 0.16-0.58) but similar incidence of colorectal cancer without CIMP (HR, 0.86; 95% CI, 0.70-1.05) or BRAF (HR, 0.90; 95% CI, 0.74-1.11). People of southern European origin had lower risk of colorectal cancers with CIMP and BRAF mutation than people of Anglo-Celtic origin, which may in part be due to genetic factors that are less common in people of southern European origin.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Molecular characterization of MSI-H colorectal cancer by MLHI promoter methylation, immunohistochemistry, and mismatch repair germline mutation screening</a>, Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology, 2008.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Poynter JN, Siegmund KD, Weisenberger DJ, Long TI, Thibodeau SN, Lindor N, Young J, Jenkins MA, Hopper JL, Baron JA, Buchanan D, Casey G, Levine AJ, Le Marchand L, Gallinger S, Bapat B, Potter JD, Newcomb PA, Haile RW, Laird PW</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Cancer epidemiology, biomarkers & prevention: a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">17</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">11</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2008</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=18990764">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=18990764</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/*genetics Aged Chi-Square Distribution Colorectal Neoplasms/*genetics *DNA Methylation DNA Mismatch Repair Female Genetic Predisposition to Disease Germ-Line Mutation Humans Immunohistochemistry Logistic Models Male *Microsatellite Instability Middle Aged Nuclear Proteins/*genetics Prevalence Registries</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Microsatellite instability (MSI) occurs in 10% to 20% of colorectal cancers (CRC) and has been attributed to both MLH1 promoter hypermethylation and germline mutation in the mismatch repair (MMR) genes. We present results from a large population- and clinic-based study of MLH1 methylation, immunohistochemistry, and MMR germline mutations that enabled us to (a) estimate the prevalence of MMR germline mutations and MLH1 methylation among MSI-H cases and help us understand if all MSI-H CRC is explained by these mechanisms and (b) estimate the associations between MLH1 methylation and sex, age, and tumor location within the colon. MLH1 methylation was measured in 1,061 population-based and 172 clinic-based cases of CRC. Overall, we observed MLH1 methylation in 60% of population-based MSI-H cases and in 13% of clinic-based MSI-H cases. Within the population-based cases with MMR mutation screening and conclusive immunohistochemistry results, we identified a molecular event in MMR in 91% of MSI-H cases: 54% had MLH1 methylation, 14% had a germline mutation in a MMR gene, and 23% had immunohistochemistry evidence for loss of a MMR protein. We observed a striking age difference, with the prevalence of a MMR germline mutation more than 4-fold lower and the prevalence of MLH1 methylation more than 4-fold higher in cases diagnosed after the age of 50 years than in cases diagnosed before that age. We also determined that female sex is an independent predictor of MLH1 methylation within the MSI-H subgroup. These results reinforce the importance of distinguishing between the underlying causes of MSI in studies of etiology and prognosis.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Molecular, pathologic, and clinical features of early-onset endometrial cancer: identifying presumptive Lynch syndrome patients</a>, Clinical cancer research: an official journal of the American Association for Cancer Research, 2008.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Walsh MD, Cummings MC, Buchanan DD, Dambacher WM, Arnold S, McKeone D, Byrnes R, Barker MA, Leggett BA, Gattas M, Jass JR, Spurdle AB, Young J, Obermair A</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Clinical cancer research: an official journal of the American Association for Cancer Research</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">14</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2008</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">18310315</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/18310315">https://www.ncbi.nlm.nih.gov/pubmed/18310315</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10"> and to identify molecular</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">PURPOSE: A woman with early-onset endometrial cancer (EC) may represent the \sentinel\" cancer event in a Lynch syndrome kindred. The aim of this study was to determine the incidence of Lynch syndrome in a series of young-onset EC</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2007</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Association of APOE with Parkinson disease age-at-onset in women</a>, Neuroscience letters, 2007.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Silburn PA, Prince JA, Mellick GD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Neuroscience letters</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">411</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2007</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">17113231</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/17113231">https://www.ncbi.nlm.nih.gov/pubmed/17113231</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">*Age of Onset Aged Analysis of Variance Apolipoproteins E/*genetics Case-Control Studies Chi-Square Distribution Female *Genetic Predisposition to Disease Humans Male Middle Aged Parkinson Disease/*genetics Polymorphism, Genetic *Sex Characteristics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">APOE polymorphism has received extensive attention as a risk factor for Parkinson's disease (PD), but findings have been equivocal. Analysis of APOE variants in an Australian PD case-control sample revealed a robust association between genotype and age-at-onset (AAO) of PD in women (P=0.0008). These data not only further implicate APOE in PD, but also provide a stark example of the effects that gender may play in complex disorders.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Stability of BAT26 in Lynch syndrome colorectal tumours</a>, European journal of human genetics: EJHG, 2007.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jaskowski L, Young J, Jackson L, Arnold S, Barker MA, Walsh MD, Buchanan DD, Holman S, Mensink KA, Jenkins MA, Hopper JL, Thibodeau SN, Jass JR, Spurdle AB</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">European journal of human genetics: EJHG</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">15</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2007</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">17133259</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/17133259">https://www.ncbi.nlm.nih.gov/pubmed/17133259</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">*Chromosomal Instability Colorectal Neoplasms/*diagnosis/genetics Colorectal Neoplasms, Hereditary Nonpolyposis/*diagnosis/genetics Humans Microsatellite Repeats Tumor Markers, Biological/analysis/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Jaskowski, Lesley Young, Joanne Jackson, Leigh Arnold, Sven Barker, Melissa A Walsh, Michael D Buchanan, Daniel D Holman, Samantha Mensink, Kara A Jenkins, Mark A Hopper, John L Thibodeau, Stephen N Jass, Jeremy R Spurdle, Amanda B CA-95-011/CA/NCI NIH HHS/United States U01 CA074800/CA/NCI NIH HHS/United States U01 CA097735/CA/NCI NIH HHS/United States Comment Letter Research Support, N.I.H., Extramural Research Support, Non-U.S. Gov't England European journal of human genetics : EJHG Eur J Hum Genet. 2007 Feb;15(2):139-41; author reply 141-2. Epub 2006 Nov 29.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Identification of BRCA1 missense substitutions that confer partial functional activity: potential moderate risk variants?</a>, Breast cancer research: BCR, 2007.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Lovelock PK, Spurdle AB, Mok MT, Farrugia DJ, Lakhani SR, Healey S, Arnold S, Buchanan D, Couch FJ, Henderson BR, Goldgar DE, Tavtigian SV, Chenevix-Trench G, Brown MA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Breast cancer research: BCR</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">9</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2007</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=18036263">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=18036263</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Alanine Arginine Breast Neoplasms/*chemistry/*genetics/pathology Centrosome DNA, Complementary Factor Analysis, Statistical Female *Genes, BRCA1 Genetic Predisposition to Disease Glutamic Acid Glutamine Glycine Humans Immunohistochemistry Keratins/analysis *Mutation, Missense Nucleic Acid Amplification Techniques Plasmids Polymorphism, Single Nucleotide Predictive Value of Tests Receptors, Estrogen/analysis Risk Assessment Risk Factors Sequence Analysis, DNA Tumor Markers, Biological/*analysis Valine</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">INTRODUCTION: Many of the DNA sequence variants identified in the breast cancer susceptibility gene BRCA1 remain unclassified in terms of their potential pathogenicity. Both multifactorial likelihood analysis and functional approaches have been proposed as a means to elucidate likely clinical significance of such variants, but analysis of the comparative value of these methods for classifying all sequence variants has been limited. METHODS: We have compared the results from multifactorial likelihood analysis with those from several functional analyses for the four BRCA1 sequence variants A1708E, G1738R, R1699Q, and A1708V. RESULTS: Our results show that multifactorial likelihood analysis, which incorporates sequence conservation, co-inheritance, segregation, and tumour immunohistochemical analysis, may improve classification of variants. For A1708E, previously shown to be functionally compromised, analysis of oestrogen receptor, cytokeratin 5/6, and cytokeratin 14 tumour expression data significantly strengthened the prediction of pathogenicity, giving a posterior probability of pathogenicity of 99%. For G1738R, shown to be functionally defective in this study, immunohistochemistry analysis confirmed previous findings of inconsistent 'BRCA1-like' phenotypes for the two tumours studied, and the posterior probability for this variant was 96%. The posterior probabilities of R1699Q and A1708V were 54% and 69%, respectively, only moderately suggestive of increased risk. Interestingly, results from functional analyses suggest that both of these variants have only partial functional activity. R1699Q was defective in foci formation in response to DNA damage and displayed intermediate transcriptional transactivation activity but showed no evidence for centrosome amplification. In contrast, A1708V displayed an intermediate transcriptional transactivation activity and a normal foci formation response in response to DNA damage but induced centrosome amplification. CONCLUSION: These data highlight the need for a range of functional studies to be performed in order to identify variants with partially compromised function. The results also raise the possibility that A1708V and R1699Q may be associated with a low or moderate risk of cancer. While data pooling strategies may provide more information for multifactorial analysis to improve the interpretation of the clinical significance of these variants, it is likely that the development of current multifactorial likelihood approaches and the consideration of alternative statistical approaches will be needed to determine whether these individually rare variants do confer a low or moderate risk of breast cancer.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2006</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Hyperplastic polyposis syndrome: phenotypic presentations and the role of MBD4 and MYH</a>, Gastroenterology, 2006.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Chow E, Lipton L, Lynch E, D'Souza R, Aragona C, Hodgkin L, Brown G, Winship I, Barker M, Buchanan D, Cowie S, Nasioulas S, du Sart D, Young J, Leggett B, Jass J, Macrae F</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gastroenterology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">131</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2006</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16831587">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16831587</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenomatous Polyposis Coli/*genetics/pathology Adult Aged Alleles Biopsy Colonoscopy DNA Glycosylases/*genetics DNA, Neoplasm/*genetics Endodeoxyribonucleases/*genetics Female Genetic Predisposition to Disease *Germ-Line Mutation Humans Male Middle Aged Pedigree Phenotype Polymerase Chain Reaction</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND & AIMS: Hyperplastic polyposis syndrome (HPS) is defined phenotypically with multiple, large and/or proximal hyperplastic polyps. There is no known germ-line predisposition. We aimed to characterize the clinicopathologic features of 38 patients with HPS and explore the role of germ-line mutations in the base excision repair genes MBD4 and MYH. METHODS: Utilizing clinical databases of The Royal Melbourne Hospital Bowel Cancer Surveillance Service and the Familial Cancer Clinic, 38 patients with HPS were recruited. The patients were analyzed for age at first diagnosis, features of hyperplastic polyposis, family histories of polyposis and colorectal cancer (CRC), coexisting adenomas, serrated adenomas, incidence of CRC, and microsatellite instability in the tumours. Mutation analysis of MBD4 and MYH were performed. RESULTS: Serrated adenomas were common (26%), and 19 (50%) of the 38 patients had a first-degree relative with CRC. Family history of HPS was uncommon, with only 2 cases found. Ten patients developed CRC, and 3 required surgery for polyposis. No pathogenic mutations in MBD4 were detected in the 27 patients tested, but 6 single nucleotide polymorphisms of uncertain functional significance were identified. Pathogenic biallelic MYH mutations were detected in 1 patient. CONCLUSIONS: Mutations in MBD4 are unlikely to be implicated in HPS; MYH mutations should be studied, especially when adenomas occur in the same patient. The clinical, histopathologic, and molecular findings of this study should contribute to our understanding of HPS and its relationship to the serrated neoplasia pathway.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Advanced colorectal polyps with the molecular and morphological features of serrated polyps and adenomas: concept of a 'fusion' pathway to colorectal cancer</a>, Histopathology, 2006.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Jass JR, Baker K, Zlobec I, Higuchi T, Barker M, Buchanan D, Young J</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Histopathology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">49</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2006</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16879389">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16879389</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adenoma/genetics/metabolism/pathology Colonic Polyps/genetics/metabolism/*pathology Colorectal Neoplasms/genetics/metabolism/*pathology DNA Mutational Analysis Humans Immunohistochemistry *Mutation O(6)-Methylguanine-DNA Methyltransferase/metabolism Proto-Oncogene Proteins B-raf/*genetics Tumor Suppressor Protein p53/metabolism ras Proteins/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">AIM: To establish and explain the pattern of molecular signatures across colorectal polyps. METHODS AND RESULTS: Thirty-two sessile serrated adenomas (SSA), 10 mixed polyps (MP), 15 traditional serrated adenomas (SA), 49 hyperplastic polyps (HP) and 84 adenomas were assessed for mutation of KRAS and BRAF and aberrant expression of p53. The findings were correlated with loss of expression of O-6-methylguanine DNA methyltransferase (MGMT). KRAS mutation occurred more frequently (26.5%) than BRAF mutation (4.8%) in adenomas (P < 0.001) and particularly in adenomas with villous architecture (50%). Loss of expression of MGMT correlated with KRAS mutation in small tubular adenomas (P < 0.04). BRAF mutation was frequent in HPs (67%) and SSAs (81%), while KRAS mutation was infrequent (4% and 3%, respectively). Of MPs and SAs, 72% had either BRAF or KRAS mutation. Aberrant expression of p53 was uncommon overall, but occurred more frequently in MPs and SAs (12%) than adenomas (1%) (P < 0.04) and there was concordant loss of expression of MGMT. CONCLUSIONS: Molecular alterations that are characteristic of the serrated pathway and adenoma-carcinoma sequence can co-occur in a minority of advanced colorectal polyps that then show morphological features of both pathways. These lesions account for only 2% of colorectal polyps, but may be relatively aggressive.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Extensive DNA methylation in normal colorectal mucosa in hyperplastic polyposis</a>, Gut, 2006.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Minoo P, Baker K, Goswami R, Chong G, Foulkes WD, Ruszkiewicz AR, Barker M, Buchanan D, Young J, Jass JR</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Gut</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">55</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">10</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2006</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16469793">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16469793</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adaptor Proteins, Signal Transducing/metabolism Adult Aged Colonic Polyps/*genetics CpG Islands *DNA Methylation DNA, Neoplasm/metabolism Female Genetic Predisposition to Disease Humans Immunohistochemistry Male Middle Aged Mutation/*genetics Nuclear Proteins/metabolism Polymerase Chain Reaction/methods Proto-Oncogene Proteins B-raf/*genetics Tumor Markers, Biological/metabolism</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND: Hyperplastic polyposis of the colorectum is a precancerous condition that has been linked with DNA methylation. The polyps in this condition have been distinguished from typical small hyperplastic polyps and renamed sessile serrated adenomas. Sessile serrated adenomas also occur sporadically and appear to be indistinguishable from their counterparts in hyperplastic polyposis. AIMS AND METHODS: The existence of distinguishing molecular features was explored in a series of serrated polyps and matched normal mucosa from patients with and without hyperplastic polyposis by assessing mutation of BRAF, DNA methylation in 14 markers (MINTs 1, 2 and 31, p16, MGMT, MLH1, RASSF1, RASSF2, NORE1 (RASSF5), RKIP, MST1, DAPK, FAS, and CHFR), and immunoexpression of MLH1. RESULTS: There was more extensive methylation in sessile serrated adenomas from subjects with hyperplastic polyposis (p<0.0001). A more clearcut difference in patients with hyperplastic polyposis was the finding of extensive DNA methylation in normal mucosa from the proximal colon. CONCLUSIONS: A genetic predisposition may underlie at least some forms of hyperplastic polyposis in which the earliest manifestation may be hypermethylation of multiple gene promoters in normal colorectal mucosa. Additionally, some of the heterogeneity within hyperplastic polyposis may be explained by different propensities for MLH1 inactivation within polyps.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>CpG island methylator phenotype underlies sporadic microsatellite instability and is tightly associated with BRAF mutation in colorectal cancer</a>, Nature genetics, 2006.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Weisenberger DJ, Siegmund KD, Campan M, Young J, Long TI, Faasse MA, Kang GH, Widschwendter M, Weener D, Buchanan D, Koh H, Simms L, Barker M, Leggett B, Levine J, Kim M, French AJ, Thibodeau SN, Jass J, Haile R, Laird PW</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Nature genetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">38</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2006</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16804544">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=16804544</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal Neoplasms/*genetics *CpG Islands *DNA Methylation DNA Repair/genetics DNA, Neoplasm/chemistry/genetics Epigenesis, Genetic Gene Silencing Genomic Instability Humans Microsatellite Repeats Models, Genetic *Mutation Phenotype Promoter Regions, Genetic Proto-Oncogene Proteins B-raf/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Aberrant DNA methylation of CpG islands has been widely observed in human colorectal tumors and is associated with gene silencing when it occurs in promoter areas. A subset of colorectal tumors has an exceptionally high frequency of methylation of some CpG islands, leading to the suggestion of a distinct trait referred to as 'CpG island methylator phenotype', or 'CIMP'. However, the existence of CIMP has been challenged. To resolve this continuing controversy, we conducted a systematic, stepwise screen of 195 CpG island methylation markers using MethyLight technology, involving 295 primary human colorectal tumors and 16,785 separate quantitative analyses. We found that CIMP-positive (CIMP+) tumors convincingly represent a distinct subset, encompassing almost all cases of tumors with BRAF mutation (odds ratio = 203). Sporadic cases of mismatch repair deficiency occur almost exclusively as a consequence of CIMP-associated methylation of MLH1 . We propose a robust new marker panel to classify CIMP+ tumors.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2005</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>GSK3B polymorphisms alter transcription and splicing in Parkinson's disease</a>, Annals of neurology, 2005.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Kwok JB, Hallupp M, Loy CT, Chan DK, Woo J, Mellick GD, Buchanan DD, Silburn PA, Halliday GM, Schofield PR</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Annals of neurology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">58</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">6</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2005</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ana.20691</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">16315267</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ana.20691">http://dx.doi.org/10.1002/ana.20691</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Brain/physiology Cohort Studies Exons/genetics Female Genetic Predisposition to Disease/epidemiology Glycogen Synthase Kinase 3/*genetics/metabolism Haplotypes Humans Linkage Disequilibrium Lymphocytes/physiology Male Parkinson Disease/epidemiology/*genetics/metabolism Phosphorylation *Polymorphism, Single Nucleotide Promoter Regions, Genetic/genetics RNA Splicing/*physiology Risk Factors Transcriptional Activation/*physiology tau Proteins/genetics/metabolism</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Parkinson's disease (PD) is a neurodegenerative disorder characterized by a combination of motor symptoms. We identified two functional single nucleotide polymorphisms in the glycogen synthase kinase-3beta gene (GSK3B). A promoter single nucleotide polymorphism (rs334558) is associated with transcriptional strength in vitro in which the T allele has greater activity. An intronic single nucleotide polymorphism (rs6438552) regulates selection of splice acceptor sites in vitro. The T allele is associated with altered splicing in lymphocytes and increased levels of GSK3B transcripts that lack exons 9 and 11 (GSKDeltaexon9+11). Increased levels of GSKDeltaexon9+11 correlated with enhanced phosphorylation of its substrate, Tau. In a comparison of PD and control brains, there was increased in frequency of T allele (rs6438552) and corresponding increase in GSKDeltaexon9+11 and Tau phosphorylation in PD brains. Conditional logistic regression indicated gene-gene interaction between T/T genotype of rs334558 and H1/H1 haplotype of microtubule-associated protein Tau (MAPT) gene (p = 0.009). There was association between a haplotype (T alleles of both GSK3B polymorphisms) and disease risk after stratification by Tau haplotypes ((H1/H2+H2/H2 individuals: odds ratio, 1.64; p = 0.007; (H1/H1 individuals: odds ratio, 0.68; p < 0.001). Ours results suggest GSK3B polymorphisms alter transcription and splicing and interact with Tau haplotypes to modify disease risk in PD.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Evidence for BRAF mutation and variable levels of microsatellite instability in a syndrome of familial colorectal cancer</a>, Clinical gastroenterology and hepatology: the official clinical practice journal of the American Gastroenterological Association, 2005.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Young J, Barker MA, Simms LA, Walsh MD, Biden KG, Buchanan D, Buttenshaw R, Whitehall VL, Arnold S, Jackson L, Kambara T, Spring KJ, Jenkins MA, Walker GJ, Hopper JL, Leggett BA, Jass JR</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Clinical gastroenterology and hepatology: the official clinical practice journal of the American Gastroenterological Association</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2005</dd>
              
                  
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=15765445">http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Citation&list_uids=15765445</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Colorectal Neoplasms/*genetics CpG Islands/genetics DNA Methylation Female Genetic Predisposition to Disease Genomic Instability/genetics Humans Male Microsatellite Repeats/*genetics Mutation Proto-Oncogene Proteins B-raf/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">BACKGROUND AND AIMS: Recently, an alternative pathway of tumorigenesis has been identified in the colorectum associated with serrated precursor lesions, variable levels of microsatellite instability (MSI-V), and driven in part by activating mutations in the BRAF proto-oncogene (V599E). Somatic BRAF mutations in hereditary nonpolyposis colon cancer (HNPCC) are rarely observed. Here, we discuss their role in the development of other familial colorectal cancers (CRC). We studied non-FAP, non-HNPCC CRC families characterized by tumors that varied in their level of MSI between individual members. METHODS: A subset of tumors from a total of 55 collected (25 polyps and 30 cancers) from 43 individuals across 11 families underwent pathology review, examination for V599E using allele-specific polymerase chain reaction, and for methylation of the MINT31 CpG island. RESULTS: All MSI-V families met the current revised Bethesda Guidelines and 6 of 11 (55%) met the Amsterdam I criteria. V599E was observed in 12 of 19 (63%) polyps and 14 of 20 (70%) cancers (4 of 4 high MSI, 2 of 4 low MSI, and 8 of 12 stable MSI), a significant increase over HNPCC (0 of 15 or 0%), and unselected CRC (30 of 197 or 15.2%) ( P < .05). Eight of the 10 (80%) cancers that underwent analysis showed hypermethylation of MINT31. CRCs showed early age at onset and were more likely to show a serrated architecture than unselected CRCs ( P < .05). CONCLUSION: These data provide evidence that the families described here represent a syndrome of familial CRC that is distinct from HNPCC. High levels of BRAF mutation and MINT31 hypermethylation suggest an origin in the serrated pathway of CRC development.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2004</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Sequence variation in the proximity of IDE may impact age at onset of both Parkinson disease and Alzheimer disease</a>, Neurogenetics, 2004.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Blomqvist ME, Silburn PA, Buchanan DD, Andreasen N, Blennow K, Pedersen NL, Brookes AJ, Mellick GD, Prince JA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Neurogenetics</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">5</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2004</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s10048-004-0173-4</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">15088150</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s10048-004-0173-4">http://dx.doi.org/10.1007/s10048-004-0173-4</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Age of Onset Aged Aged, 80 and over Alzheimer Disease/*genetics *Chromosomes, Human, Pair 10 Female Genotype Humans Insulysin/*genetics *Linkage Disequilibrium Male Middle Aged Parkinson Disease/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">We recently reported that a linkage disequilibrium (LD) block on chromosome 10q encompassing the gene encoding insulin-degrading enzyme ( IDE) harbors sequence variants that associate with Alzheimer disease (AD). Evidence also indicated effects upon a number of quantitative indices of AD severity, including age-at-onset (AAO). Since linkage of this immediate region to AAO has been shown in both AD and Parkinson disease (PD), we have explored the possibility that polymorphism within this LD block might also influence PD. Utilizing single nucleotide polymorphisms that delineate common haplotypes from this region, we observed significant evidence of association with AAO in an Australian PD case-control sample. Analyses were complemented with AAO data from two independent Swedish AD case samples, for which previously reported findings were replicated. Results were consistent between AD and PD, suggesting the presence of equivalent detrimental and protective alleles. These data highlight a genomic region in the proximity of IDE that may contribute to AD and PD in a similar manner.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Tau haplotypes regulate transcription and are associated with Parkinson's disease</a>, Annals of neurology, 2004.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Kwok JB, Teber ET, Loy C, Hallupp M, Nicholson G, Mellick GD, Buchanan DD, Silburn PA, Schofield PR</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Annals of neurology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">55</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">3</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2004</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1002/ana.10826</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">14991810</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1002/ana.10826">http://dx.doi.org/10.1002/ana.10826</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Adult Age of Onset Aged Alleles Cell Line Disease Susceptibility Female Haplotypes/*genetics Humans Linkage Disequilibrium Male Middle Aged Parkinson Disease/*genetics/physiopathology Polymerase Chain Reaction/methods Polymorphism, Single Nucleotide/genetics Promoter Regions, Genetic/genetics Transcription, Genetic/*genetics tau Proteins/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">A primary haplotype (H1) of the microtubule-associated protein Tau (MAPT) gene is associated with Parkinson's disease (PD). However, the mechanism for disease susceptibility remains unknown. We examined the promoter region of MAPT and identified single nucleotide polymorphisms and insertions of 1 to 11 nucleotides. These polymorphisms corresponded to the previously characterized haplotypes, H1 and H2, as well as a novel variant of the H1 haplotype, H1'. As observed in other studies, we demonstrated a significant association with the H1/H1 promoter genotype and PD in a cohort of 206 idiopathic late-onset cases. This is in contrast with a panel of 13 early-onset PD patients, for whom we did not detect any mutations in MAPT. By examining single nucleotide polymorphisms in adjacent genes, we showed that linkage disequilibrium does not extend beyond the MAPT haplotype to neighboring genes. To define the mechanism of disease susceptibility, we examined the transcriptional activity of the promoter haplotypes using a luciferase reporter assay. We demonstrated in two human cell lines, SK-N-MC and 293, that the H1 haplotype was more efficient at driving gene expression than the H2 haplotype. Our data suggest that an increase in expression of the MAPT gene is a susceptibility factor in idiopathic PD.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2002</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>The Cys282Tyr polymorphism in the HFE gene in Australian Parkinson's disease patients</a>, Neuroscience letters, 2002.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Buchanan DD, Silburn PA, Chalk JB, Le Couteur DG, Mellick GD</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Neuroscience letters</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">327</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2002</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">12098643</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/12098643">https://www.ncbi.nlm.nih.gov/pubmed/12098643</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Australia Case-Control Studies Cohort Studies Female Genetic Predisposition to Disease/epidemiology Genotype Hemochromatosis/*epidemiology/*genetics Humans Iron Overload/genetics Male Middle Aged Parkinson Disease/*epidemiology/*genetics *Polymorphism, Single Nucleotide Risk Factors</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Iron homeostasis is altered in Parkinson's disease (PD). The HFE protein is an important regulator of cellular iron homeostasis and variations within this gene can result in iron overload and the disorder known as hereditary haemochromatosis. We studied the Cys282Tyr single nucleotide polymorphism as a genetic risk factor for PD in two distinct and separately collected cohorts of Australian PD patients and controls. In the combined cohort comprising 438 PD patients and 485 control subjects, we revealed an odds ratio for possession of the 282Tyr allele of 0.61 (95% confidence interval, CI=0.42-0.90, P=0.011) from univariate chi-squared and 0.59 (95% CI=0.39-0.90, P=0.014) after logistic regression analyses (correcting for potential confounding factors). These results suggest that possession of the 282Tyr allele may offer some protection against the development of PD.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Lack of association between CYP1A1 polymorphism and Parkinson's disease in a Chinese population</a>, Journal of neural transmission , 2002.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Chan DK, Mellick GD, Buchanan DD, Hung WT, Ng PW, Woo J, Kay R</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of neural transmission </dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">109</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2002</dd>
              
                              <dt class="col-sm-2">DOI</dt>
                <dd class="col-sm-10">10.1007/s007020200002</dd>
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">11793160</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="http://dx.doi.org/10.1007/s007020200002">http://dx.doi.org/10.1007/s007020200002</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Asian Continental Ancestry Group/*genetics Cytochrome P-450 CYP1A1/*genetics Female Hong Kong Humans Male Parkinson Disease/*genetics *Polymorphism, Genetic</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Apart from very few families who have a direct cause from genetic mutation, causes of most Parkinson's disease (PD) remain unclear. Many allelic association studies on polymorphism of different candidate genes have been studied. Although these association studies do not imply a causal relationship, it does warrant further studies to elucidate the pathophysiologic significance. CYP1A1 polymorphisms have been reported to be associated with PD in a Japanese population sample. Since CYP1A1 transforms aromatic hydrocarbons into products that may be neurotoxic and perhaps lead to PD, we therefore undertook a study to look at the possible association of CYP1A1 polymorphism and PD in a Chinese population. Contrary to the Japanese result, we did not find any statistically significant difference between the PD group and the control group in our study with a bigger sample size.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2001</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>The parkin gene S/N167 polymorphism in Australian Parkinson's disease patients and controls</a>, Parkinsonism & related disorders, 2001.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mellick GD, Buchanan DD, Hattori N, Brookes AJ, Mizuno Y, Le Couteur DG, Silburn PA</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Parkinsonism & related disorders</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">7</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2001</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">11248588</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/11248588">https://www.ncbi.nlm.nih.gov/pubmed/11248588</a></dd>
                  
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">This study determined the frequencies of a G-to-A transition (S/N167) polymorphism in exon 4 of the parkin gene in Australian Parkinson's disease patients and control subjects. The genotype of each subject was determined using the polymerase chain reaction and restriction-fragment-length-polymorphism analysis. Overall, the A allele was significantly less common in the Parkinson's disease group (1.7%) compared with the control group (3.8%, OR=0.43, 95% CI=0.19-1.00, P<0.05), although the frequency in the young onset Parkinson's disease group (6.6%) was not significantly different to controls. The A allele is less common in Australian Caucasian subjects compared to Japanese Parkinson's disease patients and appears to be under-represented in older-onset Parkinson's disease.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">2000</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>The monoamine oxidase B gene GT repeat polymorphism and Parkinson's disease in a Chinese population</a>, Journal of neurology, 2000.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mellick GD, Buchanan DD, Silburn PA, Chan DK, Le Couteur DG, Law LK, Woo J, Pang CP</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Journal of neurology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">247</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">1</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">2000</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">10701898</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/10701898">https://www.ncbi.nlm.nih.gov/pubmed/10701898</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Alleles *Dinucleotide Repeats Female Hong Kong Humans Introns Male Monoamine Oxidase/*genetics Parkinson Disease/*genetics *Polymorphism, Genetic</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">Monoamine oxidase B (MAOB) metabolises dopamine and activates neurotoxins known to induce parkinsonism in humans and primates. Therefore the MAOB gene (MAOB; Xp15.21-4) is a candidate gene for Parkinson's disease (PD). Longer length dinucleotide repeat sequences in a highly polymorphic GT repeat region of intron 2 of this gene showed an association with PD in an Australian cohort. We repeated this allele-association study in a population of 176 Chinese PD patients (90 men, 86 women) and 203 agematched controls (99 men, 104 women). Genomic DNA was extracted from venous blood and the polymerase chain reaction was used to amplify the appropriate regions of the MAOB gene. The length of each (GT) repeat sequence was determined by 5% polyacrylamide denaturing gel electrophoresis. There was no significant difference in allele frequencies of the (GT) repeat allelic variation between patients and controls (chi2 = 2.48; df = 5, P<0.75). Therefore the longer length GT repeat alleles are not associated with PD in this Chinese population. Possible reasons for the discrepancy between Chinese and Australian populations include a different interaction between this genetic factor and environmental factors in the two populations and the possibility that the long length GT repeat alleles may represent a marker mutation, genetically linked to another susceptibility allele in whites but not in Chinese. Methodological differences in the ascertainment of cases and controls in this cohort could also explain the observed differences. Further study is required to determine whether the longer length GT repeat alleles are true susceptibility alleles in PD.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

  
  <h3 class="my-5">1999</h3>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>The ACE deletion polymorphism is not associated with Parkinson's disease</a>, European neurology, 1999.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mellick GD, Buchanan DD, McCann SJ, Davis DR, Le Couteur DG, Chan D, Johnson AG</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">European neurology</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">41</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">1999</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">10023114</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/10023114">https://www.ncbi.nlm.nih.gov/pubmed/10023114</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Female *Gene Deletion Genotype Humans Male Middle Aged Parkinson Disease/*genetics Peptidyl-Dipeptidase A/*genetics Polymorphism, Genetic/*genetics</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The deletion allele (D allele) polymorphism in the angiotensin converting enzyme (ACE) gene is associated with increased levels of the neuropeptide substance P in the basal ganglia and substantia nigra. A reduction of substance P levels in the brain occurs in Parkinson's disease (PD) and has been implicated in the pathogenesis of the disease. We investigated the hypothesis that the D allele may be protective towards PD by examining the frequency of the ACE (I/D) polymorphism in 178 PD cases (male:female ratio = 1.4) and 192 controls (male:female ratio = 1.5). ACE (I/D) genotype was determined using polymerase chain reaction and 3% agarose gel electrophoresis. Unadjusted chi-square analysis revealed no significant difference between genotype frequencies (chi2 = 3.30, p > 0.10) or allele frequencies (chi2 = 2.52, p > 0.10) between patient and control groups, although PD patients were less likely to be homozygous (OR = 0.80, 95% CI = 0.49-1.29) or heterozygous (OR = 0.80, 95% CI = 0.59-1.06) for the D allele. A stepwise logistic regression analysis of the ACE deletion and risk factor data confirmed that there was no significant association between the ACE deletion (D allele) polymorphism and PD (OR = 0.62, 95% CI = 0.35-1. 10, p = 0.10). This study does not support the hypothesis that the D allele of the ACE gene confers a protective effect with respect to PD.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

      
    <ul style="list-style-type:none">
      <li>
        <details>
          <summary>Variations in the monoamine oxidase B (MAOB) gene are associated with Parkinson's disease</a>, Movement disorders: official journal of the Movement Disorder Society, 1999.</summary> 
          <div class="card card-body my-3">
            <dl class="row">
                              <dt class="col-sm-2">Authors</dt>
                <dd class="col-sm-10">Mellick GD, Buchanan DD, McCann SJ, James KM, Johnson AG, Davis DR, Liyou N, Chan D, Le Couteur DG</dd>
                  
                              <dt class="col-sm-2">Journal</dt>
                <dd class="col-sm-10">Movement disorders: official journal of the Movement Disorder Society</dd>
                  
                              <dt class="col-sm-2">Volume</dt>
                <dd class="col-sm-10">14</dd>
              
                              <dt class="col-sm-2">Issue</dt>
                <dd class="col-sm-10">2</dd>
              
                              <dt class="col-sm-2">Year</dt>
                <dd class="col-sm-10">1999</dd>
              
                  
                              <dt class="col-sm-2">Pubmed ID</dt>
                <dd class="col-sm-10">10091612</dd>
                  
                              <dt class="col-sm-2">URL</dt>
                <dd class="col-sm-10"><a href="https://www.ncbi.nlm.nih.gov/pubmed/10091612">https://www.ncbi.nlm.nih.gov/pubmed/10091612</a></dd>
                  
                              <dt class="col-sm-2">Keywords</dt>
                <dd class="col-sm-10">Aged Analysis of Variance Case-Control Studies Cohort Studies Dinucleotide Repeats Female Gene Dosage Humans Male Monoamine Oxidase/*genetics New South Wales/epidemiology Parkinson Disease/epidemiology/*genetics Polymerase Chain Reaction Polymorphism, Genetic/*genetics Queensland/epidemiology Statistics as Topic</dd>
                  
                              <dt class="col-sm-2">Abstract</dt>
                <dd class="col-sm-10">The monoamine oxidase B gene (MAOB; Xp15.21-4) is a candidate gene for Parkinson's disease (PD) given its role in dopamine metabolism and its possible role in the activation of neurotoxins. The association of MAOB polymorphisms (a [GT] repeat allelic variation in intron 2 and an A-G transition in intron 13) with Parkinson's disease (PD) was studied in an Australian cohort of 204 (male:female ratio 1.60) people with PD and 285 (male:female ratio 1.64) age- and gender-matched control subjects. Genomic DNA was extracted from venous blood and polymerase chain reaction was used to amplify the appropriate regions of the MAOB gene. The length of each (GT) repeat sequence was determined by 5% polyacrylamide denaturing gel electrophoresis and a DNA fragment analyzer, while the G-A genotype was determined using 2% agarose gel electrophoresis. The G-A polymorphism showed no association with PD (odds ratio [OR] = 0.80; p = 0.51; 95% confidence interval [CI] = 0.42-1.53). There was a significant difference in allele frequencies of the (GT) repeat allelic variation between patients and control subjects (chi2 = 20.09; p or =188 base pairs in the intron 2 marker of the MAOB gene were significantly associated with PD (OR = 4.60; p<0.00005; 95% CI = 1.97-10.77). The 186 base pair allele was also significantly associated with PD (OR = 1.85; p = 0.048; 95% CI = 1.01-3.42). The GT repeat in intron 2 of the MAOB gene is a powerful marker for PD in this large Australian cohort.</dd>
                          </dl>
          </div>
        </details>
      </li>
    </ul>

        </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
