<!doctype html>
<html lang="en">
  <head>
     
    <title>Colorectal Oncogenomics Group: Team</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css"
          integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <link href="assets/css/site.css" rel="stylesheet" />
      </head>

  <body>
      <div class="container"> 
      <!-- <div class="container-fluid"">  -->
      
      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="index.html">COG Lab</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
              <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
              <div class="navbar-nav">

                                  <a class="nav-item nav-link"href="index.html">Home</a>
                                  <a class="nav-item nav-link"href="projects.html">Projects</a>
                                  <a class="nav-item nav-link active"href="team.html">Team</a>
                                  <a class="nav-item nav-link"href="publications.html">Publications</a>
                                  <a class="nav-item nav-link"href="presentations.html">Presentations</a>
                                  <a class="nav-item nav-link"href="funding.html">Funding</a>
                                  <a class="nav-item nav-link"href="studies.html">Studies</a>
                                  <a class="nav-item nav-link"href="contact.html">Contact</a>
                
              </div>
            </div>
          </nav>
      

<h1 class="display-4 my-5">Team members</h1>

      <div class="card my-5">
      <h4 class="card-header">Assoc Prof Daniel Buchanan</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Laboratory Head</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/dan_buchanan_bw.jpg" alt="Assoc Prof Daniel Buchanan">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Associate Professor</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Joint Appointment, Department of Clinical Pathology and Centre for Epidemiology and Biostatistics</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="https://scholar.google.com.au/citations?user=MniMp-IAAAAJ&hl=en">Google scholar</a></dd>
                                                  <dt class="col-sm-3">Phone</dt>
                  <dd class="col-sm-9">+61 3 8559 7004</dd>
                                                  <dt class="col-sm-3">Mobile</dt>
                  <dd class="col-sm-9">+61 405060245</dd>
                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:daniel.buchanan@unimelb.edu.au">Assoc Prof Daniel Buchanan</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>Dr Buchanan received his PhD in Molecular Genetics in 2010 from The University of Queensland’s School of Medicine.</p><p>In 2014, he moved his laboratory to the Department of Pathology at The University of Melbourne after 10 years at the QIMR Berghofer Medical Research Institute. His research focuses on the genetic classification, aetiology and architecture of colorectal cancer. His current research projects focus on 1) identifying novel genetic risk factors for colorectal cancer using next generation sequencing, 2) identifying biologically and clinically relevant subtypes of colorectal neoplasia through tumour profiling and integration of genetic, epigenetic, somatic mutations and histopathological features, and 3) Strategies to identify colorectal cancer-affected individuals within the population who carry mutations in established high-risk genes for Lynch syndrome and polyposis syndromes.</p><p>He has over 100 peer-reviewed publications on various aspects of hereditary cancer syndromes including diagnostic and translational applications.  As a result of his publications on tumour and germline testing to identify mismatch repair gene mutations (including the utility of testing adenomas, breast and prostate cancers), he was invited to join the scientific advisory board of the Lynch Syndrome Australia organization.</p><p>Dr Buchanan is the Co-Principal investigator of the Australasian Colorectal Cancer Family Registry, a member of the steering committee for the International Colon Cancer Family Registry Cohort (C-CFR) and the Melbourne Collaborative Cohort Study (MCCS) for colorectal cancer.  He is the Chair of both the "Gene discovery and Post-GWAS" and "Molecular Pathology" working groups of the C-CFR. Since 2006, he has led the molecular genetic testing and the biorepository activities for the Australasian site of the International Colon Cancer Family Registry (Colon-CFR), a $23m National Institution of Health (NIH) project that has been funded for four consecutive five-year periods (1997–2017) to become Australia’s largest repository of colorectal cancer family data. During this time he has successfully led several large scale molecular testing projects for the Colon-CFR and MCCS including whole-exome and whole genome sequencing of &gt;200 colorectal cancer-affected and polyp-affected individuals, SNP genotyping for &gt;10,000 participants, DNA methylation for &gt;3000 CRCs and somatic mutation testing for &gt;10,000 colorectal cancers and polyps. Dr Buchanan is also a key member of the research team on a NHMRC Centre for Research Excellence to identify inherited genetic risk factors for colorectal cancer. He is responsible for the bioresources of the Australasian Colorectal Cancer Family Registry (ACCFR). Since 2006, he has supervised over 20 researchers, post-doctoral fellows and technicians with wide-ranging skill sets required for internationally competitive research for this and other studies.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Dr Bernard Pope</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Bioinformatician</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/bernie_pope_bw.jpg" alt="Dr Bernard Pope">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Senior Research Fellow</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Lead Bioinformatician (Cancer and Clincal Genomics) <a href="http://www.melbournebioinformatics.org.au">Melbourne Bioinformatics</a>; Senior Honorary Fellow Department of Clinical Pathology; Adjunct Associate Professor (Research), Department of Medicine, Central Clinical School, Monash University</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="http://www.berniepope.id.au/publications.html">www.berniepope.id.au/publications.html</a></dd>
                                                  <dt class="col-sm-3">Phone</dt>
                  <dd class="col-sm-9">+61 3 8344 0434</dd>
                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:bjpope@unimelb.edu.au">Dr Bernard Pope</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>After completing a national-award winning PhD in computer science in 2007, and then a research internship at Microsoft Research in Cambridge, Bernard returned to the University of Melbourne to take up a lecturing position in the Department of Computer Science and Software Engineering. In 2010 he joined the newly-formed VLSCI (now Melbourne Bioinformatics), a $100M project to build a world-leading centre for bioinformatics and computational biology. In making the transition from computer science to bioinformatics, Bernard recognised the exciting new opportunities provided by the genomics revolution in biomedicine, and was inspired to apply his computing and analytic skills to important problems in human health.</p><p>During his years at Melbourne Bioinformatics, Bernard has established himself as an expert in the development of novel techniques for the analysis of large-scale genomics data sets, resulting in the production of tools which have driven key discoveries in human disease, and which can be re-applied in many different contexts. As Lead Bioinformatician for Human Genomics, Bernard heads several high profile, multidisciplinary research projects, with national and international partners.</p><p>In 2017 Bernard was awarded a Victorian Health and Medical Research Fellowship, funded by the Victorian Government, supporting his work on the prevention and treatment of colorectal cancer. Australia has one of the highest rates of colorectal cancer (CRC) in the world. Survival from CRC and patient quality of life are significantly improved by early detection, treatment, and risk management. Therefore, identifying those individuals within the population who are at the highest risk of developing CRC offers the greatest potential for its prevention. Bernard and his collaborators aim to develop new diagnostic tools that will personalise risk assessment for a large group of high-risk individuals, leading to better-targeted screening and medical intervention, aimed at those who will benefit most.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Assoc Prof Christophe Rosty</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Honorary Research Fellow</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/christophe_rosty.png" alt="Assoc Prof Christophe Rosty">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Associate Professor</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="https://scholar.google.com/citations?user=XpK3Kv4AAAAJ&hl=en">Google scholar</a></dd>
                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:c.rosty@unimelb.edu.au">Assoc Prof Christophe Rosty</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>Dr Rosty is a specialist gastrointestinal Anatomical Pathologist, trained in Paris and graduated in 1997. His interest in gastro-intestinal cancer research started during his residency when he started his PhD which was completed in 2002 (University Necker – Paris 5). He spent 2 years in the Gastrointestinal Pathology Department at the Johns Hopkins University in Baltimore, USA, as a Postdoctoral Research Fellow. In 2009, Dr Rosty moved to Brisbane and currently works as a specialist Pathologist at Envoi Pathology, a pathology practice exclusively dedicated to gastro-intestinal and liver pathology with strong teaching and academic activities (<a href="http://www.envoi.com.au">http://www.envoi.com.au</a>).</p><p>His collaboration with Dr Buchanan started in 2009 at the QIMR Berghofer Medical Research Institute in Brisbane. His research interests include the genetics and the molecular pathology of gastrointestinal neoplasms, in particular Lynch syndrome, serrated polyposis and hereditary gastrointestinal polyposis syndromes.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Dr Mark Clendenning</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Molecular Biologist and Geneticist</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/mark_clendenning.jpg" alt="Dr Mark Clendenning">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Research Fellow</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="https://scholar.google.com.au/citations?user=gGezr5oAAAAJ&hl=en">Google scholar</a></dd>
                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:mark.clendenning@unimelb.edu.au">Dr Mark Clendenning</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p><a href="https://www.researchgate.net/profile/Mark_Clendenning">Research Gate entry.</a></p><p>Mark is a serial expatriate from the UK, with a B.Sc. in Genetics from the University of Sheffield. He obtained his Ph.D. through the University of Bristol, whilst studying the genomic architecture of the chicken Natural Killer Complex and its relation to Marek’s disease resistance.</p><p>Upon the completion of his Ph.D. he took up a postdoctoral position at the Ohio State University under the mentorship of the world-renowned human geneticist Albert de la Chapelle. Here, Mark utilised the molecular techniques and knowledge acquired from working with the chicken genome to advance our understanding of the mutational spectrum involved in Lynch syndrome, a genetic disease which predisposes affected individuals to a spectrum of cancers, primarily colorectal (bowel) and endometrial (uterus).</p><p>In 2008, Mark joined the familial cancer laboratory at QIMR in Brisbane where he was able to apply his molecular knowledge to the Australian colon cancer family cohort. Currently, Mark continues to work on Lynch syndrome as well as other subtypes of familial colorectal cancer, within the Oncogenomics Group at the University of Melbourne.</p><p>In particular, Mark’s work has focused on the PMS2 gene and through the development of a novel mutation screening method he has achieved worldwide recognition as an authority on the validity and interpretation of mutations within this gene.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Dr Harindra Jayasekara</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Epidemiologist</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/harindra_jayasekara.jpg" alt="Dr Harindra Jayasekara">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Research Fellow</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology; Post-Doctoral Fellow at Cancer Epidemiology & Intelligence Division, Cancer Council Victoria; Adjunct Fellow at the Centre for Alcohol Policy Research, La Trobe University</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="https://scholar.google.com.au/citations?user=ZneAmAQAAAAJ&hl=en">Google scholar</a></dd>
                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:harindra.jayasekara@unimelb.edu.au">Dr Harindra Jayasekara</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>MBBS MSc MD PhD</p><p>Harindra is a medically qualified epidemiologist. He completed his PhD in Epidemiology at The University of Melbourne in 2014 having been awarded a National Health and Medical Research Council postgraduate scholarship. He previously worked at the Centre for Epidemiology and Biostatistics, University of Melbourne, and the Murdoch Childrens Research Institute. He joined the Colorectal Oncogenomics Group in 2017. He is also a Post-Doctoral Fellow at Cancer Epidemiology & Intelligence Division, Cancer Council Victoria and an Adjunct Fellow at the Centre for Alcohol Policy Research, La Trobe University.</p><p>Harindra’s work focuses on the factors that affect the occurrence of cancer and survival from it. In particular, he has been examining the role of long-term alcohol consumption as a risk factor for cancer, and genetic and molecular factors associated with carcinogenesis.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Dr Eric Joo</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Molecular Biologist and Bioinformatician</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/eric_joo.jpg" alt="Dr Eric Joo">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Research Fellow</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="https://www.researchgate.net/profile/Eric_Joo">https://www.researchgate.net/profile/Eric_Joo</a></dd>
                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:ji.joo@unimelb.edu.au">Dr Eric Joo</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>B.Sc (Honours), PhD</p><p>Dr Joo is a molecular biologist and a bioinformatician with a strong background in cancer epigenetics, especially DNA methylation. He brings to the team both his laboratory and data analysis expertise in genome-wide methylation techniques and methodology. He specialises in molecular techniques and analyses of genome-wide as well as loci-specific detection of DNA methylation and gene expression using a variety of platforms, including those based on microarrays and NGS.</p><p>Dr Joo completed his PhD at the Murdoch Childrens Research Institute in 2012. After carrying out his first postdoc at the Genetic Epidemiology Laboratory (The University of Melbourne), he joined the Colorectal Oncogenomics Group in 2018. He has special interests in the epigenetic aspects of colorectal cancer. His research focuses on identifying aberrant DNA methylation marks associated with the polyp-to-cancer progression as well as DNA methylation marks associated with specific subsets of colorectal cancer including early onset CRC.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Dr Khalid Mahmood</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Bioinformatician</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/khalid_mahmood_bw.jpg" alt="Dr Khalid Mahmood">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Research Fellow</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Melbourne Bioinformatics</dd>
                                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:khalid.mahmood@unimelb.edu.au">Dr Khalid Mahmood</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>Dr. Khalid Mahmood is a bioinformatician in the lab, being trained in computational biology from Monash University. The focus of his PhD was the development of computational algorithms for comparative genomics with emphasis on identifying orthologous genes, homologous genomic segments and other complex evolutionary events when comparing genomes. Khalid is also experienced in studying relationships between protein sequences and structures.</p><p>Khalid’s recent work in the lab focuses on bringing together computational methods and genomics to understand genetics of colorectal cancer. Khalid is very interested in developing methods to investigate the relationship between genetic mutations and their impact on protein function and relationship with disease.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Mr Peter Georgeson</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Bioinformatician</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/peter_georgeson_bw.jpg" alt="Mr Peter Georgeson">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">PhD candidate</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology</dd>
                                                  <dt class="col-sm-3">Publications</dt>
                  <dd class="col-sm-9"><a href="https://scholar.google.com.au/citations?user=uOi-Ge4AAAAJ&hl=en">Google scholar</a></dd>
                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:peter.georgeson@unimelb.edu.au">Mr Peter Georgeson</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>BEng (Hons I)/BMath MSc(CompSci)</p><p>Peter is a bioinformatician with a background in software engineering.</p><p>Peter started his career in industry as a software engineer, working on, among other things, establishing the UK's first independent online payment provider. Recently switching to bioinformatics, Peter completed his Master of Science (Computer Science) at the University of Melbourne in 2016. He was the recipient of numerous awards, including a place on the Dean's Merit List, the Johnny Smoes award for excellence in software development, and the Australian Artificial Intelligence Institute Prize.</p><p>More recently, while employed at Melbourne Bioinformatics, Peter specialised in the design, development and implementation of analysis pipelines and the application of novel analysis methods to large genomics datasets. Peter is particularly interested in the application of computational and statistical methods such as machine learning to genomics problems.</p><p>In 2018, Peter commenced a PhD with Associate Professor Dan Buchanan and Dr Bernard Pope, with a view to improving our understanding of the aetiology of colorectal cancer, to improve diagnosis, prevention, and treatment of the disease.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Ms Romy Walker</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Molecular Geneticist</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/romy_walker.jpg" alt="Ms Romy Walker">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">PhD candidate</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology</dd>
                                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:romy@student.unimelb.edu.au">Ms Romy Walker</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>Romy is a Human Geneticist having undergone her B. Sc. and M. Sc. Degree at the University Hospital Regensburg in Germany. Upon completion, she obtained a position in a routine Diagnostics Department, focusing on hereditary tumour and rare eye diseases.</p><p>During her 4-year stay at the Institute of Human Genetics, she commenced an advanced training program offered by The German Society of Human Genetics, which she is a member of. This 5-year educational program allows scientists certified as “clinical genetic laboratory specialist” to work as professionals in clinical genetics and patients care.</p><p>Beginning 2018, Romy has finalised her major in molecular genetics (3 years) and has started a PhD with Associate Professor Daniel Buchanan. The research focus will be set on establishing novel diagnostic approaches for risk stratification of individuals with mismatch repair-deficient cancer.</p>

      </div>
    </div>

      <div class="card my-5">
      <h4 class="card-header">Dr Mailie Gall</h4>
      <div class="card-body">
      <h6 class="card-subtitle mb-2 text-muted">Bioinformatician</h6>

        <p class="card-text">
	  <div class="row">
            <div class="col-4">
              <img class="img-thumbnail mr-3" src="assets/img/mailie_gall.jpg" alt="Dr Mailie Gall">
            </div>

            <div class="col-8">

              <dl class="row">
                                  <dt class="col-sm-3">Position</dt>
                  <dd class="col-sm-9">Masters of Science (Bioinformatics) candidate</dd>
                                                  <dt class="col-sm-3">Affiliation</dt>
                  <dd class="col-sm-9">Department of Clinical Pathology</dd>
                                                                                                  <dt class="col-sm-3">Email</dt>
                  <dd class="col-sm-9"><a href="mailto:mailie.gall@unimelb.edu.au">Dr Mailie Gall</a></dd>
                              </dl>
            </div>
	  </div>

          <h4 class="my-3">Career summary</h4>
          <p>Mailie is a bioinformatician and evolutionary (marine) ecologist, with a PhD in Marine Ecology (2016). Mailie started in the field of medical science (Bachelor in Medical Science, UNSW, 2009), but a discovered a passion for the marine sciences and went on to complete a Masters of Philosophy in Marine ecotoxicology with Professor Emma Johnston at UNSW (2011). Continuing along this theme, her PhD, Patterns of dispersal in the Tasman sea and South Pacific Ocean: population genetics and the role of environmental constraints, focussed on the ecology, larval biology, geographical distribution and population genetics of two closely-related Australian sea urchins. After finishing her PhD and with desire to upskill in computational biology fields, she started a Masters of Science (Bioinformatics) at the University of Melbourne. Alongside completion of her Masters, she has worked with the Australian Genomics Health Alliance (AGHA) and developers at Melbourne Bioinformatics, to develop a provenance language and visualisation tool to describe clinical bioinformatic pipelines. This tool has been used to describe the majority of clinical pipelines in operation across the AGHA and affiliated organisations.</p><p>Mailie joined the Colorectal Oncogenomics Group in 2018 and is currently working on Familial Colorectal Cancer Type X (FCCTX). The purpose of the project is to describe novel germline copy number variations across large familial SNP-array datasets. Mailie continues to work with collaborators in Sydney on Marine science projects including work on the Crown-of-thorns starfish on the Great Barrier Reef and population genetics of an Australian starfish Meriadastar calcar, in addition to ongoing work on manuscripts from her PhD.</p>

      </div>
    </div>

  
      </div> <!-- container -->

      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
      <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
      <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
      <script src="assets/js/site.js"></script>
      <script src="assets/js/details-element-polyfill.js"></script>
  </body>
</html>
